{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "authorship_tag": "ABX9TyNYx0mxl/DW7TUfWkqj1QmV",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/1799mrd/Bioinformatic_tools/blob/main/not_yet.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "G2PLfKtn1Cr5",
        "outputId": "e474a4fe-16c8-4f0a-8751-ba5ec1e61b2d"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Python 3.10.12\n"
          ]
        }
      ],
      "source": [
        "!python --version"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "p = \"\"\"\n",
        "import google\n",
        "import math\n",
        "from typing_extensions import override\n",
        "from Bio import pairwise2\n",
        "from Bio.SubsMat import MatrixInfo as matlist\n",
        "import pandas as pd\n",
        "import cv2\n",
        "\n",
        "\n",
        "series = ['\"Whither Canada\"', '\"Owl Stretching Time\"', '\"The Ant, an Introduction\"']\n",
        "text = f\"Do you know, which TV series is it: {', '.join(series)}\"\n",
        "\n",
        "recursion = []\n",
        "recursion = [recursion]\n",
        "\n",
        "class Base:\n",
        "  def get_seq_type(self) -> str:\n",
        "    return \"DNA\"\n",
        "\n",
        "class RnaChild(Base):\n",
        "  @override\n",
        "  def get_seq_type(self) -> str:\n",
        "    return \"RNA\"\n",
        "\n",
        "\n",
        "string = f\"{math.pi:.5f}\"\n",
        "series = pd.Series([-1.5, 0.2, None], dtype=\"float32[pyarrow]\")\n",
        "alignments = max(pairwise2.align.globalds(\"ATATCTCGATCGCTACGTC\", \"CTAGCTCGCTGCTCAGCATC\",\n",
        "                                          matlist.blosum62, -10, -0.5), key=lambda x: x.score)\n",
        "pd.DataFrame([1, 2, 3, 4], index=[1, 2, 3, 4])\n",
        "print(b'\\xd0\\x92\\xd1\\x81\\xd1\\x91 \\xd1\\x80\\xd0\\xb0\\xd0\\xb1\\xd0\\xbe\\xd1\\x82\\xd0\\xb0\\xd0\\xb5\\xd1\\x82, \\xd1\\x82\\xd1\\x8b \\xd0\\xb1\\xd0\\xbe\\xd0\\xbb\\xd1\\x8c\\xd1\\x88(\\xd0\\xbe\\xd0\\xb9/\\xd0\\xb0\\xd1\\x8f) \\xd0\\xbc\\xd0\\xbe\\xd0\\xbb\\xd0\\xbe\\xd0\\xb4\\xd0\\xb5\\xd1\\x86!!!!'.decode(\"utf-8\").removesuffix(\"!\"))\n",
        "\"\"\"\n",
        "\n",
        "c = \"\"\"text_file = open(\"textt.py\", \"w+\");text_file.write(p);text_file.close()\"\"\"\n",
        "\n",
        "exec(c)"
      ],
      "metadata": {
        "id": "BMlDKai91DQz"
      },
      "execution_count": 2,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "wGu4m1ho1Sy9",
        "outputId": "7bb0089f-cc2b-459a-d91c-de540aaae32b"
      },
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 4, in <module>\n",
            "    from Bio import pairwise2\n",
            "ModuleNotFoundError: No module named 'Bio'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install Bio"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "OLYaexVFAF9j",
        "outputId": "bb3e32b0-a882-4503-c91b-2424306deb2d"
      },
      "execution_count": 4,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting Bio\n",
            "  Downloading bio-1.6.0-py3-none-any.whl (279 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m279.4/279.4 kB\u001b[0m \u001b[31m5.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting biopython>=1.80 (from Bio)\n",
            "  Downloading biopython-1.81-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.1/3.1 MB\u001b[0m \u001b[31m20.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (from Bio) (2.31.0)\n",
            "Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (from Bio) (4.66.1)\n",
            "Collecting mygene (from Bio)\n",
            "  Downloading mygene-3.2.2-py2.py3-none-any.whl (5.4 kB)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.10/dist-packages (from Bio) (1.5.3)\n",
            "Requirement already satisfied: pooch in /usr/local/lib/python3.10/dist-packages (from Bio) (1.8.0)\n",
            "Collecting gprofiler-official (from Bio)\n",
            "  Downloading gprofiler_official-1.0.0-py3-none-any.whl (9.3 kB)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.10/dist-packages (from biopython>=1.80->Bio) (1.23.5)\n",
            "Collecting biothings-client>=0.2.6 (from mygene->Bio)\n",
            "  Downloading biothings_client-0.3.0-py2.py3-none-any.whl (29 kB)\n",
            "Requirement already satisfied: python-dateutil>=2.8.1 in /usr/local/lib/python3.10/dist-packages (from pandas->Bio) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.10/dist-packages (from pandas->Bio) (2023.3.post1)\n",
            "Requirement already satisfied: platformdirs>=2.5.0 in /usr/local/lib/python3.10/dist-packages (from pooch->Bio) (3.11.0)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.10/dist-packages (from pooch->Bio) (23.2)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests->Bio) (3.3.1)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests->Bio) (3.4)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests->Bio) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests->Bio) (2023.7.22)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/dist-packages (from python-dateutil>=2.8.1->pandas->Bio) (1.16.0)\n",
            "Installing collected packages: biopython, gprofiler-official, biothings-client, mygene, Bio\n",
            "Successfully installed Bio-1.6.0 biopython-1.81 biothings-client-0.3.0 gprofiler-official-1.0.0 mygene-3.2.2\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!apt install python3.10-venv\n",
        "!python -m venv venv"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "jZcYokTKBCp6",
        "outputId": "2f3c8d32-22a0-422f-8eea-c28db450bc52"
      },
      "execution_count": 7,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  python3-pip-whl python3-setuptools-whl\n",
            "The following NEW packages will be installed:\n",
            "  python3-pip-whl python3-setuptools-whl python3.10-venv\n",
            "0 upgraded, 3 newly installed, 0 to remove and 19 not upgraded.\n",
            "Need to get 2,473 kB of archives.\n",
            "After this operation, 2,882 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 python3-pip-whl all 22.0.2+dfsg-1ubuntu0.3 [1,679 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 python3-setuptools-whl all 59.6.0-1.2ubuntu0.22.04.1 [788 kB]\n",
            "Get:3 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 python3.10-venv amd64 3.10.12-1~22.04.2 [5,724 B]\n",
            "Fetched 2,473 kB in 1s (2,848 kB/s)\n",
            "Selecting previously unselected package python3-pip-whl.\n",
            "(Reading database ... 120874 files and directories currently installed.)\n",
            "Preparing to unpack .../python3-pip-whl_22.0.2+dfsg-1ubuntu0.3_all.deb ...\n",
            "Unpacking python3-pip-whl (22.0.2+dfsg-1ubuntu0.3) ...\n",
            "Selecting previously unselected package python3-setuptools-whl.\n",
            "Preparing to unpack .../python3-setuptools-whl_59.6.0-1.2ubuntu0.22.04.1_all.deb ...\n",
            "Unpacking python3-setuptools-whl (59.6.0-1.2ubuntu0.22.04.1) ...\n",
            "Selecting previously unselected package python3.10-venv.\n",
            "Preparing to unpack .../python3.10-venv_3.10.12-1~22.04.2_amd64.deb ...\n",
            "Unpacking python3.10-venv (3.10.12-1~22.04.2) ...\n",
            "Setting up python3-setuptools-whl (59.6.0-1.2ubuntu0.22.04.1) ...\n",
            "Setting up python3-pip-whl (22.0.2+dfsg-1ubuntu0.3) ...\n",
            "Setting up python3.10-venv (3.10.12-1~22.04.2) ...\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!source venv/bin/activate\n",
        "import sys\n",
        "print(sys.prefix)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "bfQuneVBCLW3",
        "outputId": "9baaf7d1-d6f1-41ce-f15b-9ed9b80830c5"
      },
      "execution_count": 11,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/usr\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "3oPd7cj7CvQ2",
        "outputId": "9fa8858e-2582-403b-f12d-e3d9c45d3ea2"
      },
      "execution_count": 15,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/Bio/pairwise2.py:278: BiopythonDeprecationWarning: Bio.pairwise2 has been deprecated, and we intend to remove it in a future release of Biopython. As an alternative, please consider using Bio.Align.PairwiseAligner as a replacement, and contact the Biopython developers if you still need the Bio.pairwise2 module.\n",
            "  warnings.warn(\n",
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 5, in <module>\n",
            "    from Bio.SubsMat import MatrixInfo as matlist\n",
            "ModuleNotFoundError: No module named 'Bio.SubsMat'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip uninstall biopython -y\n",
        "!pip install biopython==1.76"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "_mPaQW5zC1iv",
        "outputId": "b3fb9d51-16fa-454c-c32a-45d16e30f8c8"
      },
      "execution_count": 16,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Found existing installation: biopython 1.81\n",
            "Uninstalling biopython-1.81:\n",
            "  Successfully uninstalled biopython-1.81\n",
            "Collecting biopython==1.76\n",
            "  Downloading biopython-1.76.tar.gz (16.3 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m16.3/16.3 MB\u001b[0m \u001b[31m30.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.10/dist-packages (from biopython==1.76) (1.23.5)\n",
            "Building wheels for collected packages: biopython\n",
            "  Building wheel for biopython (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for biopython: filename=biopython-1.76-cp310-cp310-linux_x86_64.whl size=2604699 sha256=cc3e3633136fd209b9590a439b9d8b564a33ec646644d762679d5ccaac0ddae0\n",
            "  Stored in directory: /root/.cache/pip/wheels/ab/9a/f6/14f94f30a5e6772cf322b19309b6d694c136d41c70b6fed2c0\n",
            "Successfully built biopython\n",
            "Installing collected packages: biopython\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "bio 1.6.0 requires biopython>=1.80, but you have biopython 1.76 which is incompatible.\u001b[0m\u001b[31m\n",
            "\u001b[0mSuccessfully installed biopython-1.76\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "pTQoeO-6DFwH",
        "outputId": "90af27a6-0297-46dd-94a5-4cb4a2b9218a"
      },
      "execution_count": 17,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 28, in <module>\n",
            "    alignments = max(pairwise2.align.globalds(\"ATATCTCGATCGCTACGTC\", \"CTAGCTCGCTGCTCAGCATC\",\n",
            "  File \"/content/textt.py\", line 29, in <lambda>\n",
            "    matlist.blosum62, -10, -0.5), key=lambda x: x.score)\n",
            "AttributeError: 'tuple' object has no attribute 'score'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!sudo apt install python3.12 -y"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "CpttN5i_1WvT",
        "outputId": "4146994d-9071-4c5a-f240-0809e5bf059a"
      },
      "execution_count": 14,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  libpython3.12-minimal libpython3.12-stdlib mailcap mime-support\n",
            "  python3.12-minimal\n",
            "Suggested packages:\n",
            "  python3.12-venv binfmt-support\n",
            "The following NEW packages will be installed:\n",
            "  libpython3.12-minimal libpython3.12-stdlib mailcap mime-support python3.12\n",
            "  python3.12-minimal\n",
            "0 upgraded, 6 newly installed, 0 to remove and 19 not upgraded.\n",
            "Need to get 6,093 kB of archives.\n",
            "After this operation, 23.2 MB of additional disk space will be used.\n",
            "Get:1 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy/main amd64 libpython3.12-minimal amd64 3.12.0-1+jammy2 [872 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu jammy/main amd64 mailcap all 3.70+nmu1ubuntu1 [23.8 kB]\n",
            "Get:3 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy/main amd64 python3.12-minimal amd64 3.12.0-1+jammy2 [2,506 kB]\n",
            "Get:4 http://archive.ubuntu.com/ubuntu jammy/main amd64 mime-support all 3.66 [3,696 B]\n",
            "Get:5 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy/main amd64 libpython3.12-stdlib amd64 3.12.0-1+jammy2 [2,011 kB]\n",
            "Get:6 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy/main amd64 python3.12 amd64 3.12.0-1+jammy2 [676 kB]\n",
            "Fetched 6,093 kB in 1s (10.5 MB/s)\n",
            "debconf: unable to initialize frontend: Dialog\n",
            "debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 78, <> line 6.)\n",
            "debconf: falling back to frontend: Readline\n",
            "debconf: unable to initialize frontend: Readline\n",
            "debconf: (This frontend requires a controlling tty.)\n",
            "debconf: falling back to frontend: Teletype\n",
            "dpkg-preconfigure: unable to re-open stdin: \n",
            "Selecting previously unselected package libpython3.12-minimal:amd64.\n",
            "(Reading database ... 120889 files and directories currently installed.)\n",
            "Preparing to unpack .../0-libpython3.12-minimal_3.12.0-1+jammy2_amd64.deb ...\n",
            "Unpacking libpython3.12-minimal:amd64 (3.12.0-1+jammy2) ...\n",
            "Selecting previously unselected package python3.12-minimal.\n",
            "Preparing to unpack .../1-python3.12-minimal_3.12.0-1+jammy2_amd64.deb ...\n",
            "Unpacking python3.12-minimal (3.12.0-1+jammy2) ...\n",
            "Selecting previously unselected package mailcap.\n",
            "Preparing to unpack .../2-mailcap_3.70+nmu1ubuntu1_all.deb ...\n",
            "Unpacking mailcap (3.70+nmu1ubuntu1) ...\n",
            "Selecting previously unselected package mime-support.\n",
            "Preparing to unpack .../3-mime-support_3.66_all.deb ...\n",
            "Unpacking mime-support (3.66) ...\n",
            "Selecting previously unselected package libpython3.12-stdlib:amd64.\n",
            "Preparing to unpack .../4-libpython3.12-stdlib_3.12.0-1+jammy2_amd64.deb ...\n",
            "Unpacking libpython3.12-stdlib:amd64 (3.12.0-1+jammy2) ...\n",
            "Selecting previously unselected package python3.12.\n",
            "Preparing to unpack .../5-python3.12_3.12.0-1+jammy2_amd64.deb ...\n",
            "Unpacking python3.12 (3.12.0-1+jammy2) ...\n",
            "Setting up libpython3.12-minimal:amd64 (3.12.0-1+jammy2) ...\n",
            "Setting up mailcap (3.70+nmu1ubuntu1) ...\n",
            "Setting up python3.12-minimal (3.12.0-1+jammy2) ...\n",
            "Setting up mime-support (3.66) ...\n",
            "Setting up libpython3.12-stdlib:amd64 (3.12.0-1+jammy2) ...\n",
            "Setting up python3.12 (3.12.0-1+jammy2) ...\n",
            "Processing triggers for man-db (2.10.2-1) ...\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python --version"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "7V8RSFCy1jnI",
        "outputId": "f5c4c9be-dcd5-4e7e-b722-691077a1342d"
      },
      "execution_count": 18,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Python 3.10.12\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.10 1\n",
        "!sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.12 2"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "oH3v8OQF1fK7",
        "outputId": "67bc2f62-f82b-4917-c985-d4fe72d8cb9f"
      },
      "execution_count": 19,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "update-alternatives: using /usr/bin/python3.10 to provide /usr/bin/python3 (python3) in auto mode\n",
            "update-alternatives: using /usr/bin/python3.12 to provide /usr/bin/python3 (python3) in auto mode\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python --version"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "aicjCs051ky-",
        "outputId": "ec7a4795-0ea0-49f4-fd3a-5b7a21ea88b1"
      },
      "execution_count": 20,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Python 3.12.0\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "07OOeFH91o4f",
        "outputId": "f34b066f-8310-4116-a1fe-0a09e94601af"
      },
      "execution_count": 21,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 1, in <module>\n",
            "    import google\n",
            "ModuleNotFoundError: No module named 'google'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install google"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "eM3iTFMy1u11",
        "outputId": "51920c17-43d4-4e15-8e01-c08ec5b3ca93"
      },
      "execution_count": 22,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/usr/local/bin/pip3\", line 5, in <module>\n",
            "    from pip._internal.cli.main import main\n",
            "ModuleNotFoundError: No module named 'pip'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py\n",
        "!python3 get-pip.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ueiRKOOm1yMW",
        "outputId": "78ba76f4-c16b-4c6b-e102-93e23a497acd"
      },
      "execution_count": 23,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
            "                                 Dload  Upload   Total   Spent    Left  Speed\n",
            "100 2570k  100 2570k    0     0  9189k      0 --:--:-- --:--:-- --:--:-- 9213k\n",
            "Collecting pip\n",
            "  Downloading pip-23.3.1-py3-none-any.whl.metadata (3.5 kB)\n",
            "Collecting setuptools\n",
            "  Downloading setuptools-68.2.2-py3-none-any.whl.metadata (6.3 kB)\n",
            "Collecting wheel\n",
            "  Downloading wheel-0.41.2-py3-none-any.whl.metadata (2.2 kB)\n",
            "Downloading pip-23.3.1-py3-none-any.whl (2.1 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.1/2.1 MB\u001b[0m \u001b[31m14.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading setuptools-68.2.2-py3-none-any.whl (807 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m807.9/807.9 kB\u001b[0m \u001b[31m21.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading wheel-0.41.2-py3-none-any.whl (64 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m64.8/64.8 kB\u001b[0m \u001b[31m4.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: wheel, setuptools, pip\n",
            "Successfully installed pip-23.3.1 setuptools-68.2.2 wheel-0.41.2\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install --upgrade google-api-python-client"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "_BZKGJph15DP",
        "outputId": "156319ae-62f4-4171-f94e-b05195d52ab0"
      },
      "execution_count": 24,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting google-api-python-client\n",
            "  Downloading google_api_python_client-2.105.0-py2.py3-none-any.whl.metadata (6.6 kB)\n",
            "Collecting httplib2<1.dev0,>=0.15.0 (from google-api-python-client)\n",
            "  Downloading httplib2-0.22.0-py3-none-any.whl (96 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m96.9/96.9 kB\u001b[0m \u001b[31m2.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting google-auth<3.0.0.dev0,>=1.19.0 (from google-api-python-client)\n",
            "  Downloading google_auth-2.23.3-py2.py3-none-any.whl.metadata (4.2 kB)\n",
            "Collecting google-auth-httplib2>=0.1.0 (from google-api-python-client)\n",
            "  Downloading google_auth_httplib2-0.1.1-py2.py3-none-any.whl.metadata (2.1 kB)\n",
            "Collecting google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5 (from google-api-python-client)\n",
            "  Downloading google_api_core-2.12.0-py3-none-any.whl.metadata (2.7 kB)\n",
            "Collecting uritemplate<5,>=3.0.1 (from google-api-python-client)\n",
            "  Downloading uritemplate-4.1.1-py2.py3-none-any.whl (10 kB)\n",
            "Collecting googleapis-common-protos<2.0.dev0,>=1.56.2 (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client)\n",
            "  Downloading googleapis_common_protos-1.61.0-py2.py3-none-any.whl.metadata (1.5 kB)\n",
            "Collecting protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0.dev0,>=3.19.5 (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client)\n",
            "  Downloading protobuf-4.24.4-cp37-abi3-manylinux2014_x86_64.whl.metadata (540 bytes)\n",
            "Collecting requests<3.0.0.dev0,>=2.18.0 (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client)\n",
            "  Downloading requests-2.31.0-py3-none-any.whl.metadata (4.6 kB)\n",
            "Collecting cachetools<6.0,>=2.0.0 (from google-auth<3.0.0.dev0,>=1.19.0->google-api-python-client)\n",
            "  Downloading cachetools-5.3.2-py3-none-any.whl.metadata (5.2 kB)\n",
            "Collecting pyasn1-modules>=0.2.1 (from google-auth<3.0.0.dev0,>=1.19.0->google-api-python-client)\n",
            "  Downloading pyasn1_modules-0.3.0-py2.py3-none-any.whl (181 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m181.3/181.3 kB\u001b[0m \u001b[31m7.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting rsa<5,>=3.1.4 (from google-auth<3.0.0.dev0,>=1.19.0->google-api-python-client)\n",
            "  Downloading rsa-4.9-py3-none-any.whl (34 kB)\n",
            "Collecting pyparsing!=3.0.0,!=3.0.1,!=3.0.2,!=3.0.3,<4,>=2.4.2 (from httplib2<1.dev0,>=0.15.0->google-api-python-client)\n",
            "  Downloading pyparsing-3.1.1-py3-none-any.whl.metadata (5.1 kB)\n",
            "Collecting pyasn1<0.6.0,>=0.4.6 (from pyasn1-modules>=0.2.1->google-auth<3.0.0.dev0,>=1.19.0->google-api-python-client)\n",
            "  Downloading pyasn1-0.5.0-py2.py3-none-any.whl (83 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m83.9/83.9 kB\u001b[0m \u001b[31m5.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting charset-normalizer<4,>=2 (from requests<3.0.0.dev0,>=2.18.0->google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client)\n",
            "  Downloading charset_normalizer-3.3.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (33 kB)\n",
            "Collecting idna<4,>=2.5 (from requests<3.0.0.dev0,>=2.18.0->google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client)\n",
            "  Downloading idna-3.4-py3-none-any.whl (61 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m61.5/61.5 kB\u001b[0m \u001b[31m4.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting urllib3<3,>=1.21.1 (from requests<3.0.0.dev0,>=2.18.0->google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client)\n",
            "  Downloading urllib3-2.0.7-py3-none-any.whl.metadata (6.6 kB)\n",
            "Collecting certifi>=2017.4.17 (from requests<3.0.0.dev0,>=2.18.0->google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client)\n",
            "  Downloading certifi-2023.7.22-py3-none-any.whl.metadata (2.2 kB)\n",
            "Downloading google_api_python_client-2.105.0-py2.py3-none-any.whl (12.6 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m12.6/12.6 MB\u001b[0m \u001b[31m46.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading google_api_core-2.12.0-py3-none-any.whl (121 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m121.4/121.4 kB\u001b[0m \u001b[31m8.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading google_auth-2.23.3-py2.py3-none-any.whl (182 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m182.3/182.3 kB\u001b[0m \u001b[31m8.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading google_auth_httplib2-0.1.1-py2.py3-none-any.whl (9.3 kB)\n",
            "Downloading cachetools-5.3.2-py3-none-any.whl (9.3 kB)\n",
            "Downloading googleapis_common_protos-1.61.0-py2.py3-none-any.whl (230 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m230.9/230.9 kB\u001b[0m \u001b[31m13.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading protobuf-4.24.4-cp37-abi3-manylinux2014_x86_64.whl (311 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m311.6/311.6 kB\u001b[0m \u001b[31m14.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading pyparsing-3.1.1-py3-none-any.whl (103 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m103.1/103.1 kB\u001b[0m \u001b[31m7.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading requests-2.31.0-py3-none-any.whl (62 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m62.6/62.6 kB\u001b[0m \u001b[31m4.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading certifi-2023.7.22-py3-none-any.whl (158 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m158.3/158.3 kB\u001b[0m \u001b[31m11.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading charset_normalizer-3.3.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (139 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m139.1/139.1 kB\u001b[0m \u001b[31m5.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading urllib3-2.0.7-py3-none-any.whl (124 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m124.2/124.2 kB\u001b[0m \u001b[31m9.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: urllib3, uritemplate, pyparsing, pyasn1, protobuf, idna, charset-normalizer, certifi, cachetools, rsa, requests, pyasn1-modules, httplib2, googleapis-common-protos, google-auth, google-auth-httplib2, google-api-core, google-api-python-client\n",
            "Successfully installed cachetools-5.3.2 certifi-2023.7.22 charset-normalizer-3.3.1 google-api-core-2.12.0 google-api-python-client-2.105.0 google-auth-2.23.3 google-auth-httplib2-0.1.1 googleapis-common-protos-1.61.0 httplib2-0.22.0 idna-3.4 protobuf-4.24.4 pyasn1-0.5.0 pyasn1-modules-0.3.0 pyparsing-3.1.1 requests-2.31.0 rsa-4.9 uritemplate-4.1.1 urllib3-2.0.7\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.colab-display-data+json": {
              "pip_warning": {
                "packages": [
                  "certifi",
                  "google",
                  "httplib2"
                ]
              }
            }
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "hwjfsyx72Did",
        "outputId": "4085960a-b459-4c56-e4d0-de5f1a9e8db9"
      },
      "execution_count": 25,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 3, in <module>\n",
            "    from typing_extensions import override\n",
            "ModuleNotFoundError: No module named 'typing_extensions'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install typing-extensions"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "HhQktVzUDjJt",
        "outputId": "f00c664c-08c1-40cb-e65f-0e9833c3aed1"
      },
      "execution_count": 26,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting typing-extensions\n",
            "  Downloading typing_extensions-4.8.0-py3-none-any.whl.metadata (3.0 kB)\n",
            "Downloading typing_extensions-4.8.0-py3-none-any.whl (31 kB)\n",
            "Installing collected packages: typing-extensions\n",
            "Successfully installed typing-extensions-4.8.0\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "DprIbSLhDmUF",
        "outputId": "e309dc0f-342f-45a3-8894-8ee250266523"
      },
      "execution_count": 27,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 4, in <module>\n",
            "    from Bio import pairwise2\n",
            "ModuleNotFoundError: No module named 'Bio'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip freeze --path ~/usr/local/lib/python3.10/dist-packagess > requirements.txt\n",
        "!pip install --user -r requirements.txt"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "fO5di7-SDt5o",
        "outputId": "3c32d18e-1c6e-4ed4-c6fa-7af79be43c1b"
      },
      "execution_count": 28,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip freeze --path ~/usr/local/lib/python3.10/dist-packagess > requirements.txt"
      ],
      "metadata": {
        "id": "SpHTrrSnEw5D"
      },
      "execution_count": 41,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install -r requirements.txt"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "QiXwJZnAD-iM",
        "outputId": "da4715d1-1199-4729-d742-d93568a9cde7"
      },
      "execution_count": 32,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VePgYRenDxCt",
        "outputId": "016f7777-cf2c-49b3-a9de-aa20be1c7997"
      },
      "execution_count": 33,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 4, in <module>\n",
            "    from Bio import pairwise2\n",
            "ModuleNotFoundError: No module named 'Bio'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install Bio\n",
        "!pip uninstall biopython -y\n",
        "!pip install biopython==1.76"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "WJjRuEQK26vx",
        "outputId": "ee85d35c-6b5d-4947-b6a7-eaa6173e61f6"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting Bio\n",
            "  Downloading bio-1.6.0-py3-none-any.whl.metadata (5.1 kB)\n",
            "Collecting biopython>=1.80 (from Bio)\n",
            "  Downloading biopython-1.81-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (13 kB)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.12/dist-packages (from Bio) (2.31.0)\n",
            "Collecting tqdm (from Bio)\n",
            "  Downloading tqdm-4.66.1-py3-none-any.whl.metadata (57 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m57.6/57.6 kB\u001b[0m \u001b[31m2.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting mygene (from Bio)\n",
            "  Downloading mygene-3.2.2-py2.py3-none-any.whl (5.4 kB)\n",
            "Collecting pandas (from Bio)\n",
            "  Downloading pandas-2.1.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (18 kB)\n",
            "Collecting pooch (from Bio)\n",
            "  Downloading pooch-1.8.0-py3-none-any.whl.metadata (9.9 kB)\n",
            "Collecting gprofiler-official (from Bio)\n",
            "  Downloading gprofiler_official-1.0.0-py3-none-any.whl (9.3 kB)\n",
            "Collecting numpy (from biopython>=1.80->Bio)\n",
            "  Downloading numpy-1.26.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (61 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m61.2/61.2 kB\u001b[0m \u001b[31m5.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting biothings-client>=0.2.6 (from mygene->Bio)\n",
            "  Downloading biothings_client-0.3.0-py2.py3-none-any.whl (29 kB)\n",
            "Collecting python-dateutil>=2.8.2 (from pandas->Bio)\n",
            "  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m247.7/247.7 kB\u001b[0m \u001b[31m12.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting pytz>=2020.1 (from pandas->Bio)\n",
            "  Downloading pytz-2023.3.post1-py2.py3-none-any.whl.metadata (22 kB)\n",
            "Collecting tzdata>=2022.1 (from pandas->Bio)\n",
            "  Downloading tzdata-2023.3-py2.py3-none-any.whl (341 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m341.8/341.8 kB\u001b[0m \u001b[31m27.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting platformdirs>=2.5.0 (from pooch->Bio)\n",
            "  Downloading platformdirs-3.11.0-py3-none-any.whl.metadata (11 kB)\n",
            "Collecting packaging>=20.0 (from pooch->Bio)\n",
            "  Downloading packaging-23.2-py3-none-any.whl.metadata (3.2 kB)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (3.3.1)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (3.4)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (2023.7.22)\n",
            "Requirement already satisfied: six>=1.5 in /usr/lib/python3/dist-packages (from python-dateutil>=2.8.2->pandas->Bio) (1.16.0)\n",
            "Downloading bio-1.6.0-py3-none-any.whl (279 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m279.4/279.4 kB\u001b[0m \u001b[31m23.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading biopython-1.81-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.1/3.1 MB\u001b[0m \u001b[31m97.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading pandas-2.1.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (11.7 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m11.7/11.7 MB\u001b[0m \u001b[31m152.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading pooch-1.8.0-py3-none-any.whl (62 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m62.7/62.7 kB\u001b[0m \u001b[31m5.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading tqdm-4.66.1-py3-none-any.whl (78 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m78.3/78.3 kB\u001b[0m \u001b[31m7.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading numpy-1.26.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.9 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m17.9/17.9 MB\u001b[0m \u001b[31m126.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading packaging-23.2-py3-none-any.whl (53 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m53.0/53.0 kB\u001b[0m \u001b[31m3.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading platformdirs-3.11.0-py3-none-any.whl (17 kB)\n",
            "Downloading pytz-2023.3.post1-py2.py3-none-any.whl (502 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m502.5/502.5 kB\u001b[0m \u001b[31m36.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: pytz, tzdata, tqdm, python-dateutil, platformdirs, packaging, numpy, pooch, pandas, gprofiler-official, biothings-client, biopython, mygene, Bio\n",
            "Successfully installed Bio-1.6.0 biopython-1.81 biothings-client-0.3.0 gprofiler-official-1.0.0 mygene-3.2.2 numpy-1.26.1 packaging-23.2 pandas-2.1.1 platformdirs-3.11.0 pooch-1.8.0 python-dateutil-2.8.2 pytz-2023.3.post1 tqdm-4.66.1 tzdata-2023.3\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.colab-display-data+json": {
              "pip_warning": {
                "packages": [
                  "dateutil"
                ]
              }
            }
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Found existing installation: biopython 1.81\n",
            "Uninstalling biopython-1.81:\n",
            "  Successfully uninstalled biopython-1.81\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0mCollecting biopython==1.76\n",
            "  Downloading biopython-1.76.tar.gz (16.3 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m16.3/16.3 MB\u001b[0m \u001b[31m64.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.12/dist-packages (from biopython==1.76) (1.26.1)\n",
            "Building wheels for collected packages: biopython\n",
            "  \u001b[1;31merror\u001b[0m: \u001b[1msubprocess-exited-with-error\u001b[0m\n",
            "  \n",
            "  \u001b[31m×\u001b[0m \u001b[32mpython setup.py bdist_wheel\u001b[0m did not run successfully.\n",
            "  \u001b[31m│\u001b[0m exit code: \u001b[1;36m1\u001b[0m\n",
            "  \u001b[31m╰─>\u001b[0m See above for output.\n",
            "  \n",
            "  \u001b[1;35mnote\u001b[0m: This error originates from a subprocess, and is likely not a problem with pip.\n",
            "  Building wheel for biopython (setup.py) ... \u001b[?25lerror\n",
            "\u001b[31m  ERROR: Failed building wheel for biopython\u001b[0m\u001b[31m\n",
            "\u001b[0m\u001b[?25h  Running setup.py clean for biopython\n",
            "Failed to build biopython\n",
            "\u001b[31mERROR: Could not build wheels for biopython, which is required to install pyproject.toml-based projects\u001b[0m\u001b[31m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip uninstall biopython -y\n",
        "!pip install biopython==1.76"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "srifpt5wET4N",
        "outputId": "90153c3c-83d6-4e48-cc15-085c391ba743"
      },
      "execution_count": 36,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Found existing installation: biopython 1.81\n",
            "Uninstalling biopython-1.81:\n",
            "  Successfully uninstalled biopython-1.81\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0mCollecting biopython==1.76\n",
            "  Downloading biopython-1.76.tar.gz (16.3 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m16.3/16.3 MB\u001b[0m \u001b[31m61.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.12/dist-packages (from biopython==1.76) (1.26.1)\n",
            "Building wheels for collected packages: biopython\n",
            "  \u001b[1;31merror\u001b[0m: \u001b[1msubprocess-exited-with-error\u001b[0m\n",
            "  \n",
            "  \u001b[31m×\u001b[0m \u001b[32mpython setup.py bdist_wheel\u001b[0m did not run successfully.\n",
            "  \u001b[31m│\u001b[0m exit code: \u001b[1;36m1\u001b[0m\n",
            "  \u001b[31m╰─>\u001b[0m See above for output.\n",
            "  \n",
            "  \u001b[1;35mnote\u001b[0m: This error originates from a subprocess, and is likely not a problem with pip.\n",
            "  Building wheel for biopython (setup.py) ... \u001b[?25lerror\n",
            "\u001b[31m  ERROR: Failed building wheel for biopython\u001b[0m\u001b[31m\n",
            "\u001b[0m\u001b[?25h  Running setup.py clean for biopython\n",
            "Failed to build biopython\n",
            "\u001b[31mERROR: Could not build wheels for biopython, which is required to install pyproject.toml-based projects\u001b[0m\u001b[31m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "tMTv-PdLERUV",
        "outputId": "99b1da9e-7380-4edd-e399-94688ce381d3"
      },
      "execution_count": 39,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/usr/local/lib/python3.12/dist-packages/Bio/pairwise2.py:278: BiopythonDeprecationWarning: Bio.pairwise2 has been deprecated, and we intend to remove it in a future release of Biopython. As an alternative, please consider using Bio.Align.PairwiseAligner as a replacement, and contact the Biopython developers if you still need the Bio.pairwise2 module.\n",
            "  warnings.warn(\n",
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 5, in <module>\n",
            "    from Bio.SubsMat import MatrixInfo as matlist\n",
            "ModuleNotFoundError: No module named 'Bio.SubsMat'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install numpy"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "xprr64RW3MLR",
        "outputId": "81f629ed-2017-47b1-cb74-1bd999715c52"
      },
      "execution_count": 42,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: numpy in /usr/local/lib/python3.12/dist-packages (1.26.1)\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install Cmake"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "KmWau2cGFraU",
        "outputId": "a0adfadc-60f7-4b49-b834-63848bd86a6c"
      },
      "execution_count": 45,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting Cmake\n",
            "  Downloading cmake-3.27.7-py2.py3-none-manylinux2014_x86_64.manylinux_2_17_x86_64.whl.metadata (6.7 kB)\n",
            "Downloading cmake-3.27.7-py2.py3-none-manylinux2014_x86_64.manylinux_2_17_x86_64.whl (26.0 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m26.0/26.0 MB\u001b[0m \u001b[31m78.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: Cmake\n",
            "Successfully installed Cmake-3.27.7\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install Bio\n",
        "!pip uninstall biopython -y\n",
        "!pip install biopython==1.76"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "bTAZAwV_FxjT",
        "outputId": "25d4ef05-7fbf-44c8-8b40-53bdc9a04a67"
      },
      "execution_count": 46,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: Bio in /usr/local/lib/python3.12/dist-packages (1.6.0)\n",
            "Collecting biopython>=1.80 (from Bio)\n",
            "  Using cached biopython-1.81-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (13 kB)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.12/dist-packages (from Bio) (2.31.0)\n",
            "Requirement already satisfied: tqdm in /usr/local/lib/python3.12/dist-packages (from Bio) (4.66.1)\n",
            "Requirement already satisfied: mygene in /usr/local/lib/python3.12/dist-packages (from Bio) (3.2.2)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.12/dist-packages (from Bio) (2.1.2)\n",
            "Requirement already satisfied: pooch in /usr/local/lib/python3.12/dist-packages (from Bio) (1.8.0)\n",
            "Requirement already satisfied: gprofiler-official in /usr/local/lib/python3.12/dist-packages (from Bio) (1.0.0)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.12/dist-packages (from biopython>=1.80->Bio) (1.26.1)\n",
            "Requirement already satisfied: biothings-client>=0.2.6 in /usr/local/lib/python3.12/dist-packages (from mygene->Bio) (0.3.0)\n",
            "Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.12/dist-packages (from pandas->Bio) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.12/dist-packages (from pandas->Bio) (2023.3.post1)\n",
            "Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.12/dist-packages (from pandas->Bio) (2023.3)\n",
            "Requirement already satisfied: platformdirs>=2.5.0 in /usr/local/lib/python3.12/dist-packages (from pooch->Bio) (3.11.0)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.12/dist-packages (from pooch->Bio) (23.2)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (3.3.1)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (3.4)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (2023.7.22)\n",
            "Requirement already satisfied: six>=1.5 in /usr/lib/python3/dist-packages (from python-dateutil>=2.8.2->pandas->Bio) (1.16.0)\n",
            "Using cached biopython-1.81-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)\n",
            "Installing collected packages: biopython\n",
            "Successfully installed biopython-1.81\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0mFound existing installation: biopython 1.81\n",
            "Uninstalling biopython-1.81:\n",
            "  Successfully uninstalled biopython-1.81\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0mCollecting biopython==1.76\n",
            "  Using cached biopython-1.76.tar.gz (16.3 MB)\n",
            "  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.12/dist-packages (from biopython==1.76) (1.26.1)\n",
            "Building wheels for collected packages: biopython\n",
            "  \u001b[1;31merror\u001b[0m: \u001b[1msubprocess-exited-with-error\u001b[0m\n",
            "  \n",
            "  \u001b[31m×\u001b[0m \u001b[32mpython setup.py bdist_wheel\u001b[0m did not run successfully.\n",
            "  \u001b[31m│\u001b[0m exit code: \u001b[1;36m1\u001b[0m\n",
            "  \u001b[31m╰─>\u001b[0m See above for output.\n",
            "  \n",
            "  \u001b[1;35mnote\u001b[0m: This error originates from a subprocess, and is likely not a problem with pip.\n",
            "  Building wheel for biopython (setup.py) ... \u001b[?25lerror\n",
            "\u001b[31m  ERROR: Failed building wheel for biopython\u001b[0m\u001b[31m\n",
            "\u001b[0m\u001b[?25h  Running setup.py clean for biopython\n",
            "Failed to build biopython\n",
            "\u001b[31mERROR: Could not build wheels for biopython, which is required to install pyproject.toml-based projects\u001b[0m\u001b[31m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install wheel"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Y9DYg25eG-nx",
        "outputId": "85fc1701-c327-4a58-ef01-3f9567a3477a"
      },
      "execution_count": 47,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: wheel in /usr/local/lib/python3.12/dist-packages (0.41.2)\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install Bio\n",
        "!pip uninstall biopython -y\n",
        "!pip install numpy"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "KCS4G7sKHEvB",
        "outputId": "a2005b31-a874-4635-dccb-292db1199e48"
      },
      "execution_count": 57,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: Bio in /usr/local/lib/python3.12/dist-packages (1.6.0)\n",
            "Requirement already satisfied: biopython>=1.80 in /usr/local/lib/python3.12/dist-packages (from Bio) (1.81)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.12/dist-packages (from Bio) (2.31.0)\n",
            "Requirement already satisfied: tqdm in /usr/local/lib/python3.12/dist-packages (from Bio) (4.66.1)\n",
            "Requirement already satisfied: mygene in /usr/local/lib/python3.12/dist-packages (from Bio) (3.2.2)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.12/dist-packages (from Bio) (2.1.2)\n",
            "Requirement already satisfied: pooch in /usr/local/lib/python3.12/dist-packages (from Bio) (1.8.0)\n",
            "Requirement already satisfied: gprofiler-official in /usr/local/lib/python3.12/dist-packages (from Bio) (1.0.0)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.12/dist-packages (from biopython>=1.80->Bio) (1.26.1)\n",
            "Requirement already satisfied: biothings-client>=0.2.6 in /usr/local/lib/python3.12/dist-packages (from mygene->Bio) (0.3.0)\n",
            "Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.12/dist-packages (from pandas->Bio) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.12/dist-packages (from pandas->Bio) (2023.3.post1)\n",
            "Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.12/dist-packages (from pandas->Bio) (2023.3)\n",
            "Requirement already satisfied: platformdirs>=2.5.0 in /usr/local/lib/python3.12/dist-packages (from pooch->Bio) (3.11.0)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.12/dist-packages (from pooch->Bio) (23.2)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (3.3.1)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (3.4)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (2023.7.22)\n",
            "Requirement already satisfied: six>=1.5 in /usr/lib/python3/dist-packages (from python-dateutil>=2.8.2->pandas->Bio) (1.16.0)\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0mFound existing installation: biopython 1.81\n",
            "Uninstalling biopython-1.81:\n",
            "  Successfully uninstalled biopython-1.81\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0mRequirement already satisfied: numpy in /usr/local/lib/python3.12/dist-packages (1.26.1)\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m/bin/bash: line 1: py: command not found\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!tar -xvzf biopython-1.76.tar.gz"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "wxGSTzh0I5eQ",
        "outputId": "b2c1d942-0a84-491f-f20f-3f8d4fe0c028"
      },
      "execution_count": 60,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "biopython-1.76/\n",
            "biopython-1.76/Bio/\n",
            "biopython-1.76/Bio/Affy/\n",
            "biopython-1.76/Bio/Affy/CelFile.py\n",
            "biopython-1.76/Bio/Affy/__init__.py\n",
            "biopython-1.76/Bio/Align/\n",
            "biopython-1.76/Bio/Align/AlignInfo.py\n",
            "biopython-1.76/Bio/Align/Applications/\n",
            "biopython-1.76/Bio/Align/Applications/_ClustalOmega.py\n",
            "biopython-1.76/Bio/Align/Applications/_Clustalw.py\n",
            "biopython-1.76/Bio/Align/Applications/_Dialign.py\n",
            "biopython-1.76/Bio/Align/Applications/_MSAProbs.py\n",
            "biopython-1.76/Bio/Align/Applications/_Mafft.py\n",
            "biopython-1.76/Bio/Align/Applications/_Muscle.py\n",
            "biopython-1.76/Bio/Align/Applications/_Prank.py\n",
            "biopython-1.76/Bio/Align/Applications/_Probcons.py\n",
            "biopython-1.76/Bio/Align/Applications/_TCoffee.py\n",
            "biopython-1.76/Bio/Align/Applications/__init__.py\n",
            "biopython-1.76/Bio/Align/__init__.py\n",
            "biopython-1.76/Bio/Align/_aligners.c\n",
            "biopython-1.76/Bio/Align/substitution_matrices/\n",
            "biopython-1.76/Bio/Align/substitution_matrices/__init__.py\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/BENNER22\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/BENNER6\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/BENNER74\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/BLOSUM45\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/BLOSUM50\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/BLOSUM62\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/BLOSUM80\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/BLOSUM90\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/DAYHOFF\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/FENG\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/GENETIC\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/GONNET1992\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/JOHNSON\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/JONES\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/LEVIN\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/MCLACHLAN\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/MDM78\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/NUC.4.4\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/PAM250\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/PAM30\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/PAM70\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/RAO\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/RISLER\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/SCHNEIDER\n",
            "biopython-1.76/Bio/Align/substitution_matrices/data/STR\n",
            "biopython-1.76/Bio/AlignIO/\n",
            "biopython-1.76/Bio/AlignIO/ClustalIO.py\n",
            "biopython-1.76/Bio/AlignIO/EmbossIO.py\n",
            "biopython-1.76/Bio/AlignIO/FastaIO.py\n",
            "biopython-1.76/Bio/AlignIO/Interfaces.py\n",
            "biopython-1.76/Bio/AlignIO/MafIO.py\n",
            "biopython-1.76/Bio/AlignIO/MauveIO.py\n",
            "biopython-1.76/Bio/AlignIO/MsfIO.py\n",
            "biopython-1.76/Bio/AlignIO/NexusIO.py\n",
            "biopython-1.76/Bio/AlignIO/PhylipIO.py\n",
            "biopython-1.76/Bio/AlignIO/StockholmIO.py\n",
            "biopython-1.76/Bio/AlignIO/__init__.py\n",
            "biopython-1.76/Bio/Alphabet/\n",
            "biopython-1.76/Bio/Alphabet/IUPAC.py\n",
            "biopython-1.76/Bio/Alphabet/Reduced.py\n",
            "biopython-1.76/Bio/Alphabet/__init__.py\n",
            "biopython-1.76/Bio/Application/\n",
            "biopython-1.76/Bio/Application/__init__.py\n",
            "biopython-1.76/Bio/Blast/\n",
            "biopython-1.76/Bio/Blast/Applications.py\n",
            "biopython-1.76/Bio/Blast/NCBIWWW.py\n",
            "biopython-1.76/Bio/Blast/NCBIXML.py\n",
            "biopython-1.76/Bio/Blast/ParseBlastTable.py\n",
            "biopython-1.76/Bio/Blast/Record.py\n",
            "biopython-1.76/Bio/Blast/__init__.py\n",
            "biopython-1.76/Bio/CAPS/\n",
            "biopython-1.76/Bio/CAPS/__init__.py\n",
            "biopython-1.76/Bio/Cluster/\n",
            "biopython-1.76/Bio/Cluster/__init__.py\n",
            "biopython-1.76/Bio/Cluster/cluster.c\n",
            "biopython-1.76/Bio/Cluster/cluster.h\n",
            "biopython-1.76/Bio/Cluster/clustermodule.c\n",
            "biopython-1.76/Bio/Compass/\n",
            "biopython-1.76/Bio/Compass/__init__.py\n",
            "biopython-1.76/Bio/Crystal/\n",
            "biopython-1.76/Bio/Crystal/__init__.py\n",
            "biopython-1.76/Bio/Data/\n",
            "biopython-1.76/Bio/Data/CodonTable.py\n",
            "biopython-1.76/Bio/Data/IUPACData.py\n",
            "biopython-1.76/Bio/Data/SCOPData.py\n",
            "biopython-1.76/Bio/Data/__init__.py\n",
            "biopython-1.76/Bio/Emboss/\n",
            "biopython-1.76/Bio/Emboss/Applications.py\n",
            "biopython-1.76/Bio/Emboss/Primer3.py\n",
            "biopython-1.76/Bio/Emboss/PrimerSearch.py\n",
            "biopython-1.76/Bio/Emboss/__init__.py\n",
            "biopython-1.76/Bio/Entrez/\n",
            "biopython-1.76/Bio/Entrez/DTDs/\n",
            "biopython-1.76/Bio/Entrez/DTDs/Docsum_3_0.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/Docsum_3_0.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/Docsum_3_1.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/Docsum_3_1.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/Docsum_3_2.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/Docsum_3_2.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/Docsum_3_3.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/Docsum_3_3.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/Docsum_3_4.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/Docsum_3_4.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/EMBL_General.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/EMBL_General.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/GenBank_General.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/GenBank_General.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/HomoloGene.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/HomoloGene.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/INSD_INSDSeq.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/INSD_INSDSeq.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/MMDB.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/MMDB.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/MMDB_Chemical_graph.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/MMDB_Chemical_graph.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/MMDB_Features.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/MMDB_Features.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/MMDB_Structural_model.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/MMDB_Structural_model.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Access.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Access.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Biblio.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Biblio.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_BioSource.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_BioSource.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_BioTree.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_BioTree.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Blast4.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Blast4.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_BlastDL.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_BlastDL.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_BlastOutput.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_BlastOutput.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Cdd.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Cdd.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Cn3d.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Cn3d.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Entity.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Entrez2.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Entrez2.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Entrezgene.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Entrezgene.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_FeatDef.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_FeatDef.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_GBSeq.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_GBSeq.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Gene.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Gene.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_General.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_General.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_ID1Access.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_ID1Access.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_ID2Access.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_ID2Access.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_MedArchive.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_MedArchive.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Medlars.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Medlars.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Medline.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Medline.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Mim.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Mim.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Mime.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Mime.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_ObjPrt.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_ObjPrt.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Organism.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Organism.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_PCAssay.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_PCAssay.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_PCSubstance.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_PCSubstance.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Project.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Project.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Protein.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Protein.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Pub.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Pub.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_PubMed.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_PubMed.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_RNA.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_RNA.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Remap.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Remap.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Rsite.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Rsite.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_ScoreMat.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_ScoreMat.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_SeqCode.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_SeqCode.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_SeqTable.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_SeqTable.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Seq_split.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Seq_split.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Seqalign.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Seqalign.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Seqfeat.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Seqfeat.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Seqloc.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Seqloc.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Seqres.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Seqres.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Seqset.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Seqset.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Sequence.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Sequence.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Submit.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Submit.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Systems.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_TSeq.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_TSeq.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_TxInit.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_TxInit.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Variation.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_Variation.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NCBI_all.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NSE.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/NSE.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/OMSSA.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/OMSSA.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/PDB_General.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/PDB_General.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/PIR_General.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/PIR_General.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/PRF_General.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/PRF_General.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/SP_General.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/SP_General.mod.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/XHTMLtablesetup.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/archivearticle.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/archivecustom-classes.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/archivecustom-mixes.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/archivecustom-models.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/archivecustom-modules.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/articlemeta.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/backmatter.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/bookdoc_100301.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/bookdoc_110101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/bookdoc_120101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/bookdoc_130101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/bookdoc_140101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/bookdoc_150101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/chars.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/common.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/default-classes.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/default-mixes.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/display.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/eInfo_020511.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/eLink_090910.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/eLink_101123.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/ePost_020511.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/eSearch_020511.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/eSpell.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/eSummary_041029.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/egquery.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/einfo.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/elink_020122.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/esearch.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/esummary-v1.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/format.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/htmltable.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/isoamsa.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isoamsb.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isoamsc.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isoamsn.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isoamso.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isoamsr.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isobox.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isocyr1.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isocyr2.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isodia.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isogrk1.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isogrk2.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isogrk3.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isogrk4.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isolat1.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isolat2.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isomfrk.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isomopf.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isomscr.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isonum.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isopub.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/isotech.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/journalmeta.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/link.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/list.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/math.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/mathml-in-pubmed.mod\n",
            "biopython-1.76/Bio/Entrez/DTDs/mathml2-qname-1.mod\n",
            "biopython-1.76/Bio/Entrez/DTDs/mathml2.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/mathml3-qname1.mod\n",
            "biopython-1.76/Bio/Entrez/DTDs/mathml3.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/mathmlsetup.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/mmlalias.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/mmlextra.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/modules.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlm-articleset-2.0.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmcatalogrecordset_170601.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmcommon_011101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmcommon_080101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmcommon_090101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedline_011101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedline_080101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedline_090101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedlinecitation_011101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedlinecitation_080101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedlinecitation_090101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedlinecitationset_100101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedlinecitationset_100301.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedlinecitationset_110101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedlinecitationset_120101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedlinecitationset_130101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedlinecitationset_130501.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedlinecitationset_140101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmmedlinecitationset_150101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmserials_080101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmserials_100101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmsharedcatcit_080101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/nlmsharedcatcit_090101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/notat.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/para.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/phrase.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/pmc-1.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_020114.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_080101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_090101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_100101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_100301.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_110101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_120101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_130101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_130501.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_140101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_150101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_180101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_180601.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/pubmed_190101.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/references.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/section.ent\n",
            "biopython-1.76/Bio/Entrez/DTDs/taxon.dtd\n",
            "biopython-1.76/Bio/Entrez/DTDs/xmlspecchars.ent\n",
            "biopython-1.76/Bio/Entrez/Parser.py\n",
            "biopython-1.76/Bio/Entrez/XSDs/\n",
            "biopython-1.76/Bio/Entrez/XSDs/IPGReportSet.xsd\n",
            "biopython-1.76/Bio/Entrez/__init__.py\n",
            "biopython-1.76/Bio/ExPASy/\n",
            "biopython-1.76/Bio/ExPASy/Enzyme.py\n",
            "biopython-1.76/Bio/ExPASy/Prodoc.py\n",
            "biopython-1.76/Bio/ExPASy/Prosite.py\n",
            "biopython-1.76/Bio/ExPASy/ScanProsite.py\n",
            "biopython-1.76/Bio/ExPASy/__init__.py\n",
            "biopython-1.76/Bio/ExPASy/cellosaurus.py\n",
            "biopython-1.76/Bio/FSSP/\n",
            "biopython-1.76/Bio/FSSP/FSSPTools.py\n",
            "biopython-1.76/Bio/FSSP/__init__.py\n",
            "biopython-1.76/Bio/FSSP/fssp_rec.py\n",
            "biopython-1.76/Bio/File.py\n",
            "biopython-1.76/Bio/GenBank/\n",
            "biopython-1.76/Bio/GenBank/Record.py\n",
            "biopython-1.76/Bio/GenBank/Scanner.py\n",
            "biopython-1.76/Bio/GenBank/__init__.py\n",
            "biopython-1.76/Bio/GenBank/utils.py\n",
            "biopython-1.76/Bio/Geo/\n",
            "biopython-1.76/Bio/Geo/Record.py\n",
            "biopython-1.76/Bio/Geo/__init__.py\n",
            "biopython-1.76/Bio/Graphics/\n",
            "biopython-1.76/Bio/Graphics/BasicChromosome.py\n",
            "biopython-1.76/Bio/Graphics/ColorSpiral.py\n",
            "biopython-1.76/Bio/Graphics/Comparative.py\n",
            "biopython-1.76/Bio/Graphics/DisplayRepresentation.py\n",
            "biopython-1.76/Bio/Graphics/Distribution.py\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/_AbstractDrawer.py\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/_CircularDrawer.py\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/_Colors.py\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/_CrossLink.py\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/_Diagram.py\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/_Feature.py\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/_FeatureSet.py\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/_Graph.py\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/_GraphSet.py\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/_LinearDrawer.py\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/_Track.py\n",
            "biopython-1.76/Bio/Graphics/GenomeDiagram/__init__.py\n",
            "biopython-1.76/Bio/Graphics/KGML_vis.py\n",
            "biopython-1.76/Bio/Graphics/__init__.py\n",
            "biopython-1.76/Bio/HMM/\n",
            "biopython-1.76/Bio/HMM/DynamicProgramming.py\n",
            "biopython-1.76/Bio/HMM/MarkovModel.py\n",
            "biopython-1.76/Bio/HMM/Trainer.py\n",
            "biopython-1.76/Bio/HMM/Utilities.py\n",
            "biopython-1.76/Bio/HMM/__init__.py\n",
            "biopython-1.76/Bio/Index.py\n",
            "biopython-1.76/Bio/KDTree/\n",
            "biopython-1.76/Bio/KDTree/KDTree.c\n",
            "biopython-1.76/Bio/KDTree/KDTree.h\n",
            "biopython-1.76/Bio/KDTree/KDTree.py\n",
            "biopython-1.76/Bio/KDTree/KDTreemodule.c\n",
            "biopython-1.76/Bio/KDTree/Neighbor.h\n",
            "biopython-1.76/Bio/KDTree/__init__.py\n",
            "biopython-1.76/Bio/KEGG/\n",
            "biopython-1.76/Bio/KEGG/Compound/\n",
            "biopython-1.76/Bio/KEGG/Compound/__init__.py\n",
            "biopython-1.76/Bio/KEGG/Enzyme/\n",
            "biopython-1.76/Bio/KEGG/Enzyme/__init__.py\n",
            "biopython-1.76/Bio/KEGG/Gene/\n",
            "biopython-1.76/Bio/KEGG/Gene/__init__.py\n",
            "biopython-1.76/Bio/KEGG/KGML/\n",
            "biopython-1.76/Bio/KEGG/KGML/KGML_parser.py\n",
            "biopython-1.76/Bio/KEGG/KGML/KGML_pathway.py\n",
            "biopython-1.76/Bio/KEGG/KGML/__init__.py\n",
            "biopython-1.76/Bio/KEGG/Map/\n",
            "biopython-1.76/Bio/KEGG/Map/__init__.py\n",
            "biopython-1.76/Bio/KEGG/REST.py\n",
            "biopython-1.76/Bio/KEGG/__init__.py\n",
            "biopython-1.76/Bio/LogisticRegression.py\n",
            "biopython-1.76/Bio/MarkovModel.py\n",
            "biopython-1.76/Bio/MaxEntropy.py\n",
            "biopython-1.76/Bio/Medline/\n",
            "biopython-1.76/Bio/Medline/__init__.py\n",
            "biopython-1.76/Bio/NMR/\n",
            "biopython-1.76/Bio/NMR/NOEtools.py\n",
            "biopython-1.76/Bio/NMR/__init__.py\n",
            "biopython-1.76/Bio/NMR/xpktools.py\n",
            "biopython-1.76/Bio/NaiveBayes.py\n",
            "biopython-1.76/Bio/Nexus/\n",
            "biopython-1.76/Bio/Nexus/Nexus.py\n",
            "biopython-1.76/Bio/Nexus/Nodes.py\n",
            "biopython-1.76/Bio/Nexus/StandardData.py\n",
            "biopython-1.76/Bio/Nexus/Trees.py\n",
            "biopython-1.76/Bio/Nexus/__init__.py\n",
            "biopython-1.76/Bio/Nexus/cnexus.c\n",
            "biopython-1.76/Bio/PDB/\n",
            "biopython-1.76/Bio/PDB/AbstractPropertyMap.py\n",
            "biopython-1.76/Bio/PDB/Atom.py\n",
            "biopython-1.76/Bio/PDB/Chain.py\n",
            "biopython-1.76/Bio/PDB/DSSP.py\n",
            "biopython-1.76/Bio/PDB/Dice.py\n",
            "biopython-1.76/Bio/PDB/Entity.py\n",
            "biopython-1.76/Bio/PDB/FragmentMapper.py\n",
            "biopython-1.76/Bio/PDB/HSExposure.py\n",
            "biopython-1.76/Bio/PDB/MMCIF2Dict.py\n",
            "biopython-1.76/Bio/PDB/MMCIFParser.py\n",
            "biopython-1.76/Bio/PDB/Model.py\n",
            "biopython-1.76/Bio/PDB/NACCESS.py\n",
            "biopython-1.76/Bio/PDB/NeighborSearch.py\n",
            "biopython-1.76/Bio/PDB/PDBExceptions.py\n",
            "biopython-1.76/Bio/PDB/PDBIO.py\n",
            "biopython-1.76/Bio/PDB/PDBList.py\n",
            "biopython-1.76/Bio/PDB/PDBParser.py\n",
            "biopython-1.76/Bio/PDB/PSEA.py\n",
            "biopython-1.76/Bio/PDB/Polypeptide.py\n",
            "biopython-1.76/Bio/PDB/QCPSuperimposer/\n",
            "biopython-1.76/Bio/PDB/QCPSuperimposer/__init__.py\n",
            "biopython-1.76/Bio/PDB/QCPSuperimposer/qcprotmodule.c\n",
            "biopython-1.76/Bio/PDB/Residue.py\n",
            "biopython-1.76/Bio/PDB/ResidueDepth.py\n",
            "biopython-1.76/Bio/PDB/Selection.py\n",
            "biopython-1.76/Bio/PDB/Structure.py\n",
            "biopython-1.76/Bio/PDB/StructureAlignment.py\n",
            "biopython-1.76/Bio/PDB/StructureBuilder.py\n",
            "biopython-1.76/Bio/PDB/Superimposer.py\n",
            "biopython-1.76/Bio/PDB/__init__.py\n",
            "biopython-1.76/Bio/PDB/kdtrees.c\n",
            "biopython-1.76/Bio/PDB/mmcifio.py\n",
            "biopython-1.76/Bio/PDB/mmtf/\n",
            "biopython-1.76/Bio/PDB/mmtf/DefaultParser.py\n",
            "biopython-1.76/Bio/PDB/mmtf/__init__.py\n",
            "biopython-1.76/Bio/PDB/mmtf/mmtfio.py\n",
            "biopython-1.76/Bio/PDB/parse_pdb_header.py\n",
            "biopython-1.76/Bio/PDB/vectors.py\n",
            "biopython-1.76/Bio/Pathway/\n",
            "biopython-1.76/Bio/Pathway/Rep/\n",
            "biopython-1.76/Bio/Pathway/Rep/Graph.py\n",
            "biopython-1.76/Bio/Pathway/Rep/MultiGraph.py\n",
            "biopython-1.76/Bio/Pathway/Rep/__init__.py\n",
            "biopython-1.76/Bio/Pathway/__init__.py\n",
            "biopython-1.76/Bio/Phylo/\n",
            "biopython-1.76/Bio/Phylo/Applications/\n",
            "biopython-1.76/Bio/Phylo/Applications/_Fasttree.py\n",
            "biopython-1.76/Bio/Phylo/Applications/_Phyml.py\n",
            "biopython-1.76/Bio/Phylo/Applications/_Raxml.py\n",
            "biopython-1.76/Bio/Phylo/Applications/__init__.py\n",
            "biopython-1.76/Bio/Phylo/BaseTree.py\n",
            "biopython-1.76/Bio/Phylo/CDAO.py\n",
            "biopython-1.76/Bio/Phylo/CDAOIO.py\n",
            "biopython-1.76/Bio/Phylo/Consensus.py\n",
            "biopython-1.76/Bio/Phylo/NeXML.py\n",
            "biopython-1.76/Bio/Phylo/NeXMLIO.py\n",
            "biopython-1.76/Bio/Phylo/Newick.py\n",
            "biopython-1.76/Bio/Phylo/NewickIO.py\n",
            "biopython-1.76/Bio/Phylo/NexusIO.py\n",
            "biopython-1.76/Bio/Phylo/PAML/\n",
            "biopython-1.76/Bio/Phylo/PAML/__init__.py\n",
            "biopython-1.76/Bio/Phylo/PAML/_paml.py\n",
            "biopython-1.76/Bio/Phylo/PAML/_parse_baseml.py\n",
            "biopython-1.76/Bio/Phylo/PAML/_parse_codeml.py\n",
            "biopython-1.76/Bio/Phylo/PAML/_parse_yn00.py\n",
            "biopython-1.76/Bio/Phylo/PAML/baseml.py\n",
            "biopython-1.76/Bio/Phylo/PAML/chi2.py\n",
            "biopython-1.76/Bio/Phylo/PAML/codeml.py\n",
            "biopython-1.76/Bio/Phylo/PAML/yn00.py\n",
            "biopython-1.76/Bio/Phylo/PhyloXML.py\n",
            "biopython-1.76/Bio/Phylo/PhyloXMLIO.py\n",
            "biopython-1.76/Bio/Phylo/TreeConstruction.py\n",
            "biopython-1.76/Bio/Phylo/__init__.py\n",
            "biopython-1.76/Bio/Phylo/_cdao_owl.py\n",
            "biopython-1.76/Bio/Phylo/_io.py\n",
            "biopython-1.76/Bio/Phylo/_utils.py\n",
            "biopython-1.76/Bio/PopGen/\n",
            "biopython-1.76/Bio/PopGen/GenePop/\n",
            "biopython-1.76/Bio/PopGen/GenePop/Controller.py\n",
            "biopython-1.76/Bio/PopGen/GenePop/EasyController.py\n",
            "biopython-1.76/Bio/PopGen/GenePop/FileParser.py\n",
            "biopython-1.76/Bio/PopGen/GenePop/LargeFileParser.py\n",
            "biopython-1.76/Bio/PopGen/GenePop/Utils.py\n",
            "biopython-1.76/Bio/PopGen/GenePop/__init__.py\n",
            "biopython-1.76/Bio/PopGen/__init__.py\n",
            "biopython-1.76/Bio/Restriction/\n",
            "biopython-1.76/Bio/Restriction/PrintFormat.py\n",
            "biopython-1.76/Bio/Restriction/Restriction.py\n",
            "biopython-1.76/Bio/Restriction/Restriction_Dictionary.py\n",
            "biopython-1.76/Bio/Restriction/__init__.py\n",
            "biopython-1.76/Bio/SCOP/\n",
            "biopython-1.76/Bio/SCOP/Cla.py\n",
            "biopython-1.76/Bio/SCOP/Des.py\n",
            "biopython-1.76/Bio/SCOP/Dom.py\n",
            "biopython-1.76/Bio/SCOP/Hie.py\n",
            "biopython-1.76/Bio/SCOP/Raf.py\n",
            "biopython-1.76/Bio/SCOP/Residues.py\n",
            "biopython-1.76/Bio/SCOP/__init__.py\n",
            "biopython-1.76/Bio/SVDSuperimposer/\n",
            "biopython-1.76/Bio/SVDSuperimposer/__init__.py\n",
            "biopython-1.76/Bio/SearchIO/\n",
            "biopython-1.76/Bio/SearchIO/BlastIO/\n",
            "biopython-1.76/Bio/SearchIO/BlastIO/__init__.py\n",
            "biopython-1.76/Bio/SearchIO/BlastIO/blast_tab.py\n",
            "biopython-1.76/Bio/SearchIO/BlastIO/blast_text.py\n",
            "biopython-1.76/Bio/SearchIO/BlastIO/blast_xml.py\n",
            "biopython-1.76/Bio/SearchIO/BlatIO.py\n",
            "biopython-1.76/Bio/SearchIO/ExonerateIO/\n",
            "biopython-1.76/Bio/SearchIO/ExonerateIO/__init__.py\n",
            "biopython-1.76/Bio/SearchIO/ExonerateIO/_base.py\n",
            "biopython-1.76/Bio/SearchIO/ExonerateIO/exonerate_cigar.py\n",
            "biopython-1.76/Bio/SearchIO/ExonerateIO/exonerate_text.py\n",
            "biopython-1.76/Bio/SearchIO/ExonerateIO/exonerate_vulgar.py\n",
            "biopython-1.76/Bio/SearchIO/FastaIO.py\n",
            "biopython-1.76/Bio/SearchIO/HHsuiteIO/\n",
            "biopython-1.76/Bio/SearchIO/HHsuiteIO/__init__.py\n",
            "biopython-1.76/Bio/SearchIO/HHsuiteIO/hhsuite2_text.py\n",
            "biopython-1.76/Bio/SearchIO/HmmerIO/\n",
            "biopython-1.76/Bio/SearchIO/HmmerIO/__init__.py\n",
            "biopython-1.76/Bio/SearchIO/HmmerIO/_base.py\n",
            "biopython-1.76/Bio/SearchIO/HmmerIO/hmmer2_text.py\n",
            "biopython-1.76/Bio/SearchIO/HmmerIO/hmmer3_domtab.py\n",
            "biopython-1.76/Bio/SearchIO/HmmerIO/hmmer3_tab.py\n",
            "biopython-1.76/Bio/SearchIO/HmmerIO/hmmer3_text.py\n",
            "biopython-1.76/Bio/SearchIO/InterproscanIO/\n",
            "biopython-1.76/Bio/SearchIO/InterproscanIO/__init__.py\n",
            "biopython-1.76/Bio/SearchIO/InterproscanIO/interproscan_xml.py\n",
            "biopython-1.76/Bio/SearchIO/__init__.py\n",
            "biopython-1.76/Bio/SearchIO/_index.py\n",
            "biopython-1.76/Bio/SearchIO/_legacy/\n",
            "biopython-1.76/Bio/SearchIO/_legacy/NCBIStandalone.py\n",
            "biopython-1.76/Bio/SearchIO/_legacy/ParserSupport.py\n",
            "biopython-1.76/Bio/SearchIO/_legacy/__init__.py\n",
            "biopython-1.76/Bio/SearchIO/_model/\n",
            "biopython-1.76/Bio/SearchIO/_model/__init__.py\n",
            "biopython-1.76/Bio/SearchIO/_model/_base.py\n",
            "biopython-1.76/Bio/SearchIO/_model/hit.py\n",
            "biopython-1.76/Bio/SearchIO/_model/hsp.py\n",
            "biopython-1.76/Bio/SearchIO/_model/query.py\n",
            "biopython-1.76/Bio/SearchIO/_utils.py\n",
            "biopython-1.76/Bio/Seq.py\n",
            "biopython-1.76/Bio/SeqFeature.py\n",
            "biopython-1.76/Bio/SeqIO/\n",
            "biopython-1.76/Bio/SeqIO/AbiIO.py\n",
            "biopython-1.76/Bio/SeqIO/AceIO.py\n",
            "biopython-1.76/Bio/SeqIO/FastaIO.py\n",
            "biopython-1.76/Bio/SeqIO/GckIO.py\n",
            "biopython-1.76/Bio/SeqIO/IgIO.py\n",
            "biopython-1.76/Bio/SeqIO/InsdcIO.py\n",
            "biopython-1.76/Bio/SeqIO/Interfaces.py\n",
            "biopython-1.76/Bio/SeqIO/NibIO.py\n",
            "biopython-1.76/Bio/SeqIO/PdbIO.py\n",
            "biopython-1.76/Bio/SeqIO/PhdIO.py\n",
            "biopython-1.76/Bio/SeqIO/PirIO.py\n",
            "biopython-1.76/Bio/SeqIO/QualityIO.py\n",
            "biopython-1.76/Bio/SeqIO/SeqXmlIO.py\n",
            "biopython-1.76/Bio/SeqIO/SffIO.py\n",
            "biopython-1.76/Bio/SeqIO/SnapGeneIO.py\n",
            "biopython-1.76/Bio/SeqIO/SwissIO.py\n",
            "biopython-1.76/Bio/SeqIO/TabIO.py\n",
            "biopython-1.76/Bio/SeqIO/UniprotIO.py\n",
            "biopython-1.76/Bio/SeqIO/XdnaIO.py\n",
            "biopython-1.76/Bio/SeqIO/__init__.py\n",
            "biopython-1.76/Bio/SeqIO/_convert.py\n",
            "biopython-1.76/Bio/SeqIO/_index.py\n",
            "biopython-1.76/Bio/SeqRecord.py\n",
            "biopython-1.76/Bio/SeqUtils/\n",
            "biopython-1.76/Bio/SeqUtils/CheckSum.py\n",
            "biopython-1.76/Bio/SeqUtils/CodonUsage.py\n",
            "biopython-1.76/Bio/SeqUtils/CodonUsageIndices.py\n",
            "biopython-1.76/Bio/SeqUtils/IsoelectricPoint.py\n",
            "biopython-1.76/Bio/SeqUtils/MeltingTemp.py\n",
            "biopython-1.76/Bio/SeqUtils/ProtParam.py\n",
            "biopython-1.76/Bio/SeqUtils/ProtParamData.py\n",
            "biopython-1.76/Bio/SeqUtils/__init__.py\n",
            "biopython-1.76/Bio/SeqUtils/lcc.py\n",
            "biopython-1.76/Bio/Sequencing/\n",
            "biopython-1.76/Bio/Sequencing/Ace.py\n",
            "biopython-1.76/Bio/Sequencing/Applications/\n",
            "biopython-1.76/Bio/Sequencing/Applications/_Novoalign.py\n",
            "biopython-1.76/Bio/Sequencing/Applications/__init__.py\n",
            "biopython-1.76/Bio/Sequencing/Applications/_bwa.py\n",
            "biopython-1.76/Bio/Sequencing/Applications/_samtools.py\n",
            "biopython-1.76/Bio/Sequencing/Phd.py\n",
            "biopython-1.76/Bio/Sequencing/__init__.py\n",
            "biopython-1.76/Bio/Statistics/\n",
            "biopython-1.76/Bio/Statistics/__init__.py\n",
            "biopython-1.76/Bio/Statistics/lowess.py\n",
            "biopython-1.76/Bio/SubsMat/\n",
            "biopython-1.76/Bio/SubsMat/FreqTable.py\n",
            "biopython-1.76/Bio/SubsMat/MatrixInfo.py\n",
            "biopython-1.76/Bio/SubsMat/__init__.py\n",
            "biopython-1.76/Bio/SwissProt/\n",
            "biopython-1.76/Bio/SwissProt/KeyWList.py\n",
            "biopython-1.76/Bio/SwissProt/__init__.py\n",
            "biopython-1.76/Bio/TogoWS/\n",
            "biopython-1.76/Bio/TogoWS/__init__.py\n",
            "biopython-1.76/Bio/UniGene/\n",
            "biopython-1.76/Bio/UniGene/__init__.py\n",
            "biopython-1.76/Bio/UniProt/\n",
            "biopython-1.76/Bio/UniProt/GOA.py\n",
            "biopython-1.76/Bio/UniProt/__init__.py\n",
            "biopython-1.76/Bio/Wise/\n",
            "biopython-1.76/Bio/Wise/__init__.py\n",
            "biopython-1.76/Bio/Wise/dnal.py\n",
            "biopython-1.76/Bio/Wise/psw.py\n",
            "biopython-1.76/Bio/__init__.py\n",
            "biopython-1.76/Bio/_py3k/\n",
            "biopython-1.76/Bio/_py3k/__init__.py\n",
            "biopython-1.76/Bio/_utils.py\n",
            "biopython-1.76/Bio/bgzf.py\n",
            "biopython-1.76/Bio/codonalign/\n",
            "biopython-1.76/Bio/codonalign/__init__.py\n",
            "biopython-1.76/Bio/codonalign/chisq.py\n",
            "biopython-1.76/Bio/codonalign/codonalignment.py\n",
            "biopython-1.76/Bio/codonalign/codonalphabet.py\n",
            "biopython-1.76/Bio/codonalign/codonseq.py\n",
            "biopython-1.76/Bio/cpairwise2module.c\n",
            "biopython-1.76/Bio/kNN.py\n",
            "biopython-1.76/Bio/motifs/\n",
            "biopython-1.76/Bio/motifs/__init__.py\n",
            "biopython-1.76/Bio/motifs/_pwm.c\n",
            "biopython-1.76/Bio/motifs/alignace.py\n",
            "biopython-1.76/Bio/motifs/applications/\n",
            "biopython-1.76/Bio/motifs/applications/__init__.py\n",
            "biopython-1.76/Bio/motifs/applications/_xxmotif.py\n",
            "biopython-1.76/Bio/motifs/clusterbuster.py\n",
            "biopython-1.76/Bio/motifs/jaspar/\n",
            "biopython-1.76/Bio/motifs/jaspar/__init__.py\n",
            "biopython-1.76/Bio/motifs/jaspar/db.py\n",
            "biopython-1.76/Bio/motifs/mast.py\n",
            "biopython-1.76/Bio/motifs/matrix.py\n",
            "biopython-1.76/Bio/motifs/meme.py\n",
            "biopython-1.76/Bio/motifs/minimal.py\n",
            "biopython-1.76/Bio/motifs/pfm.py\n",
            "biopython-1.76/Bio/motifs/thresholds.py\n",
            "biopython-1.76/Bio/motifs/transfac.py\n",
            "biopython-1.76/Bio/motifs/xms.py\n",
            "biopython-1.76/Bio/pairwise2.py\n",
            "biopython-1.76/Bio/phenotype/\n",
            "biopython-1.76/Bio/phenotype/__init__.py\n",
            "biopython-1.76/Bio/phenotype/phen_micro.py\n",
            "biopython-1.76/Bio/phenotype/pm_fitting.py\n",
            "biopython-1.76/Bio/trie.c\n",
            "biopython-1.76/Bio/trie.h\n",
            "biopython-1.76/Bio/triefind.py\n",
            "biopython-1.76/Bio/triemodule.c\n",
            "biopython-1.76/BioSQL/\n",
            "biopython-1.76/BioSQL/BioSeq.py\n",
            "biopython-1.76/BioSQL/BioSeqDatabase.py\n",
            "biopython-1.76/BioSQL/DBUtils.py\n",
            "biopython-1.76/BioSQL/Loader.py\n",
            "biopython-1.76/BioSQL/__init__.py\n",
            "biopython-1.76/CONTRIB.rst\n",
            "biopython-1.76/CONTRIBUTING.rst\n",
            "biopython-1.76/DEPRECATED.rst\n",
            "biopython-1.76/Doc/\n",
            "biopython-1.76/Doc/Makefile\n",
            "biopython-1.76/Doc/Tutorial/\n",
            "biopython-1.76/Doc/Tutorial/chapter_advanced.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_align.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_appendix.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_blast.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_cluster.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_contributing.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_cookbook.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_entrez.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_graphics.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_introduction.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_kegg.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_learning.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_motifs.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_pdb.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_phenotype.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_phylo.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_popgen.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_quick_start.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_searchio.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_seq_annot.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_seq_objects.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_seqio.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_testing.tex\n",
            "biopython-1.76/Doc/Tutorial/chapter_uniprot.tex\n",
            "biopython-1.76/Doc/Tutorial.html\n",
            "biopython-1.76/Doc/Tutorial.pdf\n",
            "biopython-1.76/Doc/Tutorial.tex\n",
            "biopython-1.76/Doc/api/\n",
            "biopython-1.76/Doc/api/Makefile\n",
            "biopython-1.76/Doc/api/conf.py\n",
            "biopython-1.76/Doc/biopdb_faq.tex\n",
            "biopython-1.76/Doc/cookbook/\n",
            "biopython-1.76/Doc/cookbook/Restriction/\n",
            "biopython-1.76/Doc/cookbook/Restriction/Restriction.html\n",
            "biopython-1.76/Doc/doc.tex\n",
            "biopython-1.76/Doc/examples/\n",
            "biopython-1.76/Doc/examples/ACT_example.py\n",
            "biopython-1.76/Doc/examples/PF05371_seed.sth\n",
            "biopython-1.76/Doc/examples/Plates.csv\n",
            "biopython-1.76/Doc/examples/Proux_et_al_2002_Figure_6.py\n",
            "biopython-1.76/Doc/examples/alpha.faa\n",
            "biopython-1.76/Doc/examples/beta.faa\n",
            "biopython-1.76/Doc/examples/clustal_run.py\n",
            "biopython-1.76/Doc/examples/ec_5.4.2.2.txt\n",
            "biopython-1.76/Doc/examples/fasta_dictionary.py\n",
            "biopython-1.76/Doc/examples/fasta_iterator.py\n",
            "biopython-1.76/Doc/examples/getgene.py\n",
            "biopython-1.76/Doc/examples/ls_orchid.fasta\n",
            "biopython-1.76/Doc/examples/ls_orchid.gbk\n",
            "biopython-1.76/Doc/examples/ls_orchid.gbk.bgz\n",
            "biopython-1.76/Doc/examples/ls_orchid.gbk.bz2\n",
            "biopython-1.76/Doc/examples/ls_orchid.gbk.gz\n",
            "biopython-1.76/Doc/examples/m_cold.fasta\n",
            "biopython-1.76/Doc/examples/make_subsmat.py\n",
            "biopython-1.76/Doc/examples/my_blast.xml\n",
            "biopython-1.76/Doc/examples/my_blat.psl\n",
            "biopython-1.76/Doc/examples/nmr/\n",
            "biopython-1.76/Doc/examples/nmr/noed.xpk\n",
            "biopython-1.76/Doc/examples/nmr/simplepredict.py\n",
            "biopython-1.76/Doc/examples/opuntia.aln\n",
            "biopython-1.76/Doc/examples/opuntia.dnd\n",
            "biopython-1.76/Doc/examples/opuntia.fasta\n",
            "biopython-1.76/Doc/examples/protein.aln\n",
            "biopython-1.76/Doc/examples/simple.dnd\n",
            "biopython-1.76/Doc/examples/swissprot.py\n",
            "biopython-1.76/Doc/examples/www_blast.py\n",
            "biopython-1.76/Doc/images/\n",
            "biopython-1.76/Doc/images/BlastRecord.png\n",
            "biopython-1.76/Doc/images/GD_sigil_arrow_heads.png\n",
            "biopython-1.76/Doc/images/GD_sigil_arrow_shafts.png\n",
            "biopython-1.76/Doc/images/GD_sigil_labels.png\n",
            "biopython-1.76/Doc/images/GD_sigils.png\n",
            "biopython-1.76/Doc/images/PSIBlastRecord.png\n",
            "biopython-1.76/Doc/images/SRR001666.png\n",
            "biopython-1.76/Doc/images/biopython_logo.pdf\n",
            "biopython-1.76/Doc/images/biopython_logo.svg\n",
            "biopython-1.76/Doc/images/biopython_logo_l.png\n",
            "biopython-1.76/Doc/images/biopython_logo_m.png\n",
            "biopython-1.76/Doc/images/biopython_logo_old.jpg\n",
            "biopython-1.76/Doc/images/biopython_logo_s.png\n",
            "biopython-1.76/Doc/images/biopython_logo_xs.png\n",
            "biopython-1.76/Doc/images/dot_plot.png\n",
            "biopython-1.76/Doc/images/dot_plot_scatter.png\n",
            "biopython-1.76/Doc/images/gc_plot.png\n",
            "biopython-1.76/Doc/images/hist_plot.png\n",
            "biopython-1.76/Doc/images/phylo-apaf-zoom.png\n",
            "biopython-1.76/Doc/images/phylo-apaf.png\n",
            "biopython-1.76/Doc/images/phylo-color-draw.png\n",
            "biopython-1.76/Doc/images/phylo-color.png\n",
            "biopython-1.76/Doc/images/phylo-dot.png\n",
            "biopython-1.76/Doc/images/phylo-draw-example.png\n",
            "biopython-1.76/Doc/images/phylo-rooted.png\n",
            "biopython-1.76/Doc/images/phylo-simple-draw.png\n",
            "biopython-1.76/Doc/images/plasmid_circular.png\n",
            "biopython-1.76/Doc/images/plasmid_circular_nice.png\n",
            "biopython-1.76/Doc/images/plasmid_linear.png\n",
            "biopython-1.76/Doc/images/plasmid_linear_nice.png\n",
            "biopython-1.76/Doc/images/simple_chrom.pdf\n",
            "biopython-1.76/Doc/images/simple_chrom.png\n",
            "biopython-1.76/Doc/images/smcra.png\n",
            "biopython-1.76/Doc/images/tRNA_chrom.pdf\n",
            "biopython-1.76/Doc/images/tRNA_chrom.png\n",
            "biopython-1.76/Doc/images/three_track_cl.png\n",
            "biopython-1.76/Doc/images/three_track_cl2a.png\n",
            "biopython-1.76/Doc/images/three_track_simple.png\n",
            "biopython-1.76/LICENSE.rst\n",
            "biopython-1.76/MANIFEST.in\n",
            "biopython-1.76/NEWS.rst\n",
            "biopython-1.76/PKG-INFO\n",
            "biopython-1.76/README.rst\n",
            "biopython-1.76/Scripts/\n",
            "biopython-1.76/Scripts/GenBank/\n",
            "biopython-1.76/Scripts/GenBank/check_output.py\n",
            "biopython-1.76/Scripts/GenBank/check_output_simple.py\n",
            "biopython-1.76/Scripts/PDB/\n",
            "biopython-1.76/Scripts/PDB/generate_three_to_one_dict.py\n",
            "biopython-1.76/Scripts/Performance/\n",
            "biopython-1.76/Scripts/Performance/biosql_performance_load.py\n",
            "biopython-1.76/Scripts/Performance/biosql_performance_read.py\n",
            "biopython-1.76/Scripts/Restriction/\n",
            "biopython-1.76/Scripts/Restriction/ranacompiler.py\n",
            "biopython-1.76/Scripts/Restriction/rebase_update.py\n",
            "biopython-1.76/Scripts/SeqGui/\n",
            "biopython-1.76/Scripts/SeqGui/SeqGui.py\n",
            "biopython-1.76/Scripts/Structure/\n",
            "biopython-1.76/Scripts/Structure/hsexpo\n",
            "biopython-1.76/Scripts/query_pubmed.py\n",
            "biopython-1.76/Scripts/scop_pdb.py\n",
            "biopython-1.76/Scripts/update_ncbi_codon_table.py\n",
            "biopython-1.76/Scripts/xbbtools/\n",
            "biopython-1.76/Scripts/xbbtools/README\n",
            "biopython-1.76/Scripts/xbbtools/nextorf.py\n",
            "biopython-1.76/Scripts/xbbtools/test.fas\n",
            "biopython-1.76/Scripts/xbbtools/testrp.fas\n",
            "biopython-1.76/Scripts/xbbtools/xbb_blast.py\n",
            "biopython-1.76/Scripts/xbbtools/xbb_blastbg.py\n",
            "biopython-1.76/Scripts/xbbtools/xbb_help.py\n",
            "biopython-1.76/Scripts/xbbtools/xbb_search.py\n",
            "biopython-1.76/Scripts/xbbtools/xbb_translations.py\n",
            "biopython-1.76/Scripts/xbbtools/xbb_utils.py\n",
            "biopython-1.76/Scripts/xbbtools/xbb_widget.py\n",
            "biopython-1.76/Scripts/xbbtools/xbbtools.py\n",
            "biopython-1.76/Tests/\n",
            "biopython-1.76/Tests/.coveragerc\n",
            "biopython-1.76/Tests/Abi/\n",
            "biopython-1.76/Tests/Abi/310.ab1\n",
            "biopython-1.76/Tests/Abi/3100.ab1\n",
            "biopython-1.76/Tests/Abi/3730.ab1\n",
            "biopython-1.76/Tests/Abi/A6_1-DB3.ab1\n",
            "biopython-1.76/Tests/Abi/empty.ab1\n",
            "biopython-1.76/Tests/Abi/fake.ab1\n",
            "biopython-1.76/Tests/Abi/no_smpl1.ab1\n",
            "biopython-1.76/Tests/Abi/nonascii_encoding.ab1\n",
            "biopython-1.76/Tests/Abi/test.fsa\n",
            "biopython-1.76/Tests/Abi/test_data\n",
            "biopython-1.76/Tests/Ace/\n",
            "biopython-1.76/Tests/Ace/consed_sample.ace\n",
            "biopython-1.76/Tests/Ace/contig1.ace\n",
            "biopython-1.76/Tests/Ace/seq.cap.ace\n",
            "biopython-1.76/Tests/Affy/\n",
            "biopython-1.76/Tests/Affy/affy_v3_example.CEL\n",
            "biopython-1.76/Tests/Affy/affy_v4_example.CEL\n",
            "biopython-1.76/Tests/Align/\n",
            "biopython-1.76/Tests/Align/bsubtilis.fa\n",
            "biopython-1.76/Tests/Align/bsubtilis.txt\n",
            "biopython-1.76/Tests/Align/cow.fa\n",
            "biopython-1.76/Tests/Align/cow.txt\n",
            "biopython-1.76/Tests/Align/ecoli.fa\n",
            "biopython-1.76/Tests/Align/ecoli.txt\n",
            "biopython-1.76/Tests/Align/hg38.chrom.sizes\n",
            "biopython-1.76/Tests/Align/pig.fa\n",
            "biopython-1.76/Tests/Align/pig.txt\n",
            "biopython-1.76/Tests/BWA/\n",
            "biopython-1.76/Tests/BWA/HNSCC1_1_truncated.fastq\n",
            "biopython-1.76/Tests/BWA/HNSCC1_2_truncated.fastq\n",
            "biopython-1.76/Tests/BWA/human_g1k_v37_truncated.fasta\n",
            "biopython-1.76/Tests/BioSQL/\n",
            "biopython-1.76/Tests/BioSQL/biosqldb-mysql.sql\n",
            "biopython-1.76/Tests/BioSQL/biosqldb-pg.sql\n",
            "biopython-1.76/Tests/BioSQL/biosqldb-sqlite.sql\n",
            "biopython-1.76/Tests/BioSQL/cor6_6.db\n",
            "biopython-1.76/Tests/Blast/\n",
            "biopython-1.76/Tests/Blast/README.txt\n",
            "biopython-1.76/Tests/Blast/html_msgid_29_blastx_001.html\n",
            "biopython-1.76/Tests/Blast/mirna.tab\n",
            "biopython-1.76/Tests/Blast/mirna.xml\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_001.txt\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_002.txt\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_003.txt\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_004.txt\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_005.txt\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_006.txt\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_007.txt\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_008.txt\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_009.txt\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_010.txt\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_011.txt\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_012.txt\n",
            "biopython-1.76/Tests/Blast/tab_2226_tblastn_013.txt\n",
            "biopython-1.76/Tests/Blast/tab_2228_tblastn_001.txt\n",
            "biopython-1.76/Tests/Blast/tab_2228_tblastx_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastn_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastn_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastn_003.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastn_004.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastp_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastp_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastp_003.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastp_004.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastp_005.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastp_006.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastp_007.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastx_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_blastx_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_phiblast_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_phiblast_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_phiblast_003.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_tblastn_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_tblastn_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2010L_tblastx_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastn_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastn_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastn_003.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastn_004.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastp_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastp_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastp_003.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastp_004.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastp_005.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastp_006.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastp_007.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastx_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_blastx_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_phiblast_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_phiblast_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_phiblast_003.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_tblastn_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_tblastn_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2011L_tblastx_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2012L_psiblast_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2014L_blastn_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2014L_psiblast_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2102L_blastn_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2201L_blastx_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2202L_blastn_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2202L_blastn_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2202L_blastp_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2208L_psiblast_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2215L_blastx_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2216L_tblastn_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2220L_blastx_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2220L_blastx_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2221L_blastp_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2222L_blastx_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2222_blastx_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_blastn_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_blastn_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_blastn_003.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_blastn_004.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_blastp_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_blastp_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_blastp_003.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_blastp_004.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_blastx_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_blastx_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_blastx_003.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_blastx_004.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_tblastn_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_tblastn_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_tblastn_003.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_tblastn_004.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_tblastx_001.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_tblastx_002.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_tblastx_003.txt\n",
            "biopython-1.76/Tests/Blast/text_2226_tblastx_004.txt\n",
            "biopython-1.76/Tests/Blast/text_2230_blastp_001.txt\n",
            "biopython-1.76/Tests/Blast/wnts.xml\n",
            "biopython-1.76/Tests/Blast/wnts.xml.bgz\n",
            "biopython-1.76/Tests/Blast/xml_2212L_blastn_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2212L_blastp_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2212L_blastx_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2212L_tblastn_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2212L_tblastx_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2218L_blastp_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2218L_rpsblast_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2218_blastp_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2218_blastp_002.xml\n",
            "biopython-1.76/Tests/Blast/xml_2222_blastp_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2222_blastx_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastn_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastn_002.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastn_003.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastn_004.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastn_005.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastn_006.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastp_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastp_002.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastp_003.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastp_004.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastp_005.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastx_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastx_002.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastx_003.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_blastx_004.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_tblastn_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_tblastn_002.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_tblastn_003.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_tblastn_004.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_tblastn_005.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_tblastx_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_tblastx_002.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_tblastx_003.xml\n",
            "biopython-1.76/Tests/Blast/xml_2226_tblastx_004.xml\n",
            "biopython-1.76/Tests/Blast/xml_2900_blastn_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2900_blastn_001_v2.xml\n",
            "biopython-1.76/Tests/Blast/xml_2900_blastp_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2900_blastp_001_v2.xml\n",
            "biopython-1.76/Tests/Blast/xml_2900_blastx_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2900_blastx_001_v2.xml\n",
            "biopython-1.76/Tests/Blast/xml_2900_tblastn_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2900_tblastn_001_v2.xml\n",
            "biopython-1.76/Tests/Blast/xml_2900_tblastx_001.xml\n",
            "biopython-1.76/Tests/Blast/xml_2900_tblastx_001_v2.xml\n",
            "biopython-1.76/Tests/Blat/\n",
            "biopython-1.76/Tests/Blat/README.txt\n",
            "biopython-1.76/Tests/Blat/mirna.pslx\n",
            "biopython-1.76/Tests/Blat/psl_34_001.psl\n",
            "biopython-1.76/Tests/Blat/psl_34_002.psl\n",
            "biopython-1.76/Tests/Blat/psl_34_003.psl\n",
            "biopython-1.76/Tests/Blat/psl_34_004.psl\n",
            "biopython-1.76/Tests/Blat/psl_34_005.psl\n",
            "biopython-1.76/Tests/Blat/psl_35_001.psl\n",
            "biopython-1.76/Tests/Blat/psl_35_002.psl\n",
            "biopython-1.76/Tests/Blat/pslx_34_001.pslx\n",
            "biopython-1.76/Tests/Blat/pslx_34_002.pslx\n",
            "biopython-1.76/Tests/Blat/pslx_34_003.pslx\n",
            "biopython-1.76/Tests/Blat/pslx_34_004.pslx\n",
            "biopython-1.76/Tests/Blat/pslx_34_005.pslx\n",
            "biopython-1.76/Tests/Blat/pslx_35_002.pslx\n",
            "biopython-1.76/Tests/CDAO/\n",
            "biopython-1.76/Tests/CDAO/test.cdao\n",
            "biopython-1.76/Tests/Cellosaurus/\n",
            "biopython-1.76/Tests/Cellosaurus/cell_lines_1.txt\n",
            "biopython-1.76/Tests/Cellosaurus/cell_lines_2.txt\n",
            "biopython-1.76/Tests/Cellosaurus/cell_lines_3.txt\n",
            "biopython-1.76/Tests/Clustalw/\n",
            "biopython-1.76/Tests/Clustalw/cw02.aln\n",
            "biopython-1.76/Tests/Clustalw/hedgehog.aln\n",
            "biopython-1.76/Tests/Clustalw/odd_consensus.aln\n",
            "biopython-1.76/Tests/Clustalw/opuntia.aln\n",
            "biopython-1.76/Tests/Clustalw/promals3d.aln\n",
            "biopython-1.76/Tests/Clustalw/protein.aln\n",
            "biopython-1.76/Tests/Cluster/\n",
            "biopython-1.76/Tests/Cluster/README\n",
            "biopython-1.76/Tests/Cluster/cyano.txt\n",
            "biopython-1.76/Tests/CodonUsage/\n",
            "biopython-1.76/Tests/CodonUsage/HighlyExpressedGenes.txt\n",
            "biopython-1.76/Tests/Compass/\n",
            "biopython-1.76/Tests/Compass/comtest1\n",
            "biopython-1.76/Tests/Compass/comtest2\n",
            "biopython-1.76/Tests/EMBL/\n",
            "biopython-1.76/Tests/EMBL/101ma_no_coords.embl\n",
            "biopython-1.76/Tests/EMBL/A04195.imgt\n",
            "biopython-1.76/Tests/EMBL/AAA03323.embl\n",
            "biopython-1.76/Tests/EMBL/AE017046.embl\n",
            "biopython-1.76/Tests/EMBL/DD231055_edited.embl\n",
            "biopython-1.76/Tests/EMBL/DD231055_edited2.embl\n",
            "biopython-1.76/Tests/EMBL/DS830848.embl\n",
            "biopython-1.76/Tests/EMBL/Human_contigs.embl\n",
            "biopython-1.76/Tests/EMBL/RepBase23.02.embl\n",
            "biopython-1.76/Tests/EMBL/SC10H5.embl\n",
            "biopython-1.76/Tests/EMBL/TRBG361.embl\n",
            "biopython-1.76/Tests/EMBL/U87107.embl\n",
            "biopython-1.76/Tests/EMBL/embl_with_0_line.embl\n",
            "biopython-1.76/Tests/EMBL/epo_prt_selection.embl\n",
            "biopython-1.76/Tests/EMBL/hla_3260_sample.imgt\n",
            "biopython-1.76/Tests/EMBL/kipo_prt_sample.embl\n",
            "biopython-1.76/Tests/EMBL/location_wrap.embl\n",
            "biopython-1.76/Tests/EMBL/patents.embl\n",
            "biopython-1.76/Tests/EMBL/xx_after_co.embl\n",
            "biopython-1.76/Tests/Emboss/\n",
            "biopython-1.76/Tests/Emboss/NirK.primer3\n",
            "biopython-1.76/Tests/Emboss/NirK_full.primer3\n",
            "biopython-1.76/Tests/Emboss/alignret.txt\n",
            "biopython-1.76/Tests/Emboss/bac_find.primer3\n",
            "biopython-1.76/Tests/Emboss/bac_find.psearch\n",
            "biopython-1.76/Tests/Emboss/cds_forward.primer3\n",
            "biopython-1.76/Tests/Emboss/cds_reverse.primer3\n",
            "biopython-1.76/Tests/Emboss/emboss_pair_aln_full_blank_line.txt\n",
            "biopython-1.76/Tests/Emboss/internal_oligo.primer3\n",
            "biopython-1.76/Tests/Emboss/matcher_pair.txt\n",
            "biopython-1.76/Tests/Emboss/matcher_simple.txt\n",
            "biopython-1.76/Tests/Emboss/needle.txt\n",
            "biopython-1.76/Tests/Emboss/needle_asis.txt\n",
            "biopython-1.76/Tests/Emboss/no_oligo.primer3\n",
            "biopython-1.76/Tests/Emboss/short.primer3\n",
            "biopython-1.76/Tests/Emboss/water.txt\n",
            "biopython-1.76/Tests/Emboss/water2.txt\n",
            "biopython-1.76/Tests/Entrez/\n",
            "biopython-1.76/Tests/Entrez/efetch_schemas.xml\n",
            "biopython-1.76/Tests/Entrez/egquery1.xml\n",
            "biopython-1.76/Tests/Entrez/egquery2.xml\n",
            "biopython-1.76/Tests/Entrez/einfo1.xml\n",
            "biopython-1.76/Tests/Entrez/einfo2.xml\n",
            "biopython-1.76/Tests/Entrez/einfo3.xml\n",
            "biopython-1.76/Tests/Entrez/einfo4.xml\n",
            "biopython-1.76/Tests/Entrez/elink1.xml\n",
            "biopython-1.76/Tests/Entrez/elink2.xml\n",
            "biopython-1.76/Tests/Entrez/elink3.xml\n",
            "biopython-1.76/Tests/Entrez/elink4.xml\n",
            "biopython-1.76/Tests/Entrez/elink5.xml\n",
            "biopython-1.76/Tests/Entrez/elink6.xml\n",
            "biopython-1.76/Tests/Entrez/elink7.xml\n",
            "biopython-1.76/Tests/Entrez/elink8.xml\n",
            "biopython-1.76/Tests/Entrez/epost1.xml\n",
            "biopython-1.76/Tests/Entrez/epost2.xml\n",
            "biopython-1.76/Tests/Entrez/epost3.xml\n",
            "biopython-1.76/Tests/Entrez/esearch1.xml\n",
            "biopython-1.76/Tests/Entrez/esearch2.xml\n",
            "biopython-1.76/Tests/Entrez/esearch3.xml\n",
            "biopython-1.76/Tests/Entrez/esearch4.xml\n",
            "biopython-1.76/Tests/Entrez/esearch5.xml\n",
            "biopython-1.76/Tests/Entrez/esearch6.xml\n",
            "biopython-1.76/Tests/Entrez/esearch7.xml\n",
            "biopython-1.76/Tests/Entrez/esearch8.xml\n",
            "biopython-1.76/Tests/Entrez/espell.xml\n",
            "biopython-1.76/Tests/Entrez/esummary1.xml\n",
            "biopython-1.76/Tests/Entrez/esummary2.xml\n",
            "biopython-1.76/Tests/Entrez/esummary3.xml\n",
            "biopython-1.76/Tests/Entrez/esummary4.xml\n",
            "biopython-1.76/Tests/Entrez/esummary5.xml\n",
            "biopython-1.76/Tests/Entrez/esummary6.xml\n",
            "biopython-1.76/Tests/Entrez/esummary7.xml\n",
            "biopython-1.76/Tests/Entrez/esummary8.xml\n",
            "biopython-1.76/Tests/Entrez/esummary9.xml\n",
            "biopython-1.76/Tests/Entrez/journals.xml\n",
            "biopython-1.76/Tests/Entrez/ncbi_mim.xml\n",
            "biopython-1.76/Tests/Entrez/nucleotide1.xml\n",
            "biopython-1.76/Tests/Entrez/nucleotide2.xml\n",
            "biopython-1.76/Tests/Entrez/protein.xml\n",
            "biopython-1.76/Tests/Entrez/pubmed1.xml\n",
            "biopython-1.76/Tests/Entrez/pubmed2.xml\n",
            "biopython-1.76/Tests/Entrez/pubmed3.html\n",
            "biopython-1.76/Tests/Entrez/pubmed4.xml\n",
            "biopython-1.76/Tests/Entrez/pubmed5.xml\n",
            "biopython-1.76/Tests/Entrez/pubmed6.xml\n",
            "biopython-1.76/Tests/Entrez/pubmed7.xml\n",
            "biopython-1.76/Tests/Entrez/taxonomy.xml\n",
            "biopython-1.76/Tests/Enzymes/\n",
            "biopython-1.76/Tests/Enzymes/lactate.txt\n",
            "biopython-1.76/Tests/Enzymes/lipoprotein.txt\n",
            "biopython-1.76/Tests/Enzymes/proline.txt\n",
            "biopython-1.76/Tests/Enzymes/valine.txt\n",
            "biopython-1.76/Tests/Exonerate/\n",
            "biopython-1.76/Tests/Exonerate/README.txt\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_affine_local.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_cdna2genome.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_coding2coding.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_coding2coding_fshifts.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_coding2genome.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_dna2protein.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_est2genome.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_genome2genome.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_ner.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_protein2dna.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_protein2dna_fshifts.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_protein2genome.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_ungapped.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_m_ungapped_trans.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_o_vulgar.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_o_vulgar_cigar.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_o_vulgar_fshifts.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_o_vulgar_fshifts2.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_q_multiple.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_q_multiple_vulgar.exn\n",
            "biopython-1.76/Tests/Exonerate/exn_22_q_none.exn\n",
            "biopython-1.76/Tests/ExtendedPhylip/\n",
            "biopython-1.76/Tests/ExtendedPhylip/primates.phyx\n",
            "biopython-1.76/Tests/FSSP/\n",
            "biopython-1.76/Tests/FSSP/1cnv.fssp\n",
            "biopython-1.76/Tests/Fasta/\n",
            "biopython-1.76/Tests/Fasta/README\n",
            "biopython-1.76/Tests/Fasta/aster.pro\n",
            "biopython-1.76/Tests/Fasta/aster_no_wrap.pro\n",
            "biopython-1.76/Tests/Fasta/centaurea.nu\n",
            "biopython-1.76/Tests/Fasta/dups.fasta\n",
            "biopython-1.76/Tests/Fasta/elderberry.nu\n",
            "biopython-1.76/Tests/Fasta/f001\n",
            "biopython-1.76/Tests/Fasta/f002\n",
            "biopython-1.76/Tests/Fasta/f003\n",
            "biopython-1.76/Tests/Fasta/fa01\n",
            "biopython-1.76/Tests/Fasta/flowers.pro.gz\n",
            "biopython-1.76/Tests/Fasta/lavender.nu\n",
            "biopython-1.76/Tests/Fasta/loveliesbleeding.pro\n",
            "biopython-1.76/Tests/Fasta/lupine.nu\n",
            "biopython-1.76/Tests/Fasta/output001.m10\n",
            "biopython-1.76/Tests/Fasta/output002.m10\n",
            "biopython-1.76/Tests/Fasta/output003.m10\n",
            "biopython-1.76/Tests/Fasta/output004.m10\n",
            "biopython-1.76/Tests/Fasta/output005.m10\n",
            "biopython-1.76/Tests/Fasta/output006.m10\n",
            "biopython-1.76/Tests/Fasta/output007.m10\n",
            "biopython-1.76/Tests/Fasta/output008.m10\n",
            "biopython-1.76/Tests/Fasta/output009.m10\n",
            "biopython-1.76/Tests/Fasta/output010.m10\n",
            "biopython-1.76/Tests/Fasta/output011.m10\n",
            "biopython-1.76/Tests/Fasta/output012.m10\n",
            "biopython-1.76/Tests/Fasta/output013.m10\n",
            "biopython-1.76/Tests/Fasta/output014.m10\n",
            "biopython-1.76/Tests/Fasta/output015.m10\n",
            "biopython-1.76/Tests/Fasta/output016.m10\n",
            "biopython-1.76/Tests/Fasta/phlox.nu\n",
            "biopython-1.76/Tests/Fasta/rose.pro\n",
            "biopython-1.76/Tests/Fasta/rosemary.pro\n",
            "biopython-1.76/Tests/Fasta/sweetpea.nu\n",
            "biopython-1.76/Tests/Fasta/wisteria.nu\n",
            "biopython-1.76/Tests/GFF/\n",
            "biopython-1.76/Tests/GFF/NC_001422.gbk\n",
            "biopython-1.76/Tests/GFF/NC_001802.fna\n",
            "biopython-1.76/Tests/GFF/NC_001802lc.fna\n",
            "biopython-1.76/Tests/GFF/NC_001802x.fna\n",
            "biopython-1.76/Tests/GFF/multi.fna\n",
            "biopython-1.76/Tests/Gck/\n",
            "biopython-1.76/Tests/Gck/artificial.gck\n",
            "biopython-1.76/Tests/GenBank/\n",
            "biopython-1.76/Tests/GenBank/1MRR_A.gp\n",
            "biopython-1.76/Tests/GenBank/DS830848.gb\n",
            "biopython-1.76/Tests/GenBank/EU851978.gbk\n",
            "biopython-1.76/Tests/GenBank/EU851978_output.gbk\n",
            "biopython-1.76/Tests/GenBank/GU949562.1.gb\n",
            "biopython-1.76/Tests/GenBank/HM138502.gbk\n",
            "biopython-1.76/Tests/GenBank/HM138502_output.gbk\n",
            "biopython-1.76/Tests/GenBank/KF527485.gbk\n",
            "biopython-1.76/Tests/GenBank/KF527485_output.gbk\n",
            "biopython-1.76/Tests/GenBank/NC_000932.faa\n",
            "biopython-1.76/Tests/GenBank/NC_000932.gb\n",
            "biopython-1.76/Tests/GenBank/NC_000932.gb.bgz\n",
            "biopython-1.76/Tests/GenBank/NC_005816.faa\n",
            "biopython-1.76/Tests/GenBank/NC_005816.ffn\n",
            "biopython-1.76/Tests/GenBank/NC_005816.fna\n",
            "biopython-1.76/Tests/GenBank/NC_005816.gb\n",
            "biopython-1.76/Tests/GenBank/NC_005816.tsv\n",
            "biopython-1.76/Tests/GenBank/NP_416719.gbwithparts\n",
            "biopython-1.76/Tests/GenBank/NT_019265.gb\n",
            "biopython-1.76/Tests/GenBank/arab1.gb\n",
            "biopython-1.76/Tests/GenBank/bad_loc_wrap.gb\n",
            "biopython-1.76/Tests/GenBank/bad_origin_wrap.gb\n",
            "biopython-1.76/Tests/GenBank/bad_origin_wrap_CDS.gb\n",
            "biopython-1.76/Tests/GenBank/bad_origin_wrap_fuzzy.gb\n",
            "biopython-1.76/Tests/GenBank/bad_origin_wrap_linear.gb\n",
            "biopython-1.76/Tests/GenBank/blank_seq.gb\n",
            "biopython-1.76/Tests/GenBank/cor6_6.gb\n",
            "biopython-1.76/Tests/GenBank/cor6_6.gb.bgz\n",
            "biopython-1.76/Tests/GenBank/dbsource_wrap.gb\n",
            "biopython-1.76/Tests/GenBank/empty_accession.gbk\n",
            "biopython-1.76/Tests/GenBank/empty_feature_qualifier.gb\n",
            "biopython-1.76/Tests/GenBank/empty_version.gbk\n",
            "biopython-1.76/Tests/GenBank/extra_keywords.gb\n",
            "biopython-1.76/Tests/GenBank/gbvrl1_start.seq\n",
            "biopython-1.76/Tests/GenBank/invalid_locus_line_spacing.gb\n",
            "biopython-1.76/Tests/GenBank/invalid_misc_feature.gb\n",
            "biopython-1.76/Tests/GenBank/invalid_product.gb\n",
            "biopython-1.76/Tests/GenBank/iro.gb\n",
            "biopython-1.76/Tests/GenBank/negative_location.gb\n",
            "biopython-1.76/Tests/GenBank/no_end_marker.gb\n",
            "biopython-1.76/Tests/GenBank/no_origin_no_end.gb\n",
            "biopython-1.76/Tests/GenBank/noref.gb\n",
            "biopython-1.76/Tests/GenBank/one_of.gb\n",
            "biopython-1.76/Tests/GenBank/origin_line.gb\n",
            "biopython-1.76/Tests/GenBank/pBAD30.gb\n",
            "biopython-1.76/Tests/GenBank/pri1.gb\n",
            "biopython-1.76/Tests/GenBank/protein_refseq.gb\n",
            "biopython-1.76/Tests/GenBank/protein_refseq2.gb\n",
            "biopython-1.76/Tests/GenBank/qualifier_escaping_read.gb\n",
            "biopython-1.76/Tests/GenBank/qualifier_escaping_write.gb\n",
            "biopython-1.76/Tests/GenBank/tsa_acropora.gb\n",
            "biopython-1.76/Tests/GenBank/wrong_sequence_indent.gb\n",
            "biopython-1.76/Tests/Geo/\n",
            "biopython-1.76/Tests/Geo/GSE16.txt\n",
            "biopython-1.76/Tests/Geo/GSM645.txt\n",
            "biopython-1.76/Tests/Geo/GSM691.txt\n",
            "biopython-1.76/Tests/Geo/GSM700.txt\n",
            "biopython-1.76/Tests/Geo/GSM804.txt\n",
            "biopython-1.76/Tests/Geo/soft_ex_affy.txt\n",
            "biopython-1.76/Tests/Geo/soft_ex_affy_chp.txt\n",
            "biopython-1.76/Tests/Geo/soft_ex_dual.txt\n",
            "biopython-1.76/Tests/Geo/soft_ex_family.txt\n",
            "biopython-1.76/Tests/Geo/soft_ex_platform.txt\n",
            "biopython-1.76/Tests/Geo/soft_ex_series.txt\n",
            "biopython-1.76/Tests/Graphics/\n",
            "biopython-1.76/Tests/Graphics/README\n",
            "biopython-1.76/Tests/HHsuite/\n",
            "biopython-1.76/Tests/HHsuite/2uvo_hhblits.hhr\n",
            "biopython-1.76/Tests/HHsuite/2uvo_hhblits_emptytable.hhr\n",
            "biopython-1.76/Tests/HHsuite/2uvo_hhblits_onlyheader.hhr\n",
            "biopython-1.76/Tests/HHsuite/2uvo_hhsearch.hhr\n",
            "biopython-1.76/Tests/HHsuite/4p79_hhsearch_server_NOssm.hhr\n",
            "biopython-1.76/Tests/HHsuite/4y9h_hhsearch_server_NOssm.hhr\n",
            "biopython-1.76/Tests/HHsuite/README.txt\n",
            "biopython-1.76/Tests/HHsuite/allx.hhr\n",
            "biopython-1.76/Tests/HHsuite/hhpred_9590198.hhr\n",
            "biopython-1.76/Tests/HHsuite/hhsearch_q9bsu1_uniclust_w_ss_pfamA_30.hhr\n",
            "biopython-1.76/Tests/Hmmer/\n",
            "biopython-1.76/Tests/Hmmer/README.txt\n",
            "biopython-1.76/Tests/Hmmer/domtab_30_hmmscan_001.out\n",
            "biopython-1.76/Tests/Hmmer/domtab_30_hmmscan_002.out\n",
            "biopython-1.76/Tests/Hmmer/domtab_30_hmmscan_003.out\n",
            "biopython-1.76/Tests/Hmmer/domtab_30_hmmscan_004.out\n",
            "biopython-1.76/Tests/Hmmer/domtab_30_hmmsearch_001.out\n",
            "biopython-1.76/Tests/Hmmer/domtab_31b1_hmmscan_001.out\n",
            "biopython-1.76/Tests/Hmmer/domtab_31b1_hmmsearch_001.out\n",
            "biopython-1.76/Tests/Hmmer/tab_30_hmmscan_001.out\n",
            "biopython-1.76/Tests/Hmmer/tab_30_hmmscan_002.out\n",
            "biopython-1.76/Tests/Hmmer/tab_30_hmmscan_003.out\n",
            "biopython-1.76/Tests/Hmmer/tab_30_hmmscan_004.out\n",
            "biopython-1.76/Tests/Hmmer/tab_31b1_hmmscan_001.out\n",
            "biopython-1.76/Tests/Hmmer/tab_31b1_hmmsearch_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_20_hmmsearch_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_21_hmmpfam_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_22_hmmpfam_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_22_hmmsearch_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_23_hmmpfam_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_23_hmmpfam_002.out\n",
            "biopython-1.76/Tests/Hmmer/text_23_hmmpfam_003.out\n",
            "biopython-1.76/Tests/Hmmer/text_23_hmmpfam_004.out\n",
            "biopython-1.76/Tests/Hmmer/text_24_hmmpfam_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmscan_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmscan_002.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmscan_003.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmscan_004.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmscan_005.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmscan_006.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmscan_007.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmscan_008.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmscan_009.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmscan_010.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmsearch_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmsearch_002.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmsearch_003.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmsearch_004.out\n",
            "biopython-1.76/Tests/Hmmer/text_30_hmmsearch_005.out\n",
            "biopython-1.76/Tests/Hmmer/text_31b1_hmmscan_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_31b1_hmmsearch_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_31b2_hmmscan_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_31b2_hmmsearch_001.out\n",
            "biopython-1.76/Tests/Hmmer/text_31b2_hmmsearch_002.out\n",
            "biopython-1.76/Tests/Hmmer/text_31b2_phmmer_001.out\n",
            "biopython-1.76/Tests/IntelliGenetics/\n",
            "biopython-1.76/Tests/IntelliGenetics/TAT_mase_nuc.txt\n",
            "biopython-1.76/Tests/IntelliGenetics/VIF_mase-pro.txt\n",
            "biopython-1.76/Tests/IntelliGenetics/vpu_nucaligned.txt\n",
            "biopython-1.76/Tests/InterProScan/\n",
            "biopython-1.76/Tests/InterProScan/test_001.xml\n",
            "biopython-1.76/Tests/KEGG/\n",
            "biopython-1.76/Tests/KEGG/compound.irregular\n",
            "biopython-1.76/Tests/KEGG/compound.sample\n",
            "biopython-1.76/Tests/KEGG/enzyme.4letter\n",
            "biopython-1.76/Tests/KEGG/enzyme.irregular\n",
            "biopython-1.76/Tests/KEGG/enzyme.new\n",
            "biopython-1.76/Tests/KEGG/enzyme.sample\n",
            "biopython-1.76/Tests/KEGG/gene.sample\n",
            "biopython-1.76/Tests/KEGG/ko00010.xml\n",
            "biopython-1.76/Tests/KEGG/ko00253.xml\n",
            "biopython-1.76/Tests/KEGG/ko01100.xml\n",
            "biopython-1.76/Tests/KEGG/ko03070.xml\n",
            "biopython-1.76/Tests/KEGG/map00950.rea\n",
            "biopython-1.76/Tests/KEGG/map01100.png\n",
            "biopython-1.76/Tests/KEGG/map03070.png\n",
            "biopython-1.76/Tests/MAF/\n",
            "biopython-1.76/Tests/MAF/bug2453.maf\n",
            "biopython-1.76/Tests/MAF/bundle_without_target.maf\n",
            "biopython-1.76/Tests/MAF/cnksr3.fa\n",
            "biopython-1.76/Tests/MAF/corrupt.idx\n",
            "biopython-1.76/Tests/MAF/humor.maf\n",
            "biopython-1.76/Tests/MAF/invalid.idx\n",
            "biopython-1.76/Tests/MAF/length_coords_mismatch.maf\n",
            "biopython-1.76/Tests/MAF/ucsc_mm9_chr10.maf\n",
            "biopython-1.76/Tests/MAF/ucsc_mm9_chr10.mafindex\n",
            "biopython-1.76/Tests/MAF/ucsc_mm9_chr10_bad.maf\n",
            "biopython-1.76/Tests/MAF/ucsc_mm9_chr10_bad.mafindex\n",
            "biopython-1.76/Tests/MAF/ucsc_mm9_chr10_big.maf\n",
            "biopython-1.76/Tests/MAF/ucsc_mm9_chr10_big.mafindex\n",
            "biopython-1.76/Tests/MAF/ucsc_test.maf\n",
            "biopython-1.76/Tests/MAF/unfinished.idx\n",
            "biopython-1.76/Tests/MAF/wrong_version.idx\n",
            "biopython-1.76/Tests/Mauve/\n",
            "biopython-1.76/Tests/Mauve/simple.fa\n",
            "biopython-1.76/Tests/Mauve/simple.xmfa\n",
            "biopython-1.76/Tests/Mauve/simple.xmfa.backbone\n",
            "biopython-1.76/Tests/Mauve/simple.xmfa.bbcols\n",
            "biopython-1.76/Tests/Mauve/simple_short.xmfa\n",
            "biopython-1.76/Tests/Medline/\n",
            "biopython-1.76/Tests/Medline/pubmed_result1.txt\n",
            "biopython-1.76/Tests/Medline/pubmed_result2.txt\n",
            "biopython-1.76/Tests/Medline/pubmed_result3.txt\n",
            "biopython-1.76/Tests/MetaTool/\n",
            "biopython-1.76/Tests/MetaTool/Meta3.out\n",
            "biopython-1.76/Tests/MetaTool/Meta4.out\n",
            "biopython-1.76/Tests/MetaTool/Meta6.out\n",
            "biopython-1.76/Tests/MetaTool/Meta7.out\n",
            "biopython-1.76/Tests/MetaTool/Meta8.out\n",
            "biopython-1.76/Tests/MetaTool/meta.out\n",
            "biopython-1.76/Tests/MetaTool/meta2.out\n",
            "biopython-1.76/Tests/MetaTool/meta9.out\n",
            "biopython-1.76/Tests/NBRF/\n",
            "biopython-1.76/Tests/NBRF/B_nuc.pir\n",
            "biopython-1.76/Tests/NBRF/Cw_prot.pir\n",
            "biopython-1.76/Tests/NBRF/DMA_nuc.pir\n",
            "biopython-1.76/Tests/NBRF/DMB_prot.pir\n",
            "biopython-1.76/Tests/NBRF/clustalw.pir\n",
            "biopython-1.76/Tests/NMR/\n",
            "biopython-1.76/Tests/NMR/noed.xpk\n",
            "biopython-1.76/Tests/NMR/out_example.xpk\n",
            "biopython-1.76/Tests/NeXML/\n",
            "biopython-1.76/Tests/NeXML/characters.xml\n",
            "biopython-1.76/Tests/NeXML/edgelabels.xml\n",
            "biopython-1.76/Tests/NeXML/meta_taxa.xml\n",
            "biopython-1.76/Tests/NeXML/meta_types.xml\n",
            "biopython-1.76/Tests/NeXML/nexml.xml\n",
            "biopython-1.76/Tests/NeXML/phenoscape.xml\n",
            "biopython-1.76/Tests/NeXML/sets.xml\n",
            "biopython-1.76/Tests/NeXML/taxa.xml\n",
            "biopython-1.76/Tests/NeXML/timetree.xml\n",
            "biopython-1.76/Tests/NeXML/tolweb.xml\n",
            "biopython-1.76/Tests/NeXML/treebase-record.xml\n",
            "biopython-1.76/Tests/NeXML/trees-uris.xml\n",
            "biopython-1.76/Tests/NeXML/trees.xml\n",
            "biopython-1.76/Tests/NeuralNetwork/\n",
            "biopython-1.76/Tests/NeuralNetwork/enolase.fasta\n",
            "biopython-1.76/Tests/NeuralNetwork/repeat.fasta\n",
            "biopython-1.76/Tests/Nexus/\n",
            "biopython-1.76/Tests/Nexus/ByteOrderMarkFile.nwk\n",
            "biopython-1.76/Tests/Nexus/bats.nex\n",
            "biopython-1.76/Tests/Nexus/codonposset.nex\n",
            "biopython-1.76/Tests/Nexus/int_node_labels.nwk\n",
            "biopython-1.76/Tests/Nexus/test.new\n",
            "biopython-1.76/Tests/Nexus/test_Nexus_input.nex\n",
            "biopython-1.76/Tests/Nexus/vSysLab_Ganaspidium_multistate.nex\n",
            "biopython-1.76/Tests/Nexus/vSysLab_Heptascelio_no-states_10+chars.nex\n",
            "biopython-1.76/Tests/Nexus/vSysLab_Oreiscelio_discrete+continuous.nex\n",
            "biopython-1.76/Tests/Nib/\n",
            "biopython-1.76/Tests/Nib/test_even.fa\n",
            "biopython-1.76/Tests/Nib/test_even_bigendian.nib\n",
            "biopython-1.76/Tests/Nib/test_even_littleendian.nib\n",
            "biopython-1.76/Tests/Nib/test_odd.fa\n",
            "biopython-1.76/Tests/Nib/test_odd_bigendian.nib\n",
            "biopython-1.76/Tests/Nib/test_odd_littleendian.nib\n",
            "biopython-1.76/Tests/PAML/\n",
            "biopython-1.76/Tests/PAML/Alignments/\n",
            "biopython-1.76/Tests/PAML/Alignments/aa_alignment.phylip\n",
            "biopython-1.76/Tests/PAML/Alignments/alignment.phylip\n",
            "biopython-1.76/Tests/PAML/Alignments/dottednames.phylip\n",
            "biopython-1.76/Tests/PAML/Alignments/dottednum.fasta\n",
            "biopython-1.76/Tests/PAML/Alignments/longnames.fasta\n",
            "biopython-1.76/Tests/PAML/Alignments/lysinYangSwanson2002.nuc\n",
            "biopython-1.76/Tests/PAML/Control_files/\n",
            "biopython-1.76/Tests/PAML/Control_files/README.txt\n",
            "biopython-1.76/Tests/PAML/Control_files/bad1.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/bad2.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/bad3.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/SE.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/alpha1rho1.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/baseml.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/model0.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/model1.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/model2.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/model3.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/model4.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/model5.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/model6.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/model7.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/model8.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/nhomo1.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/nhomo3.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/nhomo4.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/nparK1.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/nparK2.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/nparK3.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml/nparK4.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/baseml.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/NSsite3.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/SE.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/aa_model0.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/aa_pairwise.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/all_NSsites.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/branchsiteA.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/clademodelC.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/codeml.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/freeratio.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/m2a_rel.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/ngene2_mgene02.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/ngene2_mgene1.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/ngene2_mgene34.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/codeml/pairwise.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/species.tree\n",
            "biopython-1.76/Tests/PAML/Control_files/yn00/\n",
            "biopython-1.76/Tests/PAML/Control_files/yn00/yn00.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/yn00/yn00_dotted.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/yn00/yn00_dottednum.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/yn00/yn00_long.ctl\n",
            "biopython-1.76/Tests/PAML/Control_files/yn00.ctl\n",
            "biopython-1.76/Tests/PAML/Results/\n",
            "biopython-1.76/Tests/PAML/Results/bad_results.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/\n",
            "biopython-1.76/Tests/PAML/Results/baseml/SE/\n",
            "biopython-1.76/Tests/PAML/Results/baseml/SE/SE-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/SE/SE-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/SE/SE-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/SE/SE-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/SE/SE-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/SE/SE-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/SE/SE-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/alpha1rho1/\n",
            "biopython-1.76/Tests/PAML/Results/baseml/alpha1rho1/alpha1rho1-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/alpha1rho1/alpha1rho1-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/alpha1rho1/alpha1rho1-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/baseml_SE.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/baseml_alpha1rho1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model0-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model0-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model0-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model0-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model0-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model0-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model0-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model1-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model1-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model1-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model1-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model1-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model1-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model1-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model2-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model2-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model2-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model2-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model2-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model2-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model2-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model3-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model3-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model3-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model3-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model3-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model3-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model3-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model4-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model4-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model4-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model4-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model4-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model4-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model4-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model5-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model5-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model5-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model5-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model5-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model5-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model5-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model6-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model6-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model6-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model6-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model6-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model6-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model6-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model7-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model7-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model7-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model7-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model7-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model7-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model7-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model8-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model8-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model8-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model8-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model8-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model8-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/model/model8-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo1-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo1-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo1-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo1-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo1-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo1-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo1-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo3-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo3-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo3-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo3-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo3-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo3-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo3-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo4-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo4-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo4-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo4-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo4-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo4-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nhomo/nhomo4-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/nparK1-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/nparK1-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/nparK1-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/nparK2-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/nparK2-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/nparK2-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/nparK3-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/nparK3-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/nparK3-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/nparK4-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/nparK4-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/nparK/nparK4-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/versions/\n",
            "biopython-1.76/Tests/PAML/Results/baseml/versions/baseml41.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/versions/baseml43.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/versions/baseml44.out\n",
            "biopython-1.76/Tests/PAML/Results/baseml/versions/baseml44c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/NSsite3/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/NSsite3/NSsite3-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/NSsite3/NSsite3-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/NSsite3/NSsite3-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/NSsite3/NSsite3-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/NSsite3/NSsite3-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/SE/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/SE/SE-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/SE/SE-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/SE/SE-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/SE/SE-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/SE/SE-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/SE/SE-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/SE/SE-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_model0/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_model0/aa_model0-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_model0/aa_model0-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_model0/aa_model0-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_model0/aa_model0-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_model0/aa_model0-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_model0/aa_model0-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_model0/aa_model0-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_pairwise/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_pairwise/aa_pairwise-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_pairwise/aa_pairwise-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_pairwise/aa_pairwise-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_pairwise/aa_pairwise-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_pairwise/aa_pairwise-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_pairwise/aa_pairwise-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/aa_pairwise/aa_pairwise-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/all_NSsites/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/all_NSsites/all_NSsites-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/all_NSsites/all_NSsites-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/all_NSsites/all_NSsites-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/all_NSsites/all_NSsites-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/all_NSsites/all_NSsites-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/all_NSsites/all_NSsites-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/all_NSsites/all_NSsites-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/branchsiteA/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/branchsiteA/branchsiteA-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/branchsiteA/branchsiteA-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/branchsiteA/branchsiteA-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/branchsiteA/branchsiteA-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/branchsiteA/branchsiteA-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/branchsiteA/branchsiteA-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/branchsiteA/branchsiteA-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/clademodelC/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/clademodelC/clademodelC-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/clademodelC/clademodelC-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/clademodelC/clademodelC-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/clademodelC/clademodelC-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/clademodelC/clademodelC-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/clademodelC/clademodelC-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/clademodelC/clademodelC-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/freeratio/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/freeratio/freeratio-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/freeratio/freeratio-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/freeratio/freeratio-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/freeratio/freeratio-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/freeratio/freeratio-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/freeratio/freeratio-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/freeratio/freeratio-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/m2a_rel/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/m2a_rel/m2a_rel-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/m2a_rel/m2a_rel-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/m2a_rel/m2a_rel-4_8.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/m2a_rel/m2a_rel-4_9a.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene02/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene02/ngene2_mgene02-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene02/ngene2_mgene02-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene02/ngene2_mgene02-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene02/ngene2_mgene02-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene02/ngene2_mgene02-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene02/ngene2_mgene02-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene02/ngene2_mgene02-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene1/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene1/ngene2_mgene1-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene1/ngene2_mgene1-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene1/ngene2_mgene1-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene1/ngene2_mgene1-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene34/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene34/ngene2_mgene34-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene34/ngene2_mgene34-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene34/ngene2_mgene34-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene34/ngene2_mgene34-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene34/ngene2_mgene34-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene34/ngene2_mgene34-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/ngene2_mgene34/ngene2_mgene34-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/pairwise/\n",
            "biopython-1.76/Tests/PAML/Results/codeml/pairwise/pairwise-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/pairwise/pairwise-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/pairwise/pairwise-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/pairwise/pairwise-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/pairwise/pairwise-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/pairwise/pairwise-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/pairwise/pairwise-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/codeml/tree_regexp_versatility.out\n",
            "biopython-1.76/Tests/PAML/Results/yn00/\n",
            "biopython-1.76/Tests/PAML/Results/yn00/yn00-4_1.out\n",
            "biopython-1.76/Tests/PAML/Results/yn00/yn00-4_3.out\n",
            "biopython-1.76/Tests/PAML/Results/yn00/yn00-4_4.out\n",
            "biopython-1.76/Tests/PAML/Results/yn00/yn00-4_4c.out\n",
            "biopython-1.76/Tests/PAML/Results/yn00/yn00-4_5.out\n",
            "biopython-1.76/Tests/PAML/Results/yn00/yn00-4_6.out\n",
            "biopython-1.76/Tests/PAML/Results/yn00/yn00-4_7.out\n",
            "biopython-1.76/Tests/PAML/Results/yn00/yn00-4_8.out\n",
            "biopython-1.76/Tests/PAML/Results/yn00/yn00_dotted-4_9i.out\n",
            "biopython-1.76/Tests/PAML/Results/yn00/yn00_dottednum-4_9i.out\n",
            "biopython-1.76/Tests/PAML/Results/yn00/yn00_long-4_8.out\n",
            "biopython-1.76/Tests/PAML/Trees/\n",
            "biopython-1.76/Tests/PAML/Trees/bad.tree\n",
            "biopython-1.76/Tests/PAML/Trees/lysin.trees\n",
            "biopython-1.76/Tests/PAML/Trees/lysin.trees.4.6\n",
            "biopython-1.76/Tests/PAML/Trees/species.tree\n",
            "biopython-1.76/Tests/PAML/gen_results.py\n",
            "biopython-1.76/Tests/PDB/\n",
            "biopython-1.76/Tests/PDB/1A8O.asa\n",
            "biopython-1.76/Tests/PDB/1A8O.cif\n",
            "biopython-1.76/Tests/PDB/1A8O.mmtf\n",
            "biopython-1.76/Tests/PDB/1A8O.pdb\n",
            "biopython-1.76/Tests/PDB/1A8O.rsa\n",
            "biopython-1.76/Tests/PDB/1A8O.xyzr\n",
            "biopython-1.76/Tests/PDB/1AS5.cif\n",
            "biopython-1.76/Tests/PDB/1LCD.cif\n",
            "biopython-1.76/Tests/PDB/1LCD.pdb\n",
            "biopython-1.76/Tests/PDB/1MOM_min.cif\n",
            "biopython-1.76/Tests/PDB/1SSU_mod.cif\n",
            "biopython-1.76/Tests/PDB/1SSU_mod.pdb\n",
            "biopython-1.76/Tests/PDB/2BEG.cif\n",
            "biopython-1.76/Tests/PDB/2BEG.dssp\n",
            "biopython-1.76/Tests/PDB/2BEG.pdb\n",
            "biopython-1.76/Tests/PDB/2BEG_noheader.dssp\n",
            "biopython-1.76/Tests/PDB/2OFG.cif\n",
            "biopython-1.76/Tests/PDB/2XHE.cif\n",
            "biopython-1.76/Tests/PDB/2XHE.pdb\n",
            "biopython-1.76/Tests/PDB/2n0n_M1.pdb\n",
            "biopython-1.76/Tests/PDB/3JQH.cif\n",
            "biopython-1.76/Tests/PDB/4CUP.cif\n",
            "biopython-1.76/Tests/PDB/4CUP.mmtf\n",
            "biopython-1.76/Tests/PDB/4Q9R_min.cif\n",
            "biopython-1.76/Tests/PDB/4ZHL.cif\n",
            "biopython-1.76/Tests/PDB/4ZHL.mmtf\n",
            "biopython-1.76/Tests/PDB/Miller_RASA.txt\n",
            "biopython-1.76/Tests/PDB/Sander_RASA.txt\n",
            "biopython-1.76/Tests/PDB/Wilke_RASA.txt\n",
            "biopython-1.76/Tests/PDB/a_structure.cif\n",
            "biopython-1.76/Tests/PDB/a_structure.pdb\n",
            "biopython-1.76/Tests/PDB/alignment_file.fa\n",
            "biopython-1.76/Tests/PDB/d256ba_.ent\n",
            "biopython-1.76/Tests/PDB/dssp_xtra_Sander.txt\n",
            "biopython-1.76/Tests/PDB/header.pdb\n",
            "biopython-1.76/Tests/PDB/ions.pdb\n",
            "biopython-1.76/Tests/PDB/lib_10_z_5.txt\n",
            "biopython-1.76/Tests/PDB/occupancy.pdb\n",
            "biopython-1.76/Tests/Phd/\n",
            "biopython-1.76/Tests/Phd/phd1\n",
            "biopython-1.76/Tests/Phd/phd2\n",
            "biopython-1.76/Tests/Phd/phd_454\n",
            "biopython-1.76/Tests/Phd/phd_solexa\n",
            "biopython-1.76/Tests/Phylip/\n",
            "biopython-1.76/Tests/Phylip/hennigian.phy\n",
            "biopython-1.76/Tests/Phylip/horses.phy\n",
            "biopython-1.76/Tests/Phylip/horses.tree\n",
            "biopython-1.76/Tests/Phylip/interlaced.phy\n",
            "biopython-1.76/Tests/Phylip/interlaced2.phy\n",
            "biopython-1.76/Tests/Phylip/random.phy\n",
            "biopython-1.76/Tests/Phylip/reference_dna.phy\n",
            "biopython-1.76/Tests/Phylip/reference_dna2.phy\n",
            "biopython-1.76/Tests/Phylip/sequential.phy\n",
            "biopython-1.76/Tests/Phylip/sequential2.phy\n",
            "biopython-1.76/Tests/PhyloXML/\n",
            "biopython-1.76/Tests/PhyloXML/apaf.xml\n",
            "biopython-1.76/Tests/PhyloXML/bcl_2.xml\n",
            "biopython-1.76/Tests/PhyloXML/distribution.xml\n",
            "biopython-1.76/Tests/PhyloXML/example.xml\n",
            "biopython-1.76/Tests/PhyloXML/made_up.xml\n",
            "biopython-1.76/Tests/PhyloXML/o_tol_332_d_dollo.xml\n",
            "biopython-1.76/Tests/PhyloXML/phyloxml_examples.xml\n",
            "biopython-1.76/Tests/PopGen/\n",
            "biopython-1.76/Tests/PopGen/README\n",
            "biopython-1.76/Tests/PopGen/big.gen\n",
            "biopython-1.76/Tests/PopGen/c2line.gen\n",
            "biopython-1.76/Tests/PopGen/c2space.gen\n",
            "biopython-1.76/Tests/PopGen/c3line.gen\n",
            "biopython-1.76/Tests/PopGen/c3space.gen\n",
            "biopython-1.76/Tests/PopGen/haplo.gen\n",
            "biopython-1.76/Tests/PopGen/haplo2.gen\n",
            "biopython-1.76/Tests/PopGen/haplo3.gen\n",
            "biopython-1.76/Tests/Prosite/\n",
            "biopython-1.76/Tests/Prosite/Doc/\n",
            "biopython-1.76/Tests/Prosite/Doc/pdoc00100.txt\n",
            "biopython-1.76/Tests/Prosite/Doc/pdoc00113.txt\n",
            "biopython-1.76/Tests/Prosite/Doc/pdoc00144.txt\n",
            "biopython-1.76/Tests/Prosite/Doc/pdoc00149.txt\n",
            "biopython-1.76/Tests/Prosite/Doc/pdoc00340.txt\n",
            "biopython-1.76/Tests/Prosite/Doc/pdoc00424.txt\n",
            "biopython-1.76/Tests/Prosite/Doc/pdoc00472.txt\n",
            "biopython-1.76/Tests/Prosite/Doc/pdoc00640.txt\n",
            "biopython-1.76/Tests/Prosite/Doc/pdoc00787.txt\n",
            "biopython-1.76/Tests/Prosite/Doc/pdoc00933.txt\n",
            "biopython-1.76/Tests/Prosite/Doc/prosite.excerpt.doc\n",
            "biopython-1.76/Tests/Prosite/ps00107.txt\n",
            "biopython-1.76/Tests/Prosite/ps00159.txt\n",
            "biopython-1.76/Tests/Prosite/ps00165.txt\n",
            "biopython-1.76/Tests/Prosite/ps00432.txt\n",
            "biopython-1.76/Tests/Prosite/ps00488.txt\n",
            "biopython-1.76/Tests/Prosite/ps00546.txt\n",
            "biopython-1.76/Tests/Quality/\n",
            "biopython-1.76/Tests/Quality/error_diff_ids.fastq\n",
            "biopython-1.76/Tests/Quality/error_double_qual.fastq\n",
            "biopython-1.76/Tests/Quality/error_double_seq.fastq\n",
            "biopython-1.76/Tests/Quality/error_long_qual.fastq\n",
            "biopython-1.76/Tests/Quality/error_no_qual.fastq\n",
            "biopython-1.76/Tests/Quality/error_qual_del.fastq\n",
            "biopython-1.76/Tests/Quality/error_qual_escape.fastq\n",
            "biopython-1.76/Tests/Quality/error_qual_null.fastq\n",
            "biopython-1.76/Tests/Quality/error_qual_space.fastq\n",
            "biopython-1.76/Tests/Quality/error_qual_tab.fastq\n",
            "biopython-1.76/Tests/Quality/error_qual_unit_sep.fastq\n",
            "biopython-1.76/Tests/Quality/error_qual_vtab.fastq\n",
            "biopython-1.76/Tests/Quality/error_short_qual.fastq\n",
            "biopython-1.76/Tests/Quality/error_spaces.fastq\n",
            "biopython-1.76/Tests/Quality/error_tabs.fastq\n",
            "biopython-1.76/Tests/Quality/error_trunc_at_plus.fastq\n",
            "biopython-1.76/Tests/Quality/error_trunc_at_qual.fastq\n",
            "biopython-1.76/Tests/Quality/error_trunc_at_seq.fastq\n",
            "biopython-1.76/Tests/Quality/error_trunc_in_plus.fastq\n",
            "biopython-1.76/Tests/Quality/error_trunc_in_qual.fastq\n",
            "biopython-1.76/Tests/Quality/error_trunc_in_seq.fastq\n",
            "biopython-1.76/Tests/Quality/error_trunc_in_title.fastq\n",
            "biopython-1.76/Tests/Quality/example.fasta\n",
            "biopython-1.76/Tests/Quality/example.fastq\n",
            "biopython-1.76/Tests/Quality/example.fastq.bgz\n",
            "biopython-1.76/Tests/Quality/example.fastq.gz\n",
            "biopython-1.76/Tests/Quality/example.qual\n",
            "biopython-1.76/Tests/Quality/example_dos.fastq\n",
            "biopython-1.76/Tests/Quality/example_dos.fastq.bgz\n",
            "biopython-1.76/Tests/Quality/illumina_faked.fastq\n",
            "biopython-1.76/Tests/Quality/illumina_full_range_as_illumina.fastq\n",
            "biopython-1.76/Tests/Quality/illumina_full_range_as_sanger.fastq\n",
            "biopython-1.76/Tests/Quality/illumina_full_range_as_solexa.fastq\n",
            "biopython-1.76/Tests/Quality/illumina_full_range_original_illumina.fastq\n",
            "biopython-1.76/Tests/Quality/longreads_as_illumina.fastq\n",
            "biopython-1.76/Tests/Quality/longreads_as_sanger.fastq\n",
            "biopython-1.76/Tests/Quality/longreads_as_solexa.fastq\n",
            "biopython-1.76/Tests/Quality/longreads_original_sanger.fastq\n",
            "biopython-1.76/Tests/Quality/misc_dna_as_illumina.fastq\n",
            "biopython-1.76/Tests/Quality/misc_dna_as_sanger.fastq\n",
            "biopython-1.76/Tests/Quality/misc_dna_as_solexa.fastq\n",
            "biopython-1.76/Tests/Quality/misc_dna_original_sanger.fastq\n",
            "biopython-1.76/Tests/Quality/misc_rna_as_illumina.fastq\n",
            "biopython-1.76/Tests/Quality/misc_rna_as_sanger.fastq\n",
            "biopython-1.76/Tests/Quality/misc_rna_as_solexa.fastq\n",
            "biopython-1.76/Tests/Quality/misc_rna_original_sanger.fastq\n",
            "biopython-1.76/Tests/Quality/sanger_93.fastq\n",
            "biopython-1.76/Tests/Quality/sanger_faked.fastq\n",
            "biopython-1.76/Tests/Quality/sanger_full_range_as_illumina.fastq\n",
            "biopython-1.76/Tests/Quality/sanger_full_range_as_sanger.fastq\n",
            "biopython-1.76/Tests/Quality/sanger_full_range_as_solexa.fastq\n",
            "biopython-1.76/Tests/Quality/sanger_full_range_original_sanger.fastq\n",
            "biopython-1.76/Tests/Quality/solexa_example.fastq\n",
            "biopython-1.76/Tests/Quality/solexa_faked.fastq\n",
            "biopython-1.76/Tests/Quality/solexa_full_range_as_illumina.fastq\n",
            "biopython-1.76/Tests/Quality/solexa_full_range_as_sanger.fastq\n",
            "biopython-1.76/Tests/Quality/solexa_full_range_as_solexa.fastq\n",
            "biopython-1.76/Tests/Quality/solexa_full_range_original_solexa.fastq\n",
            "biopython-1.76/Tests/Quality/tricky.fastq\n",
            "biopython-1.76/Tests/Quality/wrapping_as_illumina.fastq\n",
            "biopython-1.76/Tests/Quality/wrapping_as_sanger.fastq\n",
            "biopython-1.76/Tests/Quality/wrapping_as_solexa.fastq\n",
            "biopython-1.76/Tests/Quality/wrapping_original_sanger.fastq\n",
            "biopython-1.76/Tests/Quality/zero_length.fastq\n",
            "biopython-1.76/Tests/Rebase/\n",
            "biopython-1.76/Tests/Rebase/bamii.htm\n",
            "biopython-1.76/Tests/Rebase/bamii.txt\n",
            "biopython-1.76/Tests/Rebase/cac81.htm\n",
            "biopython-1.76/Tests/Rebase/cac81.txt\n",
            "biopython-1.76/Tests/Rebase/crei.htm\n",
            "biopython-1.76/Tests/Rebase/crei.txt\n",
            "biopython-1.76/Tests/Rebase/foki.htm\n",
            "biopython-1.76/Tests/Rebase/foki.txt\n",
            "biopython-1.76/Tests/Rebase/pvuii.htm\n",
            "biopython-1.76/Tests/Rebase/pvuii.txt\n",
            "biopython-1.76/Tests/Rebase/taqi.htm\n",
            "biopython-1.76/Tests/Rebase/taqi.txt\n",
            "biopython-1.76/Tests/Registry/\n",
            "biopython-1.76/Tests/Registry/EDD_RAT.dat\n",
            "biopython-1.76/Tests/Registry/bt001\n",
            "biopython-1.76/Tests/Registry/seqs.fasta\n",
            "biopython-1.76/Tests/Roche/\n",
            "biopython-1.76/Tests/Roche/E3MFGYR02_alt_index_at_end.sff\n",
            "biopython-1.76/Tests/Roche/E3MFGYR02_alt_index_at_start.sff\n",
            "biopython-1.76/Tests/Roche/E3MFGYR02_alt_index_in_middle.sff\n",
            "biopython-1.76/Tests/Roche/E3MFGYR02_index_at_start.sff\n",
            "biopython-1.76/Tests/Roche/E3MFGYR02_index_in_middle.sff\n",
            "biopython-1.76/Tests/Roche/E3MFGYR02_no_manifest.sff\n",
            "biopython-1.76/Tests/Roche/E3MFGYR02_random_10_reads.fasta\n",
            "biopython-1.76/Tests/Roche/E3MFGYR02_random_10_reads.qual\n",
            "biopython-1.76/Tests/Roche/E3MFGYR02_random_10_reads.sff\n",
            "biopython-1.76/Tests/Roche/E3MFGYR02_random_10_reads.sff.gz\n",
            "biopython-1.76/Tests/Roche/E3MFGYR02_random_10_reads_no_trim.fasta\n",
            "biopython-1.76/Tests/Roche/E3MFGYR02_random_10_reads_no_trim.qual\n",
            "biopython-1.76/Tests/Roche/README.txt\n",
            "biopython-1.76/Tests/Roche/greek.sff\n",
            "biopython-1.76/Tests/Roche/invalid_greek_E3MFGYR02.sff\n",
            "biopython-1.76/Tests/Roche/invalid_paired_E3MFGYR02.sff\n",
            "biopython-1.76/Tests/Roche/paired.sff\n",
            "biopython-1.76/Tests/Roche/triple_sff.idx\n",
            "biopython-1.76/Tests/Roche/triple_sff_rel_paths.idx\n",
            "biopython-1.76/Tests/SCOP/\n",
            "biopython-1.76/Tests/SCOP/dir.cla.scop.txt_test\n",
            "biopython-1.76/Tests/SCOP/dir.des.scop.txt_test\n",
            "biopython-1.76/Tests/SCOP/dir.hie.scop.txt_test\n",
            "biopython-1.76/Tests/SCOP/raftest.txt\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-all-test.fa\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-bib-10-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-bib-100-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-bib-20-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-bib-25-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-bib-30-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-bib-35-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-bib-40-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-bib-50-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-bib-70-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-bib-90-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-bib-95-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e+0,7-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e+0-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e+1-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-0,3-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-1,3-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-1-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-10-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-15-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-2,3-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-2-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-20-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-25-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-3-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-4-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-5-test.id\n",
            "biopython-1.76/Tests/SCOP/scopseq-test/astral-scopdom-seqres-sel-gs-e100m-e-50-test.id\n",
            "biopython-1.76/Tests/SCOP/testDom.txt\n",
            "biopython-1.76/Tests/Saf/\n",
            "biopython-1.76/Tests/Saf/saf1.txt\n",
            "biopython-1.76/Tests/SamBam/\n",
            "biopython-1.76/Tests/SamBam/bam1.bam\n",
            "biopython-1.76/Tests/SamBam/bam1_sorted.bam\n",
            "biopython-1.76/Tests/SamBam/bam2.bam\n",
            "biopython-1.76/Tests/SamBam/bam2_sorted.bam\n",
            "biopython-1.76/Tests/SamBam/ex1.bam\n",
            "biopython-1.76/Tests/SamBam/ex1.fa\n",
            "biopython-1.76/Tests/SamBam/ex1.sam\n",
            "biopython-1.76/Tests/SamBam/ex1_header.bam\n",
            "biopython-1.76/Tests/SamBam/ex1_header.sam\n",
            "biopython-1.76/Tests/SamBam/ex1_refresh.bam\n",
            "biopython-1.76/Tests/SamBam/readme.txt\n",
            "biopython-1.76/Tests/SamBam/sam1.sam\n",
            "biopython-1.76/Tests/SamBam/sam2.sam\n",
            "biopython-1.76/Tests/SeqXML/\n",
            "biopython-1.76/Tests/SeqXML/corrupt_example1.xml\n",
            "biopython-1.76/Tests/SeqXML/corrupt_example2.xml\n",
            "biopython-1.76/Tests/SeqXML/dna_example.xml\n",
            "biopython-1.76/Tests/SeqXML/global_species_example.xml\n",
            "biopython-1.76/Tests/SeqXML/protein_example.xml\n",
            "biopython-1.76/Tests/SeqXML/rna_example.xml\n",
            "biopython-1.76/Tests/SnapGene/\n",
            "biopython-1.76/Tests/SnapGene/sample-d.dna\n",
            "biopython-1.76/Tests/SnapGene/sample-e.dna\n",
            "biopython-1.76/Tests/Stockholm/\n",
            "biopython-1.76/Tests/Stockholm/funny.sth\n",
            "biopython-1.76/Tests/Stockholm/simple.sth\n",
            "biopython-1.76/Tests/SubsMat/\n",
            "biopython-1.76/Tests/SubsMat/aaDistrib.txt\n",
            "biopython-1.76/Tests/SubsMat/acc_rep_mat.pik\n",
            "biopython-1.76/Tests/SubsMat/protein_count.txt\n",
            "biopython-1.76/Tests/SubsMat/protein_freq.txt\n",
            "biopython-1.76/Tests/SubsMat/redAADistrib.txt\n",
            "biopython-1.76/Tests/SubsMat/redAltAADistrib.txt\n",
            "biopython-1.76/Tests/SwissProt/\n",
            "biopython-1.76/Tests/SwissProt/F2CXE6.txt\n",
            "biopython-1.76/Tests/SwissProt/F2CXE6.xml\n",
            "biopython-1.76/Tests/SwissProt/H2CNN8.txt\n",
            "biopython-1.76/Tests/SwissProt/H2CNN8.xml\n",
            "biopython-1.76/Tests/SwissProt/P84001.xml\n",
            "biopython-1.76/Tests/SwissProt/P97881.xml\n",
            "biopython-1.76/Tests/SwissProt/Q13639.txt\n",
            "biopython-1.76/Tests/SwissProt/Q13639.xml\n",
            "biopython-1.76/Tests/SwissProt/R5HY77.xml\n",
            "biopython-1.76/Tests/SwissProt/README\n",
            "biopython-1.76/Tests/SwissProt/keywlist.txt\n",
            "biopython-1.76/Tests/SwissProt/keywlist2.txt\n",
            "biopython-1.76/Tests/SwissProt/multi_ex.fasta\n",
            "biopython-1.76/Tests/SwissProt/multi_ex.gff\n",
            "biopython-1.76/Tests/SwissProt/multi_ex.list\n",
            "biopython-1.76/Tests/SwissProt/multi_ex.rdf\n",
            "biopython-1.76/Tests/SwissProt/multi_ex.txt\n",
            "biopython-1.76/Tests/SwissProt/multi_ex.xml\n",
            "biopython-1.76/Tests/SwissProt/sp001\n",
            "biopython-1.76/Tests/SwissProt/sp002\n",
            "biopython-1.76/Tests/SwissProt/sp003\n",
            "biopython-1.76/Tests/SwissProt/sp004\n",
            "biopython-1.76/Tests/SwissProt/sp005\n",
            "biopython-1.76/Tests/SwissProt/sp006\n",
            "biopython-1.76/Tests/SwissProt/sp007\n",
            "biopython-1.76/Tests/SwissProt/sp008\n",
            "biopython-1.76/Tests/SwissProt/sp009\n",
            "biopython-1.76/Tests/SwissProt/sp010\n",
            "biopython-1.76/Tests/SwissProt/sp011\n",
            "biopython-1.76/Tests/SwissProt/sp012\n",
            "biopython-1.76/Tests/SwissProt/sp013\n",
            "biopython-1.76/Tests/SwissProt/sp014\n",
            "biopython-1.76/Tests/SwissProt/sp015\n",
            "biopython-1.76/Tests/SwissProt/sp016\n",
            "biopython-1.76/Tests/SwissProt/uni001\n",
            "biopython-1.76/Tests/SwissProt/uni002\n",
            "biopython-1.76/Tests/SwissProt/uni003\n",
            "biopython-1.76/Tests/TreeConstruction/\n",
            "biopython-1.76/Tests/TreeConstruction/adam_refs.tre\n",
            "biopython-1.76/Tests/TreeConstruction/consensus_refs.tre\n",
            "biopython-1.76/Tests/TreeConstruction/majority_ref.tre\n",
            "biopython-1.76/Tests/TreeConstruction/msa.phy\n",
            "biopython-1.76/Tests/TreeConstruction/neighbor_trees.tre\n",
            "biopython-1.76/Tests/TreeConstruction/nj.tre\n",
            "biopython-1.76/Tests/TreeConstruction/nj_min.tre\n",
            "biopython-1.76/Tests/TreeConstruction/pars3.tre\n",
            "biopython-1.76/Tests/TreeConstruction/strict_refs.tre\n",
            "biopython-1.76/Tests/TreeConstruction/test.log\n",
            "biopython-1.76/Tests/TreeConstruction/trees.tre\n",
            "biopython-1.76/Tests/TreeConstruction/upgma.tre\n",
            "biopython-1.76/Tests/UniGene/\n",
            "biopython-1.76/Tests/UniGene/Eca.1.2425.data\n",
            "biopython-1.76/Tests/UniGene/Hs.2.data\n",
            "biopython-1.76/Tests/UniGene/Mdm_partial.data\n",
            "biopython-1.76/Tests/UniProt/\n",
            "biopython-1.76/Tests/UniProt/gene_association.goa_yeast.1.gaf\n",
            "biopython-1.76/Tests/UniProt/goa_yeast.gaf\n",
            "biopython-1.76/Tests/UniProt/goa_yeast.gpa.59.gpa\n",
            "biopython-1.76/Tests/UniProt/gp_information.goa_yeast.28.gpi\n",
            "biopython-1.76/Tests/Wise/\n",
            "biopython-1.76/Tests/Wise/human_114_g01_exons.fna_01\n",
            "biopython-1.76/Tests/Wise/human_114_g02_exons.fna_01\n",
            "biopython-1.76/Tests/Xdna/\n",
            "biopython-1.76/Tests/Xdna/sample-a.xdna\n",
            "biopython-1.76/Tests/Xdna/sample-b.xdna\n",
            "biopython-1.76/Tests/Xdna/sample-c.xprt\n",
            "biopython-1.76/Tests/biosql.ini.appveyor\n",
            "biopython-1.76/Tests/biosql.ini.sample\n",
            "biopython-1.76/Tests/codonalign/\n",
            "biopython-1.76/Tests/codonalign/adh.aln\n",
            "biopython-1.76/Tests/codonalign/drosophilla.fasta\n",
            "biopython-1.76/Tests/codonalign/egfr_id\n",
            "biopython-1.76/Tests/codonalign/egfr_nucl.fa\n",
            "biopython-1.76/Tests/codonalign/egfr_pro.aln\n",
            "biopython-1.76/Tests/codonalign/nucl1.fa\n",
            "biopython-1.76/Tests/codonalign/nucl2.fa\n",
            "biopython-1.76/Tests/codonalign/nucl3.fa\n",
            "biopython-1.76/Tests/codonalign/nucl4.fa\n",
            "biopython-1.76/Tests/codonalign/nucl5.fa\n",
            "biopython-1.76/Tests/codonalign/pro1.aln\n",
            "biopython-1.76/Tests/codonalign/pro2.aln\n",
            "biopython-1.76/Tests/codonalign/pro3.aln\n",
            "biopython-1.76/Tests/codonalign/pro4.aln\n",
            "biopython-1.76/Tests/codonalign/pro5.aln\n",
            "biopython-1.76/Tests/common_BioSQL.py\n",
            "biopython-1.76/Tests/common_BioSQL_online.py\n",
            "biopython-1.76/Tests/motifs/\n",
            "biopython-1.76/Tests/motifs/Arnt.sites\n",
            "biopython-1.76/Tests/motifs/MA0056.1.transfac\n",
            "biopython-1.76/Tests/motifs/REB1.pfm\n",
            "biopython-1.76/Tests/motifs/SRF.pfm\n",
            "biopython-1.76/Tests/motifs/abdb.xms\n",
            "biopython-1.76/Tests/motifs/alignace.out\n",
            "biopython-1.76/Tests/motifs/clusterbuster.pfm\n",
            "biopython-1.76/Tests/motifs/clusterbuster.pfm.out\n",
            "biopython-1.76/Tests/motifs/mast.Klf1-200.cd.oops.xml.html\n",
            "biopython-1.76/Tests/motifs/mast.Klf1-200.cd.oops.xml.txt\n",
            "biopython-1.76/Tests/motifs/mast.Klf1-200.cd.oops.xml.xml\n",
            "biopython-1.76/Tests/motifs/mast.adh.de.oops.html.html\n",
            "biopython-1.76/Tests/motifs/mast.adh.de.oops.html.txt\n",
            "biopython-1.76/Tests/motifs/mast.adh.de.oops.html.xml\n",
            "biopython-1.76/Tests/motifs/mast.crp0.de.oops.txt.html\n",
            "biopython-1.76/Tests/motifs/mast.crp0.de.oops.txt.txt\n",
            "biopython-1.76/Tests/motifs/mast.crp0.de.oops.txt.xml\n",
            "biopython-1.76/Tests/motifs/meme.dna.oops.txt\n",
            "biopython-1.76/Tests/motifs/meme.out\n",
            "biopython-1.76/Tests/motifs/meme.protein.oops.txt\n",
            "biopython-1.76/Tests/motifs/meme.protein.tcm.txt\n",
            "biopython-1.76/Tests/motifs/meme.rna.oops.txt\n",
            "biopython-1.76/Tests/motifs/meme_v_4_11_4.txt\n",
            "biopython-1.76/Tests/motifs/minimal_test.meme\n",
            "biopython-1.76/Tests/motifs/transfac.dat\n",
            "biopython-1.76/Tests/msf/\n",
            "biopython-1.76/Tests/msf/DOA_prot.msf\n",
            "biopython-1.76/Tests/msf/W_prot.msf\n",
            "biopython-1.76/Tests/pairwise2_testCases.py\n",
            "biopython-1.76/Tests/phenotype/\n",
            "biopython-1.76/Tests/phenotype/BadPlate.json\n",
            "biopython-1.76/Tests/phenotype/Plate.json\n",
            "biopython-1.76/Tests/phenotype/Plate_2.json\n",
            "biopython-1.76/Tests/phenotype/Plates.csv\n",
            "biopython-1.76/Tests/phenotype/SmallPlate.json\n",
            "biopython-1.76/Tests/phenotype/SmallPlate_2.json\n",
            "biopython-1.76/Tests/phenotype/SmallPlates.csv\n",
            "biopython-1.76/Tests/requires_internet.py\n",
            "biopython-1.76/Tests/requires_wise.py\n",
            "biopython-1.76/Tests/run_tests.py\n",
            "biopython-1.76/Tests/search_tests_common.py\n",
            "biopython-1.76/Tests/seq_tests_common.py\n",
            "biopython-1.76/Tests/test_Ace.py\n",
            "biopython-1.76/Tests/test_Affy.py\n",
            "biopython-1.76/Tests/test_AlignIO.py\n",
            "biopython-1.76/Tests/test_AlignIO_ClustalIO.py\n",
            "biopython-1.76/Tests/test_AlignIO_EmbossIO.py\n",
            "biopython-1.76/Tests/test_AlignIO_FastaIO.py\n",
            "biopython-1.76/Tests/test_AlignIO_MauveIO.py\n",
            "biopython-1.76/Tests/test_AlignIO_PhylipIO.py\n",
            "biopython-1.76/Tests/test_AlignIO_convert.py\n",
            "biopython-1.76/Tests/test_AlignInfo.py\n",
            "biopython-1.76/Tests/test_Application.py\n",
            "biopython-1.76/Tests/test_BWA_tool.py\n",
            "biopython-1.76/Tests/test_BioSQL_MySQLdb.py\n",
            "biopython-1.76/Tests/test_BioSQL_MySQLdb_online.py\n",
            "biopython-1.76/Tests/test_BioSQL_mysql_connector.py\n",
            "biopython-1.76/Tests/test_BioSQL_mysql_connector_online.py\n",
            "biopython-1.76/Tests/test_BioSQL_psycopg2.py\n",
            "biopython-1.76/Tests/test_BioSQL_psycopg2_online.py\n",
            "biopython-1.76/Tests/test_BioSQL_sqlite3.py\n",
            "biopython-1.76/Tests/test_BioSQL_sqlite3_online.py\n",
            "biopython-1.76/Tests/test_Blast_Record.py\n",
            "biopython-1.76/Tests/test_CAPS.py\n",
            "biopython-1.76/Tests/test_CelFile.py\n",
            "biopython-1.76/Tests/test_Chi2.py\n",
            "biopython-1.76/Tests/test_ClustalOmega_tool.py\n",
            "biopython-1.76/Tests/test_Clustalw_tool.py\n",
            "biopython-1.76/Tests/test_Cluster.py\n",
            "biopython-1.76/Tests/test_CodonTable.py\n",
            "biopython-1.76/Tests/test_ColorSpiral.py\n",
            "biopython-1.76/Tests/test_Compass.py\n",
            "biopython-1.76/Tests/test_Consensus.py\n",
            "biopython-1.76/Tests/test_Crystal.py\n",
            "biopython-1.76/Tests/test_DSSP_tool.py\n",
            "biopython-1.76/Tests/test_Dialign_tool.py\n",
            "biopython-1.76/Tests/test_EMBL_unittest.py\n",
            "biopython-1.76/Tests/test_Emboss.py\n",
            "biopython-1.76/Tests/test_EmbossPhylipNew.py\n",
            "biopython-1.76/Tests/test_EmbossPrimer.py\n",
            "biopython-1.76/Tests/test_Entrez.py\n",
            "biopython-1.76/Tests/test_Entrez_online.py\n",
            "biopython-1.76/Tests/test_Entrez_parser.py\n",
            "biopython-1.76/Tests/test_Enzyme.py\n",
            "biopython-1.76/Tests/test_ExPASy.py\n",
            "biopython-1.76/Tests/test_FSSP.py\n",
            "biopython-1.76/Tests/test_Fasttree_tool.py\n",
            "biopython-1.76/Tests/test_File.py\n",
            "biopython-1.76/Tests/test_GenBank.py\n",
            "biopython-1.76/Tests/test_GenomeDiagram.py\n",
            "biopython-1.76/Tests/test_GraphicsBitmaps.py\n",
            "biopython-1.76/Tests/test_GraphicsChromosome.py\n",
            "biopython-1.76/Tests/test_GraphicsDistribution.py\n",
            "biopython-1.76/Tests/test_GraphicsGeneral.py\n",
            "biopython-1.76/Tests/test_HMMCasino.py\n",
            "biopython-1.76/Tests/test_HMMGeneral.py\n",
            "biopython-1.76/Tests/test_KDTree.py\n",
            "biopython-1.76/Tests/test_KEGG.py\n",
            "biopython-1.76/Tests/test_KEGG_online.py\n",
            "biopython-1.76/Tests/test_KGML_graphics.py\n",
            "biopython-1.76/Tests/test_KGML_graphics_online.py\n",
            "biopython-1.76/Tests/test_KGML_nographics.py\n",
            "biopython-1.76/Tests/test_KeyWList.py\n",
            "biopython-1.76/Tests/test_LogisticRegression.py\n",
            "biopython-1.76/Tests/test_MSAProbs_tool.py\n",
            "biopython-1.76/Tests/test_MafIO_index.py\n",
            "biopython-1.76/Tests/test_Mafft_tool.py\n",
            "biopython-1.76/Tests/test_MarkovModel.py\n",
            "biopython-1.76/Tests/test_Medline.py\n",
            "biopython-1.76/Tests/test_Muscle_tool.py\n",
            "biopython-1.76/Tests/test_NACCESS_tool.py\n",
            "biopython-1.76/Tests/test_NCBITextParser.py\n",
            "biopython-1.76/Tests/test_NCBIXML.py\n",
            "biopython-1.76/Tests/test_NCBI_BLAST_tools.py\n",
            "biopython-1.76/Tests/test_NCBI_qblast.py\n",
            "biopython-1.76/Tests/test_NMR.py\n",
            "biopython-1.76/Tests/test_NaiveBayes.py\n",
            "biopython-1.76/Tests/test_Nexus.py\n",
            "biopython-1.76/Tests/test_PAML_baseml.py\n",
            "biopython-1.76/Tests/test_PAML_codeml.py\n",
            "biopython-1.76/Tests/test_PAML_tools.py\n",
            "biopython-1.76/Tests/test_PAML_yn00.py\n",
            "biopython-1.76/Tests/test_PDB.py\n",
            "biopython-1.76/Tests/test_PDBList.py\n",
            "biopython-1.76/Tests/test_PDB_Dice.py\n",
            "biopython-1.76/Tests/test_PDB_FragmentMapper.py\n",
            "biopython-1.76/Tests/test_PDB_KDTree.py\n",
            "biopython-1.76/Tests/test_PDB_MMCIF2Dict.py\n",
            "biopython-1.76/Tests/test_PDB_MMCIFParser.py\n",
            "biopython-1.76/Tests/test_PDB_Polypetide.py\n",
            "biopython-1.76/Tests/test_PDB_ResidueDepth.py\n",
            "biopython-1.76/Tests/test_PDB_StructureAlignment.py\n",
            "biopython-1.76/Tests/test_PDB_Superimposer.py\n",
            "biopython-1.76/Tests/test_PDB_parse_pdb_header.py\n",
            "biopython-1.76/Tests/test_PDB_vectors.py\n",
            "biopython-1.76/Tests/test_Pathway.py\n",
            "biopython-1.76/Tests/test_Phd.py\n",
            "biopython-1.76/Tests/test_Phylo.py\n",
            "biopython-1.76/Tests/test_PhyloXML.py\n",
            "biopython-1.76/Tests/test_Phylo_CDAO.py\n",
            "biopython-1.76/Tests/test_Phylo_NeXML.py\n",
            "biopython-1.76/Tests/test_Phylo_matplotlib.py\n",
            "biopython-1.76/Tests/test_Phylo_networkx.py\n",
            "biopython-1.76/Tests/test_PopGen_GenePop.py\n",
            "biopython-1.76/Tests/test_PopGen_GenePop_EasyController.py\n",
            "biopython-1.76/Tests/test_PopGen_GenePop_nodepend.py\n",
            "biopython-1.76/Tests/test_Prank_tool.py\n",
            "biopython-1.76/Tests/test_Probcons_tool.py\n",
            "biopython-1.76/Tests/test_ProtParam.py\n",
            "biopython-1.76/Tests/test_QCPSuperimposer.py\n",
            "biopython-1.76/Tests/test_RCSBFormats.py\n",
            "biopython-1.76/Tests/test_Restriction.py\n",
            "biopython-1.76/Tests/test_SCOP_Astral.py\n",
            "biopython-1.76/Tests/test_SCOP_Cla.py\n",
            "biopython-1.76/Tests/test_SCOP_Des.py\n",
            "biopython-1.76/Tests/test_SCOP_Dom.py\n",
            "biopython-1.76/Tests/test_SCOP_Hie.py\n",
            "biopython-1.76/Tests/test_SCOP_Raf.py\n",
            "biopython-1.76/Tests/test_SCOP_Residues.py\n",
            "biopython-1.76/Tests/test_SCOP_Scop.py\n",
            "biopython-1.76/Tests/test_SCOP_online.py\n",
            "biopython-1.76/Tests/test_SVDSuperimposer.py\n",
            "biopython-1.76/Tests/test_SearchIO_blast_tab.py\n",
            "biopython-1.76/Tests/test_SearchIO_blast_tab_index.py\n",
            "biopython-1.76/Tests/test_SearchIO_blast_text.py\n",
            "biopython-1.76/Tests/test_SearchIO_blast_xml.py\n",
            "biopython-1.76/Tests/test_SearchIO_blast_xml_index.py\n",
            "biopython-1.76/Tests/test_SearchIO_blat_psl.py\n",
            "biopython-1.76/Tests/test_SearchIO_blat_psl_index.py\n",
            "biopython-1.76/Tests/test_SearchIO_exonerate.py\n",
            "biopython-1.76/Tests/test_SearchIO_exonerate_text_index.py\n",
            "biopython-1.76/Tests/test_SearchIO_exonerate_vulgar_index.py\n",
            "biopython-1.76/Tests/test_SearchIO_fasta_m10.py\n",
            "biopython-1.76/Tests/test_SearchIO_fasta_m10_index.py\n",
            "biopython-1.76/Tests/test_SearchIO_hhsuite2_text.py\n",
            "biopython-1.76/Tests/test_SearchIO_hmmer2_text.py\n",
            "biopython-1.76/Tests/test_SearchIO_hmmer2_text_index.py\n",
            "biopython-1.76/Tests/test_SearchIO_hmmer3_domtab.py\n",
            "biopython-1.76/Tests/test_SearchIO_hmmer3_domtab_index.py\n",
            "biopython-1.76/Tests/test_SearchIO_hmmer3_tab.py\n",
            "biopython-1.76/Tests/test_SearchIO_hmmer3_tab_index.py\n",
            "biopython-1.76/Tests/test_SearchIO_hmmer3_text.py\n",
            "biopython-1.76/Tests/test_SearchIO_hmmer3_text_index.py\n",
            "biopython-1.76/Tests/test_SearchIO_interproscan_xml.py\n",
            "biopython-1.76/Tests/test_SearchIO_legacy.py\n",
            "biopython-1.76/Tests/test_SearchIO_model.py\n",
            "biopython-1.76/Tests/test_SearchIO_write.py\n",
            "biopython-1.76/Tests/test_SeqFeature.py\n",
            "biopython-1.76/Tests/test_SeqIO.py\n",
            "biopython-1.76/Tests/test_SeqIO_AbiIO.py\n",
            "biopython-1.76/Tests/test_SeqIO_FastaIO.py\n",
            "biopython-1.76/Tests/test_SeqIO_Gck.py\n",
            "biopython-1.76/Tests/test_SeqIO_Insdc.py\n",
            "biopython-1.76/Tests/test_SeqIO_NibIO.py\n",
            "biopython-1.76/Tests/test_SeqIO_PdbIO.py\n",
            "biopython-1.76/Tests/test_SeqIO_QualityIO.py\n",
            "biopython-1.76/Tests/test_SeqIO_SeqXML.py\n",
            "biopython-1.76/Tests/test_SeqIO_SnapGene.py\n",
            "biopython-1.76/Tests/test_SeqIO_Xdna.py\n",
            "biopython-1.76/Tests/test_SeqIO_convert.py\n",
            "biopython-1.76/Tests/test_SeqIO_features.py\n",
            "biopython-1.76/Tests/test_SeqIO_index.py\n",
            "biopython-1.76/Tests/test_SeqIO_online.py\n",
            "biopython-1.76/Tests/test_SeqIO_write.py\n",
            "biopython-1.76/Tests/test_SeqRecord.py\n",
            "biopython-1.76/Tests/test_SeqUtils.py\n",
            "biopython-1.76/Tests/test_Seq_objs.py\n",
            "biopython-1.76/Tests/test_SffIO.py\n",
            "biopython-1.76/Tests/test_SubsMat.py\n",
            "biopython-1.76/Tests/test_SwissProt.py\n",
            "biopython-1.76/Tests/test_TCoffee_tool.py\n",
            "biopython-1.76/Tests/test_TogoWS.py\n",
            "biopython-1.76/Tests/test_TreeConstruction.py\n",
            "biopython-1.76/Tests/test_Tutorial.py\n",
            "biopython-1.76/Tests/test_UniGene.py\n",
            "biopython-1.76/Tests/test_UniProt_GOA.py\n",
            "biopython-1.76/Tests/test_Uniprot.py\n",
            "biopython-1.76/Tests/test_Wise.py\n",
            "biopython-1.76/Tests/test_XXmotif_tool.py\n",
            "biopython-1.76/Tests/test_align.py\n",
            "biopython-1.76/Tests/test_align_substitution_matrices.py\n",
            "biopython-1.76/Tests/test_bgzf.py\n",
            "biopython-1.76/Tests/test_cellosaurus.py\n",
            "biopython-1.76/Tests/test_codonalign.py\n",
            "biopython-1.76/Tests/test_geo.py\n",
            "biopython-1.76/Tests/test_kNN.py\n",
            "biopython-1.76/Tests/test_lowess.py\n",
            "biopython-1.76/Tests/test_mmtf.py\n",
            "biopython-1.76/Tests/test_mmtf_online.py\n",
            "biopython-1.76/Tests/test_motifs.py\n",
            "biopython-1.76/Tests/test_motifs_online.py\n",
            "biopython-1.76/Tests/test_pairwise2.py\n",
            "biopython-1.76/Tests/test_pairwise2_no_C.py\n",
            "biopython-1.76/Tests/test_pairwise_aligner.py\n",
            "biopython-1.76/Tests/test_phenotype.py\n",
            "biopython-1.76/Tests/test_phenotype_fit.py\n",
            "biopython-1.76/Tests/test_phyml_tool.py\n",
            "biopython-1.76/Tests/test_prodoc.py\n",
            "biopython-1.76/Tests/test_prosite1.py\n",
            "biopython-1.76/Tests/test_prosite2.py\n",
            "biopython-1.76/Tests/test_psw.py\n",
            "biopython-1.76/Tests/test_raxml_tool.py\n",
            "biopython-1.76/Tests/test_samtools_tool.py\n",
            "biopython-1.76/Tests/test_seq.py\n",
            "biopython-1.76/Tests/test_translate.py\n",
            "biopython-1.76/Tests/test_trie.py\n",
            "biopython-1.76/biopython.egg-info/\n",
            "biopython-1.76/biopython.egg-info/PKG-INFO\n",
            "biopython-1.76/biopython.egg-info/SOURCES.txt\n",
            "biopython-1.76/biopython.egg-info/dependency_links.txt\n",
            "biopython-1.76/biopython.egg-info/requires.txt\n",
            "biopython-1.76/biopython.egg-info/top_level.txt\n",
            "biopython-1.76/setup.cfg\n",
            "biopython-1.76/setup.py\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install Bio"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "x7E4MlZ4H5Wy",
        "outputId": "fcc48bf6-01b2-42d0-d804-b1c7aab925fa"
      },
      "execution_count": 62,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: Bio in /usr/local/lib/python3.12/dist-packages (1.6.0)\n",
            "Collecting biopython>=1.80 (from Bio)\n",
            "  Using cached biopython-1.81-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (13 kB)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.12/dist-packages (from Bio) (2.31.0)\n",
            "Requirement already satisfied: tqdm in /usr/local/lib/python3.12/dist-packages (from Bio) (4.66.1)\n",
            "Requirement already satisfied: mygene in /usr/local/lib/python3.12/dist-packages (from Bio) (3.2.2)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.12/dist-packages (from Bio) (2.1.2)\n",
            "Requirement already satisfied: pooch in /usr/local/lib/python3.12/dist-packages (from Bio) (1.8.0)\n",
            "Requirement already satisfied: gprofiler-official in /usr/local/lib/python3.12/dist-packages (from Bio) (1.0.0)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.12/dist-packages (from biopython>=1.80->Bio) (1.26.1)\n",
            "Requirement already satisfied: biothings-client>=0.2.6 in /usr/local/lib/python3.12/dist-packages (from mygene->Bio) (0.3.0)\n",
            "Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.12/dist-packages (from pandas->Bio) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.12/dist-packages (from pandas->Bio) (2023.3.post1)\n",
            "Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.12/dist-packages (from pandas->Bio) (2023.3)\n",
            "Requirement already satisfied: platformdirs>=2.5.0 in /usr/local/lib/python3.12/dist-packages (from pooch->Bio) (3.11.0)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.12/dist-packages (from pooch->Bio) (23.2)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (3.3.1)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (3.4)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.12/dist-packages (from requests->Bio) (2023.7.22)\n",
            "Requirement already satisfied: six>=1.5 in /usr/lib/python3/dist-packages (from python-dateutil>=2.8.2->pandas->Bio) (1.16.0)\n",
            "Using cached biopython-1.81-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)\n",
            "Installing collected packages: biopython\n",
            "Successfully installed biopython-1.81\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from Bio import SubsMat"
      ],
      "metadata": {
        "id": "cqvlEDJ6LcTX"
      },
      "execution_count": 66,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "T5b8G1Dl27vx",
        "outputId": "a3816e5d-679e-4d5c-d6aa-2a4db6e99db0"
      },
      "execution_count": 72,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/usr/local/lib/python3.12/dist-packages/Bio/pairwise2.py:278: BiopythonDeprecationWarning: Bio.pairwise2 has been deprecated, and we intend to remove it in a future release of Biopython. As an alternative, please consider using Bio.Align.PairwiseAligner as a replacement, and contact the Biopython developers if you still need the Bio.pairwise2 module.\n",
            "  warnings.warn(\n",
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 5, in <module>\n",
            "    from Bio.SubsMat import MatrixInfo as matlist\n",
            "ModuleNotFoundError: No module named 'Bio.SubsMat'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!mv biopython-1.76/Bio/SubsMat /usr/local/lib/python3.12/dist-packages/Bio/\n",
        "!mv biopython-1.76/Bio/Alphabet/__init__.py /usr/local/lib/python3.12/dist-packages/Bio/"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "PiQMRwh9NIiu",
        "outputId": "b309c9ef-e82a-43ed-f4a8-ce3dbce86bd4"
      },
      "execution_count": 77,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "mv: cannot stat 'biopython-1.76/Bio/SubsMat': No such file or directory\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "vd36r4ChNYOc",
        "outputId": "e3351f2f-e2b1-4911-acaa-8352c5ec66c2"
      },
      "execution_count": 80,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 4, in <module>\n",
            "    from Bio import pairwise2\n",
            "  File \"/usr/local/lib/python3.12/dist-packages/Bio/pairwise2.py\", line 274, in <module>\n",
            "    from Bio import BiopythonWarning\n",
            "ImportError: cannot import name 'BiopythonWarning' from 'Bio' (/usr/local/lib/python3.12/dist-packages/Bio/__init__.py)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "uHEVi98IN0qr",
        "outputId": "3c6f7a50-fe31-4fcd-db85-7d8e251b0a5d"
      },
      "execution_count": 81,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2023-10-27 08:27:41--  https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh\n",
            "Resolving repo.anaconda.com (repo.anaconda.com)... 104.16.130.3, 104.16.131.3, 2606:4700::6810:8303, ...\n",
            "Connecting to repo.anaconda.com (repo.anaconda.com)|104.16.130.3|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 120771089 (115M) [application/x-sh]\n",
            "Saving to: ‘Miniconda3-latest-Linux-x86_64.sh’\n",
            "\n",
            "Miniconda3-latest-L 100%[===================>] 115.18M   175MB/s    in 0.7s    \n",
            "\n",
            "2023-10-27 08:27:41 (175 MB/s) - ‘Miniconda3-latest-Linux-x86_64.sh’ saved [120771089/120771089]\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!chmod +x Miniconda3-latest-Linux-x86_64.sh"
      ],
      "metadata": {
        "id": "ZGR3zvt8Pypz"
      },
      "execution_count": 82,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!./Miniconda3-latest-Linux-x86_64.sh -b -f -p /usr/local"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "lIpJRJTBP2oB",
        "outputId": "3b2ce869-9a4f-4c0e-8614-aeabe6d924b7"
      },
      "execution_count": 83,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PREFIX=/usr/local\n",
            "Unpacking payload ...\n",
            "                                                                                    \n",
            "Installing base environment...\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "\n",
            "Preparing transaction: - \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "Executing transaction: | \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "installation finished.\n",
            "WARNING:\n",
            "    You currently have a PYTHONPATH environment variable set. This may cause\n",
            "    unexpected behavior when running the Python interpreter in Miniconda3.\n",
            "    For best results, please verify that your PYTHONPATH only points to\n",
            "    directories of packages that are compatible with the Python interpreter\n",
            "    in Miniconda3: /usr/local\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!conda install biopythonBio.1.76"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "LNT4gjkUP_rS",
        "outputId": "564795f6-91c9-498e-aa6f-da1867ae52ce"
      },
      "execution_count": 84,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "Solving environment: / \b\bunsuccessful initial attempt using frozen solve. Retrying with flexible solve.\n",
            "Collecting package metadata (repodata.json): \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "Solving environment: / \b\bunsuccessful initial attempt using frozen solve. Retrying with flexible solve.\n",
            "\n",
            "PackagesNotFoundError: The following packages are not available from current channels:\n",
            "\n",
            "  - biopythonbio.1.76\n",
            "\n",
            "Current channels:\n",
            "\n",
            "  - https://repo.anaconda.com/pkgs/main/linux-64\n",
            "  - https://repo.anaconda.com/pkgs/main/noarch\n",
            "  - https://repo.anaconda.com/pkgs/r/linux-64\n",
            "  - https://repo.anaconda.com/pkgs/r/noarch\n",
            "\n",
            "To search for alternate channels that may provide the conda package you're\n",
            "looking for, navigate to\n",
            "\n",
            "    https://anaconda.org\n",
            "\n",
            "and use the search bar at the top of the page.\n",
            "\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "LqVtwchMQEy4",
        "outputId": "86ca7fc4-ff3d-4066-b3a2-e00e518254cc"
      },
      "execution_count": 85,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 1, in <module>\n",
            "    import google\n",
            "ModuleNotFoundError: No module named 'google'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install --upgrade google-api-python-client"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "wJDyPVEGQfto",
        "outputId": "609e8199-e00e-43b6-b8e7-4fd2b870794e"
      },
      "execution_count": 86,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting google-api-python-client\n",
            "  Obtaining dependency information for google-api-python-client from https://files.pythonhosted.org/packages/0b/9a/69b094da2ddce819b905e3d63e3bf8713a519a5cec0b109f39a1e5da8926/google_api_python_client-2.105.0-py2.py3-none-any.whl.metadata\n",
            "  Downloading google_api_python_client-2.105.0-py2.py3-none-any.whl.metadata (6.6 kB)\n",
            "Collecting httplib2<1.dev0,>=0.15.0 (from google-api-python-client)\n",
            "  Downloading httplib2-0.22.0-py3-none-any.whl (96 kB)\n",
            "\u001b[?25l     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m0.0/96.9 kB\u001b[0m \u001b[31m?\u001b[0m eta \u001b[36m-:--:--\u001b[0m\r\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m96.9/96.9 kB\u001b[0m \u001b[31m4.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting google-auth<3.0.0.dev0,>=1.19.0 (from google-api-python-client)\n",
            "  Obtaining dependency information for google-auth<3.0.0.dev0,>=1.19.0 from https://files.pythonhosted.org/packages/39/7c/2e4fa55a99f83ef9ef229ac5d59c44ceb90e2d0145711590c0fa39669f32/google_auth-2.23.3-py2.py3-none-any.whl.metadata\n",
            "  Downloading google_auth-2.23.3-py2.py3-none-any.whl.metadata (4.2 kB)\n",
            "Collecting google-auth-httplib2>=0.1.0 (from google-api-python-client)\n",
            "  Obtaining dependency information for google-auth-httplib2>=0.1.0 from https://files.pythonhosted.org/packages/d3/3d/e4991229886c0d522d9552151a43ff7adcc61e026e60ce8bd508387f84cf/google_auth_httplib2-0.1.1-py2.py3-none-any.whl.metadata\n",
            "  Downloading google_auth_httplib2-0.1.1-py2.py3-none-any.whl.metadata (2.1 kB)\n",
            "Collecting google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5 (from google-api-python-client)\n",
            "  Obtaining dependency information for google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5 from https://files.pythonhosted.org/packages/4d/ce/4fd62ea66b3508debc795e475336ce915929765870f0ad52328426ba016e/google_api_core-2.12.0-py3-none-any.whl.metadata\n",
            "  Downloading google_api_core-2.12.0-py3-none-any.whl.metadata (2.7 kB)\n",
            "Collecting uritemplate<5,>=3.0.1 (from google-api-python-client)\n",
            "  Downloading uritemplate-4.1.1-py2.py3-none-any.whl (10 kB)\n",
            "Collecting googleapis-common-protos<2.0.dev0,>=1.56.2 (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client)\n",
            "  Obtaining dependency information for googleapis-common-protos<2.0.dev0,>=1.56.2 from https://files.pythonhosted.org/packages/21/49/12996dc0238e017504dceea1d121a48bd49fb3f4416f40d59fc3e924b4f3/googleapis_common_protos-1.61.0-py2.py3-none-any.whl.metadata\n",
            "  Downloading googleapis_common_protos-1.61.0-py2.py3-none-any.whl.metadata (1.5 kB)\n",
            "Collecting protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0.dev0,>=3.19.5 (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client)\n",
            "  Obtaining dependency information for protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0.dev0,>=3.19.5 from https://files.pythonhosted.org/packages/c8/2c/03046cac73f46bfe98fc846ef629cf4f84c2f59258216aa2cc0d22bfca8f/protobuf-4.24.4-cp37-abi3-manylinux2014_x86_64.whl.metadata\n",
            "  Downloading protobuf-4.24.4-cp37-abi3-manylinux2014_x86_64.whl.metadata (540 bytes)\n",
            "Requirement already satisfied: requests<3.0.0.dev0,>=2.18.0 in /usr/local/lib/python3.11/site-packages (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client) (2.31.0)\n",
            "Collecting cachetools<6.0,>=2.0.0 (from google-auth<3.0.0.dev0,>=1.19.0->google-api-python-client)\n",
            "  Obtaining dependency information for cachetools<6.0,>=2.0.0 from https://files.pythonhosted.org/packages/a2/91/2d843adb9fbd911e0da45fbf6f18ca89d07a087c3daa23e955584f90ebf4/cachetools-5.3.2-py3-none-any.whl.metadata\n",
            "  Downloading cachetools-5.3.2-py3-none-any.whl.metadata (5.2 kB)\n",
            "Collecting pyasn1-modules>=0.2.1 (from google-auth<3.0.0.dev0,>=1.19.0->google-api-python-client)\n",
            "  Downloading pyasn1_modules-0.3.0-py2.py3-none-any.whl (181 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m181.3/181.3 kB\u001b[0m \u001b[31m11.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting rsa<5,>=3.1.4 (from google-auth<3.0.0.dev0,>=1.19.0->google-api-python-client)\n",
            "  Downloading rsa-4.9-py3-none-any.whl (34 kB)\n",
            "Collecting pyparsing!=3.0.0,!=3.0.1,!=3.0.2,!=3.0.3,<4,>=2.4.2 (from httplib2<1.dev0,>=0.15.0->google-api-python-client)\n",
            "  Obtaining dependency information for pyparsing!=3.0.0,!=3.0.1,!=3.0.2,!=3.0.3,<4,>=2.4.2 from https://files.pythonhosted.org/packages/39/92/8486ede85fcc088f1b3dba4ce92dd29d126fd96b0008ea213167940a2475/pyparsing-3.1.1-py3-none-any.whl.metadata\n",
            "  Downloading pyparsing-3.1.1-py3-none-any.whl.metadata (5.1 kB)\n",
            "Collecting pyasn1<0.6.0,>=0.4.6 (from pyasn1-modules>=0.2.1->google-auth<3.0.0.dev0,>=1.19.0->google-api-python-client)\n",
            "  Downloading pyasn1-0.5.0-py2.py3-none-any.whl (83 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m83.9/83.9 kB\u001b[0m \u001b[31m9.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.11/site-packages (from requests<3.0.0.dev0,>=2.18.0->google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client) (2.0.4)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.11/site-packages (from requests<3.0.0.dev0,>=2.18.0->google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client) (3.4)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.11/site-packages (from requests<3.0.0.dev0,>=2.18.0->google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client) (1.26.16)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.11/site-packages (from requests<3.0.0.dev0,>=2.18.0->google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0.dev0,>=1.31.5->google-api-python-client) (2023.7.22)\n",
            "Downloading google_api_python_client-2.105.0-py2.py3-none-any.whl (12.6 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m12.6/12.6 MB\u001b[0m \u001b[31m58.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading google_api_core-2.12.0-py3-none-any.whl (121 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m121.4/121.4 kB\u001b[0m \u001b[31m15.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading google_auth-2.23.3-py2.py3-none-any.whl (182 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m182.3/182.3 kB\u001b[0m \u001b[31m23.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading google_auth_httplib2-0.1.1-py2.py3-none-any.whl (9.3 kB)\n",
            "Downloading cachetools-5.3.2-py3-none-any.whl (9.3 kB)\n",
            "Downloading googleapis_common_protos-1.61.0-py2.py3-none-any.whl (230 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m230.9/230.9 kB\u001b[0m \u001b[31m26.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading protobuf-4.24.4-cp37-abi3-manylinux2014_x86_64.whl (311 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m311.6/311.6 kB\u001b[0m \u001b[31m35.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading pyparsing-3.1.1-py3-none-any.whl (103 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m103.1/103.1 kB\u001b[0m \u001b[31m13.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: uritemplate, pyparsing, pyasn1, protobuf, cachetools, rsa, pyasn1-modules, httplib2, googleapis-common-protos, google-auth, google-auth-httplib2, google-api-core, google-api-python-client\n",
            "Successfully installed cachetools-5.3.2 google-api-core-2.12.0 google-api-python-client-2.105.0 google-auth-2.23.3 google-auth-httplib2-0.1.1 googleapis-common-protos-1.61.0 httplib2-0.22.0 protobuf-4.24.4 pyasn1-0.5.0 pyasn1-modules-0.3.0 pyparsing-3.1.1 rsa-4.9 uritemplate-4.1.1\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.colab-display-data+json": {
              "pip_warning": {
                "packages": [
                  "cachetools",
                  "google",
                  "httplib2"
                ]
              }
            }
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "YwZipwrqQjjU",
        "outputId": "f07fa66c-9afb-4816-90a9-783e17fc12f4"
      },
      "execution_count": 87,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 3, in <module>\n",
            "    from typing_extensions import override\n",
            "ModuleNotFoundError: No module named 'typing_extensions'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install typing-extensions"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "El-xAEC1QxSc",
        "outputId": "270d0494-ae38-4bd9-bb2b-db1b9c34b045"
      },
      "execution_count": 88,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting typing-extensions\n",
            "  Obtaining dependency information for typing-extensions from https://files.pythonhosted.org/packages/24/21/7d397a4b7934ff4028987914ac1044d3b7d52712f30e2ac7a2ae5bc86dd0/typing_extensions-4.8.0-py3-none-any.whl.metadata\n",
            "  Downloading typing_extensions-4.8.0-py3-none-any.whl.metadata (3.0 kB)\n",
            "Downloading typing_extensions-4.8.0-py3-none-any.whl (31 kB)\n",
            "Installing collected packages: typing-extensions\n",
            "Successfully installed typing-extensions-4.8.0\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "a88rZJY-Q0k5",
        "outputId": "8a1f8308-d5ea-4ac2-a914-aebf8390b0a7"
      },
      "execution_count": 89,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 4, in <module>\n",
            "    from Bio import pairwise2\n",
            "ModuleNotFoundError: No module named 'Bio'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!conda install python=3.12"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "xAA6IdOjSxep",
        "outputId": "cef02a91-bbea-4a5e-dc93-35fc8dbeffb9"
      },
      "execution_count": 100,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Solving environment: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bunsuccessful initial attempt using frozen solve. Retrying with flexible solve.\n",
            "Solving environment: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bunsuccessful attempt using repodata from current_repodata.json, retrying with next repodata source.\n",
            "Collecting package metadata (repodata.json): / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Solving environment: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bunsuccessful initial attempt using frozen solve. Retrying with flexible solve.\n",
            "Solving environment: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \n",
            "Found conflicts! Looking for incompatible packages.\n",
            "This can take several minutes.  Press CTRL-C to abort.\b\b| \n",
            "\b\b/ \b\bfailed\n",
            "\n",
            "UnsatisfiableError: The following specifications were found to be incompatible with a past\n",
            "explicit spec that is not an explicit spec in this operation (pip):\n",
            "\n",
            "  - python=3.12 -> pip -> setuptools\n",
            "  - python=3.12 -> pip -> wheel\n",
            "\n",
            "The following specifications were found to be incompatible with each other:\n",
            "\n",
            "Output in format: Requested package -> Available versions\n",
            "\n",
            "Package libuuid conflicts for:\n",
            "pysocks -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "cffi -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "jsonpatch -> python -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "ruamel.yaml -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "pyopenssl -> python[version='>=3.10,<3.11.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "pycparser -> python[version='>=3.6'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "brotlipy -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "urllib3 -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "conda-content-trust -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "cryptography -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "conda-libmamba-solver -> python[version='>=3.10,<3.11.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "jsonpointer -> python -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "conda-package-handling -> python[version='>=3.10,<3.11.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "requests -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "conda-package-streaming -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "python=3.12 -> libuuid[version='>=1.41.5,<2.0a0']\n",
            "packaging -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "pycosat -> python[version='>=3.10,<3.11.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "charset-normalizer -> python[version='>=3.5'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "libuuid\n",
            "libmambapy -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "wheel -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "zstandard -> python[version='>=3.11,<3.12.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> libuuid[version='>=1.0.3,<2.0a0|>=1.41.5,<2.0a0']\n",
            "\n",
            "Package _libgcc_mutex conflicts for:\n",
            "libcurl -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "_libgcc_mutex\n",
            "libsolv -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "_openmp_mutex -> _libgcc_mutex==0.1=main\n",
            "libgcc-ng -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "libarchive -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "libmambapy -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "conda-package-handling -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "lz4-c -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "bzip2 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "cryptography -> libgcc-ng -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "brotlipy -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "openssl -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "zlib -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "reproc-cpp -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "c-ares -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "libssh2 -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "icu -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "libedit -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "cffi -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "reproc -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "pcre2 -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "xz -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "sqlite -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "libffi -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "tk -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "python=3.12 -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "readline -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "libuuid -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "zstd -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "ncurses -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "pycosat -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "krb5 -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "ruamel.yaml -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "libnghttp2 -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "libgomp -> _libgcc_mutex==0.1=main\n",
            "libmamba -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "libev -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "zstandard -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "fmt -> libgcc-ng[version='>=11.2.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "yaml-cpp -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1',build=main]\n",
            "\n",
            "Package libgcc-ng conflicts for:\n",
            "yaml-cpp -> libgcc-ng[version='>=7.5.0']\n",
            "brotlipy -> libgcc-ng[version='>=11.2.0|>=7.3.0|>=7.5.0']\n",
            "tk -> zlib[version='>=1.2.12,<1.3.0a0'] -> libgcc-ng[version='>=11.2.0']\n",
            "conda-package-handling -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0']\n",
            "libarchive -> libxml2[version='>=2.9.12,<2.10.0a0'] -> libgcc-ng[version='>=9.3.0']\n",
            "libmamba -> libgcc-ng[version='>=11.2.0|>=7.5.0']\n",
            "pycosat -> libgcc-ng[version='>=11.2.0|>=7.3.0|>=7.5.0|>=7.2.0']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0']\n",
            "pcre2 -> zlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=7.2.0']\n",
            "pycparser -> python[version='>=3.6'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "zstd -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libssh2 -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libev -> libgcc-ng[version='>=7.3.0|>=7.5.0']\n",
            "pysocks -> python[version='>=3.7,<3.8.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "fmt -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0']\n",
            "packaging -> python[version='>=3.8,<3.9.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libuuid -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.2.0']\n",
            "libsolv -> libgcc-ng[version='>=11.2.0|>=7.5.0']\n",
            "bzip2 -> libgcc-ng[version='>=7.2.0|>=7.3.0']\n",
            "pcre2 -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0']\n",
            "conda-libmamba-solver -> libmambapy[version='>=1.5.1'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "tk -> libgcc-ng[version='>=7.2.0|>=7.3.0|>=7.5.0']\n",
            "jsonpointer -> python -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "c-ares -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0']\n",
            "icu -> libgcc-ng[version='>=11.2.0|>=7.3.0|>=7.2.0']\n",
            "conda-package-handling -> python[version='>=3.7,<3.8.0a0'] -> libgcc-ng[version='>=7.2.0']\n",
            "conda[version='>=23.9.0'] -> pycosat[version='>=0.6.3'] -> libgcc-ng[version='>=11.2.0|>=7.3.0|>=7.5.0|>=7.2.0']\n",
            "ruamel.yaml -> libgcc-ng[version='>=11.2.0|>=7.3.0|>=7.5.0']\n",
            "xz -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libarchive -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libcurl -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "zstandard -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0']\n",
            "wheel -> python[version='>=3.9,<3.10.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "charset-normalizer -> python[version='>=3.5'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libnghttp2 -> zlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=7.2.0']\n",
            "zlib -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "urllib3 -> brotli-python[version='>=1.0.9'] -> libgcc-ng[version='>=11.2.0|>=7.3.0|>=7.5.0|>=7.2.0']\n",
            "libmambapy -> libgcc-ng[version='>=11.2.0|>=7.5.0']\n",
            "reproc-cpp -> libgcc-ng[version='>=7.5.0']\n",
            "cffi -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "jsonpatch -> python -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "sqlite -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "conda-package-streaming -> python[version='>=3.8,<3.9.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "reproc -> libgcc-ng[version='>=7.5.0']\n",
            "readline -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "cryptography -> cffi[version='>=1.12'] -> libgcc-ng[version='>=11.2.0|>=7.5.0']\n",
            "conda-content-trust -> cryptography[version='>=41'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "krb5 -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "brotlipy -> cffi[version='>=1.0.0'] -> libgcc-ng[version='>=7.2.0']\n",
            "ruamel.yaml -> python[version='>=3.7,<3.8.0a0'] -> libgcc-ng[version='>=7.2.0']\n",
            "python=3.12 -> libgcc-ng[version='>=11.2.0']\n",
            "libedit -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libnghttp2 -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0']\n",
            "python=3.12 -> bzip2[version='>=1.0.8,<2.0a0'] -> libgcc-ng[version='>=7.3.0|>=7.5.0']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libffi -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "requests -> python[version='>=3.9,<3.10.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libsolv -> zlib[version='>=1.2.11,<1.3.0a0'] -> libgcc-ng[version='>=7.2.0|>=7.3.0']\n",
            "zstandard -> cffi[version='>=1.11'] -> libgcc-ng[version='>=7.2.0']\n",
            "pyopenssl -> cryptography[version='>=38.0.0,<42,!=40.0.0,!=40.0.1'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "ncurses -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "cryptography -> libgcc-ng[version='>=7.2.0|>=7.3.0']\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libgcc-ng\n",
            "openssl -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libmamba -> bzip2[version='>=1.0.8,<2.0a0'] -> libgcc-ng[version='>=7.3.0']\n",
            "libmambapy -> python[version='>=3.9,<3.10.0a0'] -> libgcc-ng[version='>=7.2.0|>=7.3.0']\n",
            "lz4-c -> libgcc-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "\n",
            "Package python conflicts for:\n",
            "setuptools -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.12,<3.13.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "zstandard -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0']\n",
            "cffi -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0|>=3.9,<3.10.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "conda[version='>=23.9.0'] -> boltons[version='>=23.0.0'] -> python[version='>=2.7|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0|>=3.6|>=3.12,<3.13.0a0']\n",
            "conda-content-trust -> cryptography[version='<41.0.0a0'] -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "zstandard -> cffi[version='>=1.11'] -> python[version='>=3.5,<3.6.0a0']\n",
            "conda-package-handling -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.11,<3.12.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0']\n",
            "requests -> certifi[version='>=2017.4.17'] -> python[version='<4.0|>=3.12,<3.13.0a0|>=3.5|>=3.6,<4.0']\n",
            "packaging -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "ruamel.yaml -> ruamel.yaml.clib[version='>=0.1.2'] -> python[version='>=2.7,<2.8.0a0|>=3.12,<3.13.0a0|>=3.5,<3.6.0a0']\n",
            "jsonpointer -> python\n",
            "cffi -> pycparser -> python[version='>=3.6']\n",
            "idna -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.5|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "pycparser -> python[version='>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0']\n",
            "pyopenssl -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.11,<3.12.0a0|>=3.7,<3.8.0a0|>=3.6|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "conda-package-handling -> tqdm -> python[version='>=2.7|>=3.5,<3.6.0a0']\n",
            "boltons -> python[version='>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0']\n",
            "brotlipy -> python[version='>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0']\n",
            "urllib3 -> python[version='<4.0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<4.0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']\n",
            "requests -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "conda-package-streaming -> zstandard[version='>=0.15'] -> python[version='>=3.6,<3.7.0a0']\n",
            "conda-libmamba-solver -> importlib-metadata -> python[version='>=2.7,<2.8.0a0|>=3|>=3.6,<3.7.0a0']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0|>=3.11,<3.12.0a0']\n",
            "charset-normalizer -> python[version='>=3.5']\n",
            "wheel -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "libmambapy -> python[version='>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0']\n",
            "pycosat -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.8,<3.9.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "ruamel.yaml -> python[version='>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0']\n",
            "tqdm -> python[version='>=2.7,<2.8.0a0|>=2.7|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "jsonpatch -> python\n",
            "conda-content-trust -> python[version='>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']\n",
            "pip -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.12,<3.13.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0|>=3.11,<3.12.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "brotlipy -> cffi[version='>=1.0.0'] -> python[version='>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']\n",
            "pip -> wheel -> python\n",
            "setuptools -> certifi[version='>=2016.9.26'] -> python\n",
            "urllib3 -> certifi -> python[version='<=3.3|>=3.12,<3.13.0a0|>=3.5|>=3.6']\n",
            "conda-package-streaming -> python[version='>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0']\n",
            "cryptography -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0']\n",
            "conda-libmamba-solver -> python[version='>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0|>=3.6']\n",
            "pysocks -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.7,<3.8.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "wheel -> setuptools -> python[version='>=3.12,<3.13.0a0']\n",
            "cryptography -> six[version='>=1.4.1'] -> python[version='<=3.3|>=3.5']\n",
            "certifi -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.12,<3.13.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "pluggy -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']\n",
            "\n",
            "Package ca-certificates conflicts for:\n",
            "python=3.12 -> openssl[version='>=3.0.11,<4.0a0'] -> ca-certificates\n",
            "cffi -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n",
            "tqdm -> python[version='>=2.7'] -> ca-certificates\n",
            "libmambapy -> openssl[version='>=1.1.1v,<1.1.2a'] -> ca-certificates\n",
            "libmamba -> openssl[version='>=1.1.1v,<1.1.2a'] -> ca-certificates\n",
            "libnghttp2 -> openssl[version='>=3.0.11,<4.0a0'] -> ca-certificates\n",
            "openssl -> ca-certificates\n",
            "certifi -> python -> ca-certificates\n",
            "libssh2 -> openssl[version='>=3.0.8,<4.0a0'] -> ca-certificates\n",
            "pysocks -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n",
            "requests -> python -> ca-certificates\n",
            "urllib3 -> python[version='<4.0'] -> ca-certificates\n",
            "ca-certificates\n",
            "pip -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n",
            "libcurl -> openssl[version='>=3.0.11,<4.0a0'] -> ca-certificates\n",
            "conda-package-handling -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n",
            "packaging -> python -> ca-certificates\n",
            "idna -> python -> ca-certificates\n",
            "pyopenssl -> python -> ca-certificates\n",
            "conda-content-trust -> python -> ca-certificates\n",
            "jsonpointer -> python -> ca-certificates\n",
            "setuptools -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n",
            "pluggy -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n",
            "pycparser -> python -> ca-certificates\n",
            "krb5 -> openssl[version='>=1.1.1u,<1.1.2a'] -> ca-certificates\n",
            "jsonpatch -> python -> ca-certificates\n",
            "zstandard -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n",
            "pycosat -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates\n",
            "libarchive -> openssl=3 -> ca-certificates\n",
            "cryptography -> openssl[version='>=1.1.1v,<1.1.2a'] -> ca-certificates\n",
            "wheel -> python -> ca-certificates\n",
            "\n",
            "Package xz conflicts for:\n",
            "pyopenssl -> python[version='>=3.8,<3.9.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.8,<6.0a0|>=5.2.3,<6.0a0']\n",
            "charset-normalizer -> python[version='>=3.5'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "brotlipy -> python[version='>=3.7,<3.8.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0']\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "jsonpatch -> python -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "pycosat -> python[version='>=3.8,<3.9.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.8,<6.0a0|>=5.2.3,<6.0a0']\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0']\n",
            "urllib3 -> python[version='>=3.8,<3.9.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.8,<6.0a0|>=5.2.3,<6.0a0']\n",
            "ruamel.yaml -> python[version='>=3.11,<3.12.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "xz\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.11,<3.12.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0']\n",
            "conda-content-trust -> python[version='>=3.11,<3.12.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "zstandard -> python[version='>=3.11,<3.12.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "pycparser -> python[version='>=3.6'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "conda-libmamba-solver -> python[version='>=3.10,<3.11.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0']\n",
            "requests -> python[version='>=3.9,<3.10.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "libmambapy -> python[version='>=3.9,<3.10.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0']\n",
            "python=3.12 -> xz[version='>=5.4.2,<6.0a0']\n",
            "cryptography -> python[version='>=3.11,<3.12.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "conda-package-streaming -> python[version='>=3.8,<3.9.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.8,<6.0a0']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.8,<6.0a0|>=5.2.3,<6.0a0']\n",
            "libarchive -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "jsonpointer -> python -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "libmamba -> libarchive[version='>=3.6.2,<3.7.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.5,<6.0a0']\n",
            "conda-package-handling -> python[version='>=3.10,<3.11.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "wheel -> python[version='>=3.9,<3.10.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "cffi -> python[version='>=3.11,<3.12.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.3,<6.0a0']\n",
            "zstd -> xz[version='>=5.2.10,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0']\n",
            "packaging -> python[version='>=3.8,<3.9.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.4.2,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.2.8,<6.0a0|>=5.2.3,<6.0a0']\n",
            "pysocks -> python[version='>=3.7,<3.8.0a0'] -> xz[version='>=5.2.10,<6.0a0|>=5.2.6,<6.0a0|>=5.2.5,<6.0a0|>=5.2.4,<6.0a0|>=5.4.2,<6.0a0|>=5.2.8,<6.0a0|>=5.2.3,<6.0a0']\n",
            "\n",
            "Package _openmp_mutex conflicts for:\n",
            "c-ares -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "reproc -> libgcc-ng[version='>=7.5.0'] -> _openmp_mutex[version='>=4.5']\n",
            "libssh2 -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "zstd -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "pcre2 -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "xz -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "cryptography -> libgcc-ng -> _openmp_mutex[version='>=4.5']\n",
            "_openmp_mutex\n",
            "ruamel.yaml -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "icu -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "libuuid -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "readline -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "krb5 -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "ncurses -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "libgcc-ng -> _openmp_mutex[version='>=4.5']\n",
            "pycosat -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "libnghttp2 -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "conda-package-handling -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "libcurl -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "libev -> libgcc-ng[version='>=7.5.0'] -> _openmp_mutex[version='>=4.5']\n",
            "python=3.12 -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex\n",
            "zstandard -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "brotlipy -> libgcc-ng[version='>=7.3.0'] -> _openmp_mutex[version='>=4.5']\n",
            "zlib -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "libarchive -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "cffi -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "sqlite -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "libmambapy -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "libmamba -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "lz4-c -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "libffi -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "openssl -> libgcc-ng[version='>=7.5.0'] -> _openmp_mutex[version='>=4.5']\n",
            "fmt -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "yaml-cpp -> libgcc-ng[version='>=7.5.0'] -> _openmp_mutex[version='>=4.5']\n",
            "tk -> libgcc-ng[version='>=7.5.0'] -> _openmp_mutex[version='>=4.5']\n",
            "bzip2 -> libgcc-ng[version='>=7.3.0'] -> _openmp_mutex[version='>=4.5']\n",
            "reproc-cpp -> libgcc-ng[version='>=7.5.0'] -> _openmp_mutex[version='>=4.5']\n",
            "libsolv -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "libedit -> libgcc-ng[version='>=11.2.0'] -> _openmp_mutex[version='>=4.5']\n",
            "\n",
            "Package bzip2 conflicts for:\n",
            "libarchive -> bzip2[version='>=1.0.6,<2.0a0|>=1.0.8,<2.0a0']\n",
            "brotlipy -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "cryptography -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "jsonpointer -> python -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "libmambapy -> libmamba==1.5.1=hba0046a_0 -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "bzip2\n",
            "conda-package-streaming -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "charset-normalizer -> python[version='>=3.5'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "packaging -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "cffi -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "pcre2 -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "libmamba -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "urllib3 -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "pycosat -> python[version='>=3.10,<3.11.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "ruamel.yaml -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "jsonpatch -> python -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "wheel -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "libsolv -> pcre2[version='>=10.42,<10.43.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "pyopenssl -> python[version='>=3.10,<3.11.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "zstandard -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "pycparser -> python[version='>=3.6'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "conda-package-handling -> python[version='>=3.10,<3.11.0a0'] -> bzip2[version='>=1.0.6,<2.0a0|>=1.0.8,<2.0a0']\n",
            "conda-libmamba-solver -> python[version='>=3.10,<3.11.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "python=3.12 -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "pysocks -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "requests -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "conda-content-trust -> python[version='>=3.11,<3.12.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']\n",
            "\n",
            "Package jsonpatch conflicts for:\n",
            "conda-libmamba-solver -> conda[version='>=23.7.4'] -> jsonpatch[version='>=1.32']\n",
            "conda[version='>=23.9.0'] -> jsonpatch[version='>=1.32']\n",
            "jsonpatch\n",
            "\n",
            "Package libstdcxx-ng conflicts for:\n",
            "pysocks -> python[version='>=3.7,<3.8.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.2.0|>=7.3.0|>=7.5.0']\n",
            "libmamba -> libstdcxx-ng[version='>=11.2.0|>=7.5.0']\n",
            "libnghttp2 -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0']\n",
            "requests -> python[version='>=3.9,<3.10.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "pluggy -> python[version='>=3.7,<3.8.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.2.0|>=7.3.0|>=7.5.0']\n",
            "libsolv -> libstdcxx-ng[version='>=11.2.0|>=7.5.0']\n",
            "libstdcxx-ng\n",
            "charset-normalizer -> python[version='>=3.5'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "zstd -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0']\n",
            "fmt -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0']\n",
            "urllib3 -> brotli-python[version='>=1.0.9'] -> libstdcxx-ng[version='>=11.2.0|>=7.3.0|>=7.5.0|>=7.2.0']\n",
            "brotlipy -> python[version='>=3.7,<3.8.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.2.0|>=7.3.0|>=7.5.0']\n",
            "conda-libmamba-solver -> libmambapy[version='>=1.5.1'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libedit -> ncurses[version='>=6.2,<7.0a0'] -> libstdcxx-ng[version='>=7.2.0|>=7.3.0']\n",
            "wheel -> python[version='>=3.9,<3.10.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "cryptography -> python[version='>=3.9,<3.10.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "reproc-cpp -> libstdcxx-ng[version='>=7.5.0']\n",
            "boltons -> python[version='>=3.7,<3.8.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.2.0|>=7.3.0|>=7.5.0']\n",
            "libffi -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "krb5 -> libstdcxx-ng[version='>=7.2.0|>=7.3.0']\n",
            "conda-content-trust -> python[version='>=3.9,<3.10.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "sqlite -> ncurses[version='>=6.2,<7.0a0'] -> libstdcxx-ng[version='>=7.3.0']\n",
            "zstandard -> libstdcxx-ng[version='>=7.3.0']\n",
            "tqdm -> python[version='>=3.8,<3.9.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libcurl -> libnghttp2[version='>=1.57.0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "pycosat -> python[version='>=3.8,<3.9.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "jsonpatch -> python -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.9,<3.10.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0']\n",
            "ncurses -> libstdcxx-ng[version='>=7.2.0|>=7.3.0']\n",
            "readline -> ncurses[version='>=6.2,<7.0a0'] -> libstdcxx-ng[version='>=7.2.0|>=7.3.0']\n",
            "python=3.12 -> expat[version='>=2.5.0,<3.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0']\n",
            "conda-package-handling -> python[version='>=3.8,<3.9.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "setuptools -> python[version='>=3.8,<3.9.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libmambapy -> libstdcxx-ng[version='>=11.2.0|>=7.5.0']\n",
            "cffi -> libffi[version='>=3.4,<3.5'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "pyopenssl -> python[version='>=3.8,<3.9.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "pycparser -> python[version='>=3.6'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libarchive -> lz4-c[version='>=1.9.4,<1.10.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "zstandard -> python[version='>=3.9,<3.10.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.2.0']\n",
            "conda-package-streaming -> python[version='>=3.8,<3.9.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "packaging -> python[version='>=3.8,<3.9.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "lz4-c -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0']\n",
            "pip -> python[version='>=3.9,<3.10.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "libmambapy -> python[version='>=3.9,<3.10.0a0'] -> libstdcxx-ng[version='>=7.2.0|>=7.3.0']\n",
            "idna -> python[version='>=3.7,<3.8.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.2.0|>=7.3.0|>=7.5.0']\n",
            "certifi -> python[version='>=3.8,<3.9.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "yaml-cpp -> libstdcxx-ng[version='>=7.5.0']\n",
            "icu -> libstdcxx-ng[version='>=11.2.0|>=7.3.0|>=7.2.0']\n",
            "jsonpointer -> python -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "ruamel.yaml -> python[version='>=3.9,<3.10.0a0'] -> libstdcxx-ng[version='>=11.2.0|>=7.5.0|>=7.3.0|>=7.2.0']\n",
            "\n",
            "Package tk conflicts for:\n",
            "charset-normalizer -> python[version='>=3.5'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "cffi -> python[version='>=3.11,<3.12.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "libcurl -> krb5[version='>=1.17.1,<1.18.0a0'] -> tk[version='>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0']\n",
            "pycosat -> python[version='>=3.8,<3.9.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "urllib3 -> python[version='>=3.8,<3.9.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "jsonpatch -> python -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "krb5 -> tk[version='8.6.*|>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0']\n",
            "pyopenssl -> python[version='>=3.8,<3.9.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.11,<3.12.0a0'] -> tk[version='>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0']\n",
            "conda-libmamba-solver -> python[version='>=3.10,<3.11.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "zstandard -> python[version='>=3.11,<3.12.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "pycparser -> python[version='>=3.6'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "ruamel.yaml -> python[version='>=3.11,<3.12.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "libmambapy -> python[version='>=3.9,<3.10.0a0'] -> tk[version='>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "conda-content-trust -> python[version='>=3.11,<3.12.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> tk[version='>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "conda-package-streaming -> python[version='>=3.8,<3.9.0a0'] -> tk[version='>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> tk[version='>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0']\n",
            "brotlipy -> python[version='>=3.7,<3.8.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "requests -> python[version='>=3.9,<3.10.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "python=3.12 -> tk[version='>=8.6.12,<8.7.0a0']\n",
            "pysocks -> python[version='>=3.7,<3.8.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "packaging -> python[version='>=3.8,<3.9.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "tk\n",
            "cryptography -> python[version='>=3.11,<3.12.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "wheel -> python[version='>=3.9,<3.10.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "jsonpointer -> python -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "conda-package-handling -> python[version='>=3.10,<3.11.0a0'] -> tk[version='8.6.*|>=8.6.10,<8.7.0a0|>=8.6.11,<8.7.0a0|>=8.6.12,<8.7.0a0|>=8.6.8,<8.7.0a0|>=8.6.7,<8.7.0a0']\n",
            "\n",
            "Package libmambapy conflicts for:\n",
            "libmambapy\n",
            "conda-libmamba-solver -> libmambapy[version='0.22.*|>=0.23|>=1.0.0|>=1.4.1|>=1.5.1']\n",
            "\n",
            "Package ruamel.yaml.clib conflicts for:\n",
            "conda[version='>=23.9.0'] -> ruamel.yaml[version='>=0.11.14,<0.18'] -> ruamel.yaml.clib[version='>=0.1.2|>=0.2.6']\n",
            "ruamel.yaml -> ruamel.yaml.clib[version='>=0.1.2|>=0.2.6']\n",
            "\n",
            "Package libffi conflicts for:\n",
            "wheel -> python[version='>=3.9,<3.10.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "conda-package-handling -> python[version='>=3.10,<3.11.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "jsonpatch -> python -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.11,<3.12.0a0'] -> libffi[version='>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "brotlipy -> cffi[version='>=1.0.0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3|>=3.4,<3.5|>=3.3,<3.4.0a0|>=3.4,<4.0a0']\n",
            "urllib3 -> python[version='>=3.8,<3.9.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "zstandard -> cffi[version='>=1.11'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3|>=3.4,<3.5|>=3.3,<3.4.0a0|>=3.4,<4.0a0']\n",
            "conda-package-streaming -> python[version='>=3.8,<3.9.0a0'] -> libffi[version='>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "ruamel.yaml -> python[version='>=3.11,<3.12.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "python=3.12 -> libffi[version='>=3.4,<4.0a0']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> libffi[version='>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "libmambapy -> python[version='>=3.9,<3.10.0a0'] -> libffi[version='>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "requests -> python[version='>=3.9,<3.10.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "pycparser -> python[version='>=3.6'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "packaging -> python[version='>=3.8,<3.9.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "conda-content-trust -> python[version='>=3.11,<3.12.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "cryptography -> cffi[version='>=1.12'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3|>=3.4,<3.5|>=3.3,<3.4.0a0|>=3.4,<4.0a0']\n",
            "pyopenssl -> python[version='>=3.8,<3.9.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "libffi\n",
            "cffi -> python[version='>=3.11,<3.12.0a0'] -> libffi[version='3.2.*|>=3.4,<4.0a0']\n",
            "conda-libmamba-solver -> python[version='>=3.10,<3.11.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "jsonpointer -> python -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "pysocks -> python[version='>=3.7,<3.8.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "charset-normalizer -> python[version='>=3.5'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "cffi -> libffi[version='>=3.2.1,<3.3a0|>=3.3|>=3.4,<3.5|>=3.3,<3.4.0a0']\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> libffi[version='>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "pycosat -> python[version='>=3.8,<3.9.0a0'] -> libffi[version='3.2.*|>=3.2.1,<3.3a0|>=3.3,<3.4.0a0|>=3.4,<3.5|>=3.4,<4.0a0']\n",
            "\n",
            "Package six conflicts for:\n",
            "packaging -> six\n",
            "conda-content-trust -> six\n",
            "conda-package-handling -> six\n",
            "conda[version='>=23.9.0'] -> pyopenssl[version='>=16.2.0'] -> six[version='>=1.5.2']\n",
            "urllib3 -> cryptography[version='>=1.3.4'] -> six[version='>=1.4.1|>=1.5.2']\n",
            "pyopenssl -> cryptography[version='>=3.3'] -> six[version='>=1.4.1']\n",
            "pyopenssl -> six[version='>=1.5.2']\n",
            "conda-content-trust -> cryptography[version='<41.0.0a0'] -> six[version='>=1.4.1']\n",
            "cryptography -> six[version='>=1.4.1']\n",
            "\n",
            "Package boltons conflicts for:\n",
            "conda-libmamba-solver -> boltons[version='>=23.0.0']\n",
            "conda[version='>=23.9.0'] -> boltons[version='>=23.0.0']\n",
            "boltons\n",
            "\n",
            "Package sqlite conflicts for:\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.32.3,<4.0a0|>=3.30.0,<4.0a0|>=3.37.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.37.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "cryptography -> python[version='>=3.11,<3.12.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.38.2,<4.0a0|>=3.38.3,<4.0a0|>=3.39.2,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.37.2,<4.0a0']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.38.2,<4.0a0|>=3.38.3,<4.0a0|>=3.39.2,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.2,<4.0a0|>=3.41.1,<4.0a0|>=3.37.2,<4.0a0']\n",
            "libmambapy -> python[version='>=3.9,<3.10.0a0'] -> sqlite[version='>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.38.2,<4.0a0|>=3.38.3,<4.0a0|>=3.39.2,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.2,<4.0a0|>=3.41.1,<4.0a0|>=3.37.2,<4.0a0']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> sqlite[version='>=3.36.0,<4.0a0|>=3.37.2,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0']\n",
            "pycparser -> python[version='>=3.6'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.37.2,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "packaging -> python[version='>=3.8,<3.9.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.2,<4.0a0|>=3.41.1,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0|>=3.37.2,<4.0a0']\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.32.3,<4.0a0|>=3.30.0,<4.0a0|>=3.37.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "brotlipy -> python[version='>=3.7,<3.8.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.2,<4.0a0|>=3.32.3,<4.0a0|>=3.30.0,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0|>=3.41.1,<4.0a0|>=3.37.2,<4.0a0']\n",
            "zstandard -> python[version='>=3.11,<3.12.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.37.2,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "conda-package-handling -> python[version='>=3.10,<3.11.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.37.2,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "conda-package-streaming -> python[version='>=3.8,<3.9.0a0'] -> sqlite[version='>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.2,<4.0a0|>=3.41.1,<4.0a0|>=3.37.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "pysocks -> python[version='>=3.7,<3.8.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.2,<4.0a0|>=3.41.1,<4.0a0|>=3.37.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0|>=3.32.3,<4.0a0|>=3.30.0,<4.0a0']\n",
            "pycosat -> python[version='>=3.8,<3.9.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.2,<4.0a0|>=3.41.1,<4.0a0|>=3.37.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "jsonpointer -> python -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.37.2,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.11,<3.12.0a0'] -> sqlite[version='>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.37.2,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "charset-normalizer -> python[version='>=3.5'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.37.2,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "ruamel.yaml -> python[version='>=3.11,<3.12.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.38.2,<4.0a0|>=3.38.3,<4.0a0|>=3.39.2,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.37.2,<4.0a0']\n",
            "urllib3 -> python[version='>=3.8,<3.9.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.2,<4.0a0|>=3.41.1,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0|>=3.37.2,<4.0a0']\n",
            "conda-content-trust -> python[version='>=3.11,<3.12.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.38.2,<4.0a0|>=3.38.3,<4.0a0|>=3.39.2,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.37.2,<4.0a0']\n",
            "jsonpatch -> python -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.37.2,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "python=3.12 -> sqlite[version='>=3.41.2,<4.0a0']\n",
            "requests -> python[version='>=3.9,<3.10.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.38.2,<4.0a0|>=3.38.3,<4.0a0|>=3.39.2,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.2,<4.0a0|>=3.41.1,<4.0a0|>=3.37.2,<4.0a0']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.37.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "sqlite\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> sqlite[version='>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.37.2,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.32.3,<4.0a0|>=3.30.0,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.2,<4.0a0|>=3.41.1,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0|>=3.37.2,<4.0a0']\n",
            "cffi -> python[version='>=3.11,<3.12.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0|>=3.37.2,<4.0a0']\n",
            "pyopenssl -> python[version='>=3.8,<3.9.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0|>=3.41.1,<4.0a0|>=3.37.2,<4.0a0']\n",
            "conda-libmamba-solver -> python[version='>=3.10,<3.11.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.37.2,<4.0a0|>=3.38.0,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.1,<4.0a0|>=3.41.2,<4.0a0|>=3.39.2,<4.0a0|>=3.38.3,<4.0a0|>=3.38.2,<4.0a0']\n",
            "wheel -> python[version='>=3.9,<3.10.0a0'] -> sqlite[version='>=3.20.1,<4.0a0|>=3.22.0,<4.0a0|>=3.23.1,<4.0a0|>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0|>=3.30.0,<4.0a0|>=3.30.1,<4.0a0|>=3.31.1,<4.0a0|>=3.32.3,<4.0a0|>=3.33.0,<4.0a0|>=3.35.4,<4.0a0|>=3.36.0,<4.0a0|>=3.38.0,<4.0a0|>=3.38.2,<4.0a0|>=3.38.3,<4.0a0|>=3.39.2,<4.0a0|>=3.39.3,<4.0a0|>=3.40.0,<4.0a0|>=3.40.1,<4.0a0|>=3.41.2,<4.0a0|>=3.41.1,<4.0a0|>=3.37.2,<4.0a0']\n",
            "\n",
            "Package ncurses conflicts for:\n",
            "pyopenssl -> python[version='>=3.8,<3.9.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "conda-libmamba-solver -> python[version='>=3.10,<3.11.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "brotlipy -> python[version='>=3.7,<3.8.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "requests -> python[version='>=3.9,<3.10.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "python=3.12 -> readline[version='>=8.0,<9.0a0'] -> ncurses[version='>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> ncurses[version='>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "cryptography -> python[version='>=3.11,<3.12.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> ncurses[version='>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "sqlite -> ncurses[version='>=6.2,<7.0a0|>=6.4,<7.0a0']\n",
            "jsonpointer -> python -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "wheel -> python[version='>=3.9,<3.10.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "readline -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "conda-package-handling -> python[version='>=3.10,<3.11.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "libedit -> ncurses[version='6.0.*|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0']\n",
            "pysocks -> python[version='>=3.7,<3.8.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "charset-normalizer -> python[version='>=3.5'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "pycosat -> python[version='>=3.8,<3.9.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "cffi -> python[version='>=3.11,<3.12.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "urllib3 -> python[version='>=3.8,<3.9.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.11,<3.12.0a0'] -> ncurses[version='>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "sqlite -> readline[version='>=8.0,<9.0a0'] -> ncurses[version='6.0.*|>=6.1,<7.0a0|>=6.3,<7.0a0']\n",
            "ncurses\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "conda-package-streaming -> python[version='>=3.8,<3.9.0a0'] -> ncurses[version='>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "zstandard -> python[version='>=3.11,<3.12.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "libmambapy -> python[version='>=3.9,<3.10.0a0'] -> ncurses[version='>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "pycparser -> python[version='>=3.6'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "python=3.12 -> ncurses[version='>=6.4,<7.0a0']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "jsonpatch -> python -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "ruamel.yaml -> python[version='>=3.11,<3.12.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "packaging -> python[version='>=3.8,<3.9.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "krb5 -> libedit[version='>=3.1.20221030,<3.2.0a0'] -> ncurses[version='6.0.*|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0']\n",
            "conda-content-trust -> python[version='>=3.11,<3.12.0a0'] -> ncurses[version='6.0.*|>=6.0,<7.0a0|>=6.1,<7.0a0|>=6.2,<7.0a0|>=6.3,<7.0a0|>=6.4,<7.0a0']\n",
            "\n",
            "Package reproc conflicts for:\n",
            "libmamba -> reproc[version='>=14.2|>=14.2,<15.0a0|>=14.2.4,<15.0a0']\n",
            "reproc-cpp -> reproc==14.2.4=h295c915_1\n",
            "libmambapy -> libmamba==1.5.1=hba0046a_0 -> reproc[version='>=14.2|>=14.2,<15.0a0|>=14.2.4,<15.0a0']\n",
            "libmamba -> reproc-cpp[version='>=14.2,<15.0a0'] -> reproc==14.2.4=h295c915_1\n",
            "reproc\n",
            "\n",
            "Package libmamba conflicts for:\n",
            "libmambapy -> libmamba[version='0.22.1|0.23.1|0.23.3|0.24.0|1.0.0|1.3.1|1.3.1|1.4.1|1.4.1|1.5.1',build='h1566912_0|hdb19cb5_0|hdb19cb5_0|haf1ee3a_0|hba0046a_0|h2dafd23_1|hdb19cb5_1|ha06983f_0|h1566912_0|h1566912_0|h1566912_0']\n",
            "libmamba\n",
            "conda-libmamba-solver -> libmambapy[version='>=1.5.1'] -> libmamba[version='0.22.1|0.23.1|0.23.3|0.24.0|1.0.0|1.3.1|1.3.1|1.4.1|1.4.1|1.5.1',build='h1566912_0|hdb19cb5_0|hdb19cb5_0|haf1ee3a_0|hba0046a_0|h2dafd23_1|hdb19cb5_1|ha06983f_0|h1566912_0|h1566912_0|h1566912_0']\n",
            "\n",
            "Package libedit conflicts for:\n",
            "libedit\n",
            "libcurl -> krb5[version='>=1.20.1,<1.21.0a0'] -> libedit[version='>=3.1.20170329,<3.2.0a0|>=3.1.20181209,<3.2.0a0|>=3.1.20191231,<3.2.0a0|>=3.1.20210216,<3.2.0a0|>=3.1.20221030,<3.2.0a0|>=3.1.20221030,<4.0a0|>=3.1.20210216,<4.0a0|>=3.1.20191231,<4.0a0|>=3.1.20181209,<4.0a0|>=3.1.20170329,<4.0a0']\n",
            "sqlite -> libedit[version='>=3.1.20170329,<3.2.0a0|>=3.1.20181209,<3.2.0a0|>=3.1.20191231,<3.2.0a0']\n",
            "krb5 -> libedit[version='>=3.1,<4.0a0|>=3.1.20170329,<3.2.0a0|>=3.1.20181209,<3.2.0a0|>=3.1.20191231,<3.2.0a0|>=3.1.20210216,<3.2.0a0|>=3.1.20221030,<3.2.0a0|>=3.1.20221030,<4.0a0|>=3.1.20210216,<4.0a0|>=3.1.20191231,<4.0a0|>=3.1.20181209,<4.0a0|>=3.1.20170329,<4.0a0']\n",
            "\n",
            "Package openssl conflicts for:\n",
            "cryptography -> python[version='>=3.11,<3.12.0a0'] -> openssl[version='>=1.0.2l,<1.0.3a|>=1.1.1e,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=3.0.8,<4.0a0']\n",
            "cryptography -> openssl[version='1.0.*|3.*|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1a,<1.1.2a|>=1.0.2p,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2m,<1.0.3a']\n",
            "charset-normalizer -> python[version='>=3.5'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.11,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a|>=1.0.2l,<1.0.3a']\n",
            "libmambapy -> python[version='>=3.9,<3.10.0a0'] -> openssl[version='>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1u,<1.1.2a|>=3.0.9,<4.0a0|>=1.1.1m,<1.1.2a']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.11,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1m,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.0.2l,<1.0.3a']\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=3.0.11,<4.0a0|>=1.0.2l,<1.0.3a']\n",
            "wheel -> python[version='>=3.9,<3.10.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1m,<1.1.2a|>=3.0.11,<4.0a0|>=1.0.2l,<1.0.3a']\n",
            "libcurl -> krb5[version='>=1.20.1,<1.21.0a0'] -> openssl[version='3.*|>=1.1.1h,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.0.2m,<1.0.3a']\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.0.2l,<1.0.3a|>=3.0.11,<4.0a0']\n",
            "jsonpatch -> python -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.11,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a|>=1.0.2l,<1.0.3a']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.11,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.0.2l,<1.0.3a']\n",
            "pysocks -> python[version='>=3.7,<3.8.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=3.0.10,<4.0a0|>=1.1.1v,<1.1.2a|>=1.1.1u,<1.1.2a|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1t,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.0.2l,<1.0.3a']\n",
            "urllib3 -> python[version='>=3.8,<3.9.0a0'] -> openssl[version='1.0.*|3.*|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1m,<1.1.2a|<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1a,<1.1.2a|>=1.0.2p,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2m,<1.0.3a|>=3.0.11,<4.0a0|>=1.0.2l,<1.0.3a']\n",
            "conda-package-handling -> python[version='>=3.10,<3.11.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.0.2l,<1.0.3a|3.*|>=1.1.1f,<1.1.2a|>=3.0.11,<4.0a0']\n",
            "pycparser -> python[version='>=3.6'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.11,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a|>=1.0.2l,<1.0.3a']\n",
            "packaging -> python[version='>=3.8,<3.9.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1m,<1.1.2a|>=3.0.11,<4.0a0|>=1.0.2l,<1.0.3a']\n",
            "openssl\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> openssl[version='>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> openssl[version='>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0']\n",
            "ruamel.yaml -> python[version='>=3.11,<3.12.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1m,<1.1.2a|>=1.0.2l,<1.0.3a']\n",
            "conda-package-streaming -> python[version='>=3.8,<3.9.0a0'] -> openssl[version='>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1m,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a']\n",
            "requests -> python[version='>=3.9,<3.10.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1m,<1.1.2a|>=3.0.11,<4.0a0|>=1.0.2l,<1.0.3a']\n",
            "libmamba -> openssl[version='>=1.1.1m,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.8,<4.0a0']\n",
            "cffi -> python[version='>=3.11,<3.12.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.0.2l,<1.0.3a']\n",
            "zstandard -> python[version='>=3.11,<3.12.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a|>=1.0.2l,<1.0.3a']\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1m,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=3.0.11,<4.0a0|>=1.0.2l,<1.0.3a']\n",
            "libssh2 -> openssl[version='3.*|>=1.1.1u,<1.1.2a|>=3.0.8,<4.0a0|>=1.1.1n,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1a,<1.1.2a|>=1.0.2n,<1.0.3a|>=1.0.2m,<1.0.3a']\n",
            "conda-content-trust -> cryptography[version='>=41'] -> openssl[version='1.0.*|3.*|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1m,<1.1.2a|<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1a,<1.1.2a|>=1.0.2p,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2m,<1.0.3a|>=3.0.11,<4.0a0|>=1.0.2l,<1.0.3a']\n",
            "libcurl -> openssl[version='>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.11,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0']\n",
            "libmamba -> libarchive[version='>=3.6.2,<3.7.0a0'] -> openssl[version='3.*|>=3.0.11,<4.0a0|>=3.0.9,<4.0a0|>=1.1.1u,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1f,<1.1.2a|>=1.1.1l,<1.1.2a']\n",
            "pyopenssl -> cryptography[version='>=38.0.0,<42,!=40.0.0,!=40.0.1'] -> openssl[version='1.0.*|3.*|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|<1.1.2a|>=3.0.8,<4.0a0|>=1.1.1t,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1a,<1.1.2a|>=1.0.2p,<1.0.3a|>=1.0.2o,<1.0.3a|>=3.0.11,<4.0a0|>=1.0.2n,<1.0.3a|>=1.0.2m,<1.0.3a|>=1.0.2l,<1.0.3a']\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.11,<3.12.0a0'] -> openssl[version='>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a']\n",
            "brotlipy -> python[version='>=3.7,<3.8.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.0.2l,<1.0.3a|>=3.0.10,<4.0a0|>=1.1.1v,<1.1.2a|>=3.0.9,<4.0a0|>=1.1.1u,<1.1.2a|>=3.0.8,<4.0a0|>=1.1.1t,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1m,<1.1.2a']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.11,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1m,<1.1.2a|>=1.0.2l,<1.0.3a']\n",
            "libarchive -> openssl[version='3.*|>=3.0.8,<4.0a0|>=1.1.1t,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1f,<1.1.2a|>=1.1.1a,<1.1.2a|>=1.0.2p,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2m,<1.0.3a|>=1.0.2l,<1.0.3a']\n",
            "libnghttp2 -> openssl[version='3.*|>=1.1.1u,<1.1.2a|>=3.0.11,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1l,<1.1.2a|>=1.1.1h,<1.1.2a']\n",
            "python=3.12 -> openssl[version='>=3.0.11,<4.0a0']\n",
            "conda-libmamba-solver -> libmambapy[version='>=1.5.1'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a|>=3.0.11,<4.0a0|>=1.0.2l,<1.0.3a']\n",
            "pycosat -> python[version='>=3.8,<3.9.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1m,<1.1.2a|>=1.1.1o,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.0.2l,<1.0.3a']\n",
            "jsonpointer -> python -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.1.1n,<1.1.2a|>=1.1.1q,<1.1.2a|>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1u,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.11,<4.0a0|>=3.0.9,<4.0a0|>=3.0.8,<4.0a0|>=1.1.1o,<1.1.2a|>=1.0.2l,<1.0.3a']\n",
            "krb5 -> openssl[version='1.0.*|3.*|>=1.1.1u,<1.1.2a|>=3.0.8,<4.0a0|>=1.1.1s,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1a,<1.1.2a|>=1.0.2o,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2m,<1.0.3a|>=1.0.2l,<1.0.3a']\n",
            "libmambapy -> openssl[version='>=1.1.1s,<1.1.2a|>=1.1.1t,<1.1.2a|>=1.1.1v,<1.1.2a|>=3.0.10,<4.0a0|>=3.0.8,<4.0a0']\n",
            "\n",
            "Package lz4-c conflicts for:\n",
            "zstd -> lz4-c[version='>=1.9.2,<1.10.0a0|>=1.9.3,<1.10.0a0|>=1.9.4,<1.10.0a0']\n",
            "conda-package-handling -> libarchive[version='>=3.3.3'] -> lz4-c[version='>=1.8.1.2,<1.9.0a0|>=1.9.2,<1.10.0a0|>=1.9.3,<1.10.0a0|>=1.9.4,<1.10.0a0']\n",
            "libarchive -> lz4-c[version='>=1.8.0,<1.9.0a0|>=1.8.1.2,<1.9.0a0|>=1.9.2,<1.10.0a0|>=1.9.3,<1.10.0a0|>=1.9.4,<1.10.0a0']\n",
            "libmamba -> libarchive[version='>=3.6.2,<3.7.0a0'] -> lz4-c[version='>=1.9.2,<1.10.0a0|>=1.9.3,<1.10.0a0|>=1.9.4,<1.10.0a0']\n",
            "lz4-c\n",
            "\n",
            "Package tzdata conflicts for:\n",
            "conda-package-streaming -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "jsonpointer -> python -> tzdata\n",
            "packaging -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "charset-normalizer -> python[version='>=3.5'] -> tzdata\n",
            "conda-package-handling -> python[version='>=3.10,<3.11.0a0'] -> tzdata\n",
            "pyopenssl -> python[version='>=3.9,<3.10.0a0'] -> tzdata\n",
            "jsonpatch -> python -> tzdata\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "ruamel.yaml -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "conda-libmamba-solver -> python[version='>=3.10,<3.11.0a0'] -> tzdata\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> tzdata\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> tzdata\n",
            "libmambapy -> python[version='>=3.9,<3.10.0a0'] -> tzdata\n",
            "python=3.12 -> tzdata\n",
            "pycosat -> python[version='>=3.10,<3.11.0a0'] -> tzdata\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "pysocks -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> tzdata\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "cffi -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> tzdata\n",
            "urllib3 -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "tzdata\n",
            "zstandard -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "requests -> python[version='>=3.9,<3.10.0a0'] -> tzdata\n",
            "wheel -> python[version='>=3.9,<3.10.0a0'] -> tzdata\n",
            "pycparser -> python[version='>=3.6'] -> tzdata\n",
            "brotlipy -> python[version='>=3.9,<3.10.0a0'] -> tzdata\n",
            "conda-content-trust -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "cryptography -> python[version='>=3.11,<3.12.0a0'] -> tzdata\n",
            "\n",
            "Package zlib conflicts for:\n",
            "libcurl -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "libarchive -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "zstd -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "brotlipy -> python[version='>=3.7,<3.8.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "krb5 -> tk[version='>=8.6.8,<8.7.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0']\n",
            "wheel -> python[version='>=3.9,<3.10.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "conda-package-handling -> python[version='>=3.10,<3.11.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "libsolv -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "cryptography -> python[version='>=3.11,<3.12.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "ruamel.yaml -> python[version='>=3.11,<3.12.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "jsonpointer -> python -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "urllib3 -> python[version='>=3.8,<3.9.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "charset-normalizer -> python[version='>=3.5'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "sqlite -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0|>=1.2.13,<2.0a0']\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "tk -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0']\n",
            "cffi -> python[version='>=3.11,<3.12.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "conda-package-streaming -> python[version='>=3.8,<3.9.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "pycosat -> python[version='>=3.8,<3.9.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.11,<3.12.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "jsonpatch -> python -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "libmambapy -> python[version='>=3.9,<3.10.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "zstandard -> python[version='>=3.11,<3.12.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "python=3.12 -> sqlite[version='>=3.41.2,<4.0a0'] -> zlib[version='>=1.2.12,<1.3.0a0|>=1.2.13,<2.0a0']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "zlib\n",
            "pycparser -> python[version='>=3.6'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "libmamba -> libarchive[version='>=3.6.2,<3.7.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "pyopenssl -> python[version='>=3.8,<3.9.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "conda-libmamba-solver -> python[version='>=3.10,<3.11.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "pysocks -> python[version='>=3.7,<3.8.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "libnghttp2 -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "packaging -> python[version='>=3.8,<3.9.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "conda-content-trust -> python[version='>=3.11,<3.12.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "pcre2 -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "python=3.12 -> zlib[version='>=1.2.13,<1.3.0a0']\n",
            "requests -> python[version='>=3.9,<3.10.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0|>=1.2.12,<1.3.0a0|>=1.2.13,<1.3.0a0']\n",
            "\n",
            "Package tqdm conflicts for:\n",
            "conda-libmamba-solver -> conda[version='>=23.7.4'] -> tqdm[version='>=4']\n",
            "tqdm\n",
            "conda[version='>=23.9.0'] -> tqdm[version='>=4']\n",
            "conda-package-handling -> tqdm\n",
            "\n",
            "Package readline conflicts for:\n",
            "pysocks -> python[version='>=3.7,<3.8.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "conda-package-handling -> python[version='>=3.10,<3.11.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "conda-content-trust -> python[version='>=3.11,<3.12.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "cryptography -> python[version='>=3.11,<3.12.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "urllib3 -> python[version='>=3.8,<3.9.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "jsonpointer -> python -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "brotlipy -> python[version='>=3.7,<3.8.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> readline[version='>=8.0,<9.0a0']\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "charset-normalizer -> python[version='>=3.5'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "sqlite -> readline[version='>=8.0,<9.0a0']\n",
            "pycosat -> python[version='>=3.8,<3.9.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "cffi -> python[version='>=3.11,<3.12.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "jsonpatch -> python -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "ruamel.yaml -> python[version='>=3.11,<3.12.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> readline[version='>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "conda-package-streaming -> python[version='>=3.8,<3.9.0a0'] -> readline[version='>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.11,<3.12.0a0'] -> readline[version='>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "conda-libmamba-solver -> python[version='>=3.10,<3.11.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "python=3.12 -> readline[version='>=8.0,<9.0a0']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "readline\n",
            "libmambapy -> python[version='>=3.9,<3.10.0a0'] -> readline[version='>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "zstandard -> python[version='>=3.11,<3.12.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "pyopenssl -> python[version='>=3.8,<3.9.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "requests -> python[version='>=3.9,<3.10.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "wheel -> python[version='>=3.9,<3.10.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "packaging -> python[version='>=3.8,<3.9.0a0'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "pycparser -> python[version='>=3.6'] -> readline[version='7.*|>=7.0,<8.0a0|>=8.0,<9.0a0|>=8.2,<9.0a0']\n",
            "\n",
            "Package ld_impl_linux-64 conflicts for:\n",
            "pyopenssl -> python[version='>=3.8,<3.9.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "cryptography -> python[version='>=3.11,<3.12.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "jsonpointer -> python -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "pysocks -> python[version='>=3.7,<3.8.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "pycparser -> python[version='>=3.6'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "brotlipy -> python[version='>=3.7,<3.8.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "packaging -> python[version='>=3.8,<3.9.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "charset-normalizer -> python[version='>=3.5'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "cffi -> python[version='>=3.11,<3.12.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "urllib3 -> python[version='>=3.8,<3.9.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "wheel -> python[version='>=3.9,<3.10.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "truststore -> python[version='>=3.10,<3.11.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "conda-package-handling -> python[version='>=3.10,<3.11.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "boltons -> python[version='>=3.11,<3.12.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "idna -> python[version='>=3.11,<3.12.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "jsonpatch -> python -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "conda-package-streaming -> python[version='>=3.8,<3.9.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "conda[version='>=23.9.0'] -> python[version='>=3.11,<3.12.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "pluggy -> python[version='>=3.11,<3.12.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "ruamel.yaml -> python[version='>=3.11,<3.12.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "python=3.12 -> ld_impl_linux-64[version='>=2.35.1']\n",
            "conda-libmamba-solver -> python[version='>=3.10,<3.11.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "libmambapy -> python[version='>=3.9,<3.10.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "requests -> python[version='>=3.9,<3.10.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "ld_impl_linux-64\n",
            "zstandard -> python[version='>=3.11,<3.12.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "pycosat -> python[version='>=3.8,<3.9.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "tqdm -> python[version='>=3.11,<3.12.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "conda-content-trust -> python[version='>=3.11,<3.12.0a0'] -> ld_impl_linux-64[version='>=2.35.1|>=2.36.1']\n",
            "\n",
            "Package pycosat conflicts for:\n",
            "pycosat\n",
            "conda-libmamba-solver -> conda[version='>=23.7.4'] -> pycosat[version='>=0.6.3']\n",
            "conda[version='>=23.9.0'] -> pycosat[version='>=0.6.3']\n",
            "\n",
            "Package idna conflicts for:\n",
            "conda[version='>=23.9.0'] -> requests[version='>=2.27.0,<3'] -> idna[version='>=2.5,<4']\n",
            "requests -> urllib3[version='>=1.21.1,<2'] -> idna[version='>=2.0.0']\n",
            "urllib3 -> idna[version='>=2.0.0']\n",
            "requests -> idna[version='>=2.5,<2.7|>=2.5,<2.8|>=2.5,<2.9|>=2.5,<3|>=2.5,<4']\n",
            "urllib3 -> cryptography[version='>=1.3.4'] -> idna[version='>=2.1']\n",
            "idna\n",
            "cryptography -> idna[version='>=2.1']\n",
            "conda-content-trust -> cryptography[version='<41.0.0a0'] -> idna[version='>=2.1']\n",
            "pyopenssl -> cryptography[version='>=2.8'] -> idna[version='>=2.1']\n",
            "\n",
            "Package pcre2 conflicts for:\n",
            "libmambapy -> libsolv[version='>=0.7.20,<0.8.0a0'] -> pcre2[version='>=10.37,<10.38.0a0|>=10.42,<10.43.0a0']\n",
            "libsolv -> pcre2[version='>=10.37,<10.38.0a0|>=10.42,<10.43.0a0']\n",
            "pcre2\n",
            "libmamba -> libsolv[version='>=0.7.24,<0.8.0a0'] -> pcre2[version='>=10.37,<10.38.0a0|>=10.42,<10.43.0a0']\n",
            "\n",
            "Package conda-package-handling conflicts for:\n",
            "conda-libmamba-solver -> conda[version='>=23.7.4'] -> conda-package-handling[version='>=1.3.0|>=2.2.0']\n",
            "conda[version='>=23.9.0'] -> conda-package-handling[version='>=2.2.0']\n",
            "conda-package-handling\n",
            "\n",
            "Package pluggy conflicts for:\n",
            "pluggy\n",
            "conda-libmamba-solver -> conda[version='>=23.7.4'] -> pluggy[version='>=1.0.0']\n",
            "conda[version='>=23.9.0'] -> pluggy[version='>=1.0.0']\n",
            "\n",
            "Package lzo conflicts for:\n",
            "conda-package-handling -> libarchive[version='>=3.3.3'] -> lzo[version='>=2.10,<3.0a0']\n",
            "libarchive -> lzo[version='>=2.10,<3.0a0']\n",
            "\n",
            "Package cryptography conflicts for:\n",
            "conda[version='>=23.9.0'] -> pyopenssl[version='>=16.2.0'] -> cryptography[version='>=1.9|>=2.1.4|>=2.2.1|>=2.8|>=3.3|>=35.0|>=38.0.0,<40|>=38.0.0,<42,!=40.0.0,!=40.0.1']\n",
            "cryptography\n",
            "urllib3 -> pyopenssl[version='>=0.14'] -> cryptography[version='>=1.9|>=2.1.4|>=2.2.1|>=2.8|>=3.3|>=35.0|>=38.0.0,<40|>=38.0.0,<42,!=40.0.0,!=40.0.1']\n",
            "pyopenssl -> cryptography[version='>=1.9|>=2.1.4|>=2.2.1|>=2.8|>=3.3|>=35.0|>=38.0.0,<40|>=38.0.0,<42,!=40.0.0,!=40.0.1']\n",
            "conda-content-trust -> cryptography[version='<41.0.0a0|>=41']\n",
            "requests -> urllib3[version='>=1.21.1,<2'] -> cryptography[version='>=1.3.4']\n",
            "urllib3 -> cryptography[version='>=1.3.4']\n",
            "\n",
            "Package cffi conflicts for:\n",
            "brotlipy -> cffi[version='>=1.0.0']\n",
            "urllib3 -> cryptography[version='>=1.3.4'] -> cffi[version='!=1.11.3,>=1.8|>=1.12|>=1.8|>=1.7|>=1.0.0']\n",
            "conda-package-streaming -> zstandard[version='>=0.15'] -> cffi[version='>=1.11|>=1.8']\n",
            "pyopenssl -> cryptography[version='>=38.0.0,<42,!=40.0.0,!=40.0.1'] -> cffi[version='!=1.11.3,>=1.8|>=1.12|>=1.8|>=1.7']\n",
            "cffi\n",
            "conda-package-handling -> zstandard[version='>=0.15'] -> cffi[version='>=1.11|>=1.8']\n",
            "cryptography -> cffi[version='!=1.11.3,>=1.8|>=1.12|>=1.8|>=1.7']\n",
            "zstandard -> cffi[version='>=1.11|>=1.8']\n",
            "conda-content-trust -> cryptography[version='>=41'] -> cffi[version='!=1.11.3,>=1.8|>=1.12|>=1.8|>=1.7']\n",
            "\n",
            "Package colorama conflicts for:\n",
            "conda-package-handling -> tqdm -> colorama\n",
            "conda[version='>=23.9.0'] -> tqdm[version='>=4'] -> colorama\n",
            "tqdm -> colorama\n",
            "\n",
            "Package setuptools conflicts for:\n",
            "pip -> setuptools\n",
            "wheel -> setuptools\n",
            "conda[version='>=23.9.0'] -> setuptools[version='>=60.0.0']\n",
            "conda[version='>=23.9.0'] -> ruamel.yaml[version='>=0.11.14,<0.18'] -> setuptools\n",
            "setuptools\n",
            "conda-libmamba-solver -> conda[version='>=23.7.4'] -> setuptools[version='>=31.0.1|>=60.0.0']\n",
            "ruamel.yaml -> setuptools\n",
            "\n",
            "Package pyopenssl conflicts for:\n",
            "requests -> urllib3[version='>=1.21.1,<2'] -> pyopenssl[version='>=0.14']\n",
            "urllib3 -> pyopenssl[version='>=0.14']\n",
            "pyopenssl\n",
            "conda-libmamba-solver -> conda[version='>=23.7.4'] -> pyopenssl[version='>=16.2.0']\n",
            "conda[version='>=23.9.0'] -> pyopenssl[version='>=16.2.0']\n",
            "\n",
            "Package conda conflicts for:\n",
            "conda-libmamba-solver -> conda[version='>=22.11.0,<23.1.0a|>=22.11.0,<23.2.0a|>=23.3.1|>=23.5.0|>=23.7.4|>=4.13,<22.11.0a|>=4.12']\n",
            "conda[version='>=23.9.0']\n",
            "\n",
            "Package libcurl conflicts for:\n",
            "libmambapy -> libmamba==1.5.1=hba0046a_0 -> libcurl[version='>=7.80.0,<9.0a0|>=7.82.0,<9.0a0|>=7.85.0,<9.0a0|>=7.87.0,<9.0a0|>=7.88.1,<9.0a0|>=8.2.1,<9.0a0']\n",
            "libcurl\n",
            "libmamba -> libcurl[version='>=7.80.0,<9.0a0|>=7.82.0,<9.0a0|>=7.85.0,<9.0a0|>=7.87.0,<9.0a0|>=7.88.1,<9.0a0|>=8.2.1,<9.0a0']\n",
            "\n",
            "Package expat conflicts for:\n",
            "pluggy -> python -> expat[version='>=2.5.0,<3.0a0']\n",
            "conda-libmamba-solver -> python[version='>=3.6'] -> expat[version='>=2.5.0,<3.0a0']\n",
            "jsonpointer -> python -> expat[version='>=2.5.0,<3.0a0']\n",
            "conda-content-trust -> python -> expat[version='>=2.5.0,<3.0a0']\n",
            "certifi -> python[version='>=3.12,<3.13.0a0'] -> expat[version='>=2.5.0,<3.0a0']\n",
            "charset-normalizer -> python[version='>=3.5'] -> expat[version='>=2.5.0,<3.0a0']\n",
            "pyopenssl -> python[version='>=3.6'] -> expat[version='>=2.5.0,<3.0a0']\n",
            "setuptools -> python[version='>=3.12,<3.13.0a0'] -> expat[version='>=2.5.0,<3.0a0']\n",
            "packaging -> python[version='>=3.6'] -> expat[version='>=2.5.0,<3.0a0']\n",
            "pip -> python[version='>=3.12,<3.13.0a0'] -> expat[version='>=2.5.0,<3.0a0']\n",
            "python=3.12 -> expat[version='>=2.5.0,<3.0a0']\n",
            "jsonpatch -> python -> expat[version='>=2.5.0,<3.0a0']\n",
            "libarchive -> expat[version='>=2.2.4,<3.0a0']\n",
            "idna -> python[version='>=3.5'] -> expat[version='>=2.5.0,<3.0a0']\n",
            "conda-package-handling -> python -> expat[version='>=2.5.0,<3.0a0']\n",
            "requests -> python[version='>=3.6'] -> expat[version='>=2.5.0,<3.0a0']\n",
            "wheel -> python -> expat[version='>=2.5.0,<3.0a0']\n",
            "urllib3 -> python[version='<4.0'] -> expat[version='>=2.5.0,<3.0a0']\n",
            "tqdm -> python[version='>=2.7'] -> expat[version='>=2.5.0,<3.0a0']\n",
            "pycparser -> python[version='>=3.6'] -> expat[version='>=2.5.0,<3.0a0']\n",
            "\n",
            "Package brotlipy conflicts for:\n",
            "requests -> urllib3[version='>=1.21.1,<2'] -> brotlipy[version='>=0.6.0']\n",
            "urllib3 -> brotlipy[version='>=0.6.0']\n",
            "brotlipy\n",
            "\n",
            "Package krb5 conflicts for:\n",
            "libmamba -> libcurl[version='>=8.2.1,<9.0a0'] -> krb5[version='>=1.19.2,<1.20.0a0|>=1.19.4,<1.20.0a0|>=1.20.1,<1.21.0a0']\n",
            "krb5\n",
            "libcurl -> krb5[version='>=1.16.1,<1.17.0a0|>=1.17.1,<1.18.0a0|>=1.18.2,<1.19.0a0|>=1.19.1,<1.20.0a0|>=1.19.2,<1.20.0a0|>=1.19.4,<1.20.0a0|>=1.20.1,<1.21.0a0']\n",
            "\n",
            "Package c-ares conflicts for:\n",
            "libnghttp2 -> c-ares[version='>=1.16.1,<2.0a0|>=1.17.1,<2.0a0|>=1.19.0,<2.0a0|>=1.19.1,<2.0a0|>=1.7.5']\n",
            "libcurl -> libnghttp2[version='>=1.57.0'] -> c-ares[version='>=1.16.1,<2.0a0|>=1.17.1,<2.0a0|>=1.19.0,<2.0a0|>=1.19.1,<2.0a0|>=1.7.5']\n",
            "c-ares\n",
            "\n",
            "Package pybind11-abi conflicts for:\n",
            "pybind11-abi\n",
            "libmambapy -> pybind11-abi==4\n",
            "conda-libmamba-solver -> libmambapy[version='>=1.5.1'] -> pybind11-abi==4\n",
            "\n",
            "Package ipaddress conflicts for:\n",
            "pyopenssl -> cryptography[version='>=2.8'] -> ipaddress\n",
            "cryptography -> ipaddress\n",
            "urllib3 -> ipaddress\n",
            "conda-content-trust -> cryptography[version='<41.0.0a0'] -> ipaddress\n",
            "requests -> urllib3[version='>=1.21.1,<2'] -> ipaddress\n",
            "\n",
            "Package certifi conflicts for:\n",
            "requests -> urllib3[version='>=1.21.1,<2'] -> certifi\n",
            "ruamel.yaml -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']\n",
            "wheel -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']\n",
            "setuptools -> certifi[version='>=2016.09|>=2016.9.26']\n",
            "urllib3 -> certifi\n",
            "conda[version='>=23.9.0'] -> requests[version='>=2.27.0,<3'] -> certifi[version='>=2016.9.26|>=2017.4.17']\n",
            "pip -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']\n",
            "certifi\n",
            "requests -> certifi[version='>=2017.4.17']\n",
            "\n",
            "Package jsonpointer conflicts for:\n",
            "jsonpatch -> jsonpointer[version='>=1.9']\n",
            "conda[version='>=23.9.0'] -> jsonpatch[version='>=1.32'] -> jsonpointer[version='>=1.9']\n",
            "jsonpointer\n",
            "\n",
            "Package enum34 conflicts for:\n",
            "cryptography -> enum34\n",
            "pyopenssl -> cryptography[version='>=2.8'] -> enum34\n",
            "conda-content-trust -> cryptography[version='<41.0.0a0'] -> enum34\n",
            "urllib3 -> cryptography[version='>=1.3.4'] -> enum34\n",
            "\n",
            "Package libarchive conflicts for:\n",
            "conda-package-handling -> libarchive[version='>=3.3.3']\n",
            "conda-package-handling -> python-libarchive-c -> libarchive\n",
            "libarchive\n",
            "libmamba -> libarchive[version='>=3.4.2,<3.5.0a0|>=3.5.2,<3.6.0a0|>=3.6.1,<3.7.0a0|>=3.6.2,<3.7.0a0']\n",
            "libmambapy -> libmamba==1.5.1=hba0046a_0 -> libarchive[version='>=3.4.2,<3.5.0a0|>=3.5.2,<3.6.0a0|>=3.6.1,<3.7.0a0|>=3.6.2,<3.7.0a0']\n",
            "\n",
            "Package wheel conflicts for:\n",
            "pip -> wheel\n",
            "wheel\n",
            "\n",
            "Package libssh2 conflicts for:\n",
            "libmamba -> libcurl[version='>=8.2.1,<9.0a0'] -> libssh2[version='>=1.10.0|>=1.10.0,<2.0a0|>=1.9.0,<2.0a0']\n",
            "libcurl -> libssh2[version='>=1.10.0|>=1.10.0,<2.0a0|>=1.9.0,<2.0a0|>=1.8.2,<2.0a0|>=1.8.0,<2.0a0']\n",
            "libssh2\n",
            "\n",
            "Package urllib3 conflicts for:\n",
            "requests -> urllib3[version='>=1.21.1,<1.23|>=1.21.1,<1.24|>=1.21.1,<1.25|>=1.21.1,<1.26,!=1.25.0,!=1.25.1|>=1.21.1,<1.27|>=1.21.1,<2']\n",
            "urllib3\n",
            "conda[version='>=23.9.0'] -> requests[version='>=2.27.0,<3'] -> urllib3[version='>=1.21.1,<1.27|>=1.21.1,<2']\n",
            "\n",
            "Package yaml-cpp conflicts for:\n",
            "conda-libmamba-solver -> libmambapy[version='>=1.5.1'] -> yaml-cpp[version='>=0.7.0,<0.8.0a0']\n",
            "libmamba -> yaml-cpp[version='>=0.7.0,<0.8.0a0']\n",
            "libmambapy -> yaml-cpp[version='>=0.7.0,<0.8.0a0']\n",
            "yaml-cpp\n",
            "\n",
            "Package importlib-metadata conflicts for:\n",
            "conda-libmamba-solver -> importlib-metadata\n",
            "conda-libmamba-solver -> importlib_metadata -> importlib-metadata[version='>=1.1.3,<1.1.4.0a0|>=1.6.0,<1.6.1.0a0|>=1.6.1,<1.6.2.0a0|>=1.7.0,<1.7.1.0a0|>=2.0.0,<2.0.1.0a0|>=3.10.0,<3.10.1.0a0|>=4.11.3,<4.11.4.0a0|>=6.0.0,<6.0.1.0a0|>=4.8.2,<4.8.3.0a0|>=4.8.1,<4.8.2.0a0|>=4.6.4,<4.6.5.0a0|>=3.7.3,<3.7.4.0a0']\n",
            "\n",
            "Package requests conflicts for:\n",
            "requests\n",
            "conda-libmamba-solver -> conda[version='>=23.7.4'] -> requests[version='>=2.18.4,<3|>=2.20.1,<3|>=2.27.0,<3']\n",
            "conda[version='>=23.9.0'] -> requests[version='>=2.27.0,<3']\n",
            "\n",
            "Package libnghttp2 conflicts for:\n",
            "libnghttp2\n",
            "libcurl -> libnghttp2[version='>=1.41.0,<2.0a0|>=1.46.0,<2.0a0|>=1.46.0|>=1.52.0,<2.0a0|>=1.52.0|>=1.57.0|>=1.57.0,<2.0a0']\n",
            "libmamba -> libcurl[version='>=8.2.1,<9.0a0'] -> libnghttp2[version='>=1.46.0,<2.0a0|>=1.46.0|>=1.52.0,<2.0a0|>=1.52.0|>=1.57.0|>=1.57.0,<2.0a0']\n",
            "\n",
            "Package packaging conflicts for:\n",
            "conda-libmamba-solver -> conda[version='>=23.7.4'] -> packaging[version='>=23.0']\n",
            "conda[version='>=23.9.0'] -> packaging[version='>=23.0']\n",
            "packaging\n",
            "\n",
            "Package fmt conflicts for:\n",
            "fmt\n",
            "libmamba -> fmt[version='>=8.1.1,<9.0a0|>=9.1.0,<10.0a0']\n",
            "libmambapy -> fmt[version='>=8.1.1,<9.0a0|>=9.1.0,<10.0a0']\n",
            "conda-libmamba-solver -> libmambapy[version='>=1.5.1'] -> fmt[version='>=8.1.1,<9.0a0|>=9.1.0,<10.0a0']\n",
            "\n",
            "Package pycparser conflicts for:\n",
            "zstandard -> cffi[version='>=1.11'] -> pycparser\n",
            "cryptography -> cffi[version='>=1.12'] -> pycparser\n",
            "cffi -> pycparser\n",
            "pycparser\n",
            "brotlipy -> cffi[version='>=1.0.0'] -> pycparser\n",
            "\n",
            "Package zstd conflicts for:\n",
            "libmambapy -> libmamba==1.5.1=hba0046a_0 -> zstd[version='>=1.5.2,<1.6.0a0|>=1.5.4,<1.6.0a0']\n",
            "zstd\n",
            "libmamba -> zstd[version='>=1.5.2,<1.6.0a0|>=1.5.4,<1.6.0a0']\n",
            "libarchive -> zstd[version='>=1.3.3,<1.3.4.0a0|>=1.3.7,<1.3.8.0a0|>=1.4.4,<1.5.0a0|>=1.5.2,<1.6.0a0']\n",
            "libmamba -> libarchive[version='>=3.4.2,<3.5.0a0'] -> zstd[version='>=1.4.4,<1.5.0a0']\n",
            "conda-package-handling -> libarchive[version='>=3.3.3'] -> zstd[version='>=1.3.3,<1.3.4.0a0|>=1.3.7,<1.3.8.0a0|>=1.4.4,<1.5.0a0|>=1.5.2,<1.6.0a0']\n",
            "\n",
            "Package charset-normalizer conflicts for:\n",
            "conda[version='>=23.9.0'] -> requests[version='>=2.27.0,<3'] -> charset-normalizer[version='>=2,<3|>=2,<4|>=2.0.0,<3|>=2.0.0,<2.1.0']\n",
            "charset-normalizer\n",
            "requests -> charset-normalizer[version='>=2,<3|>=2,<4|>=2.0.0,<3|>=2.0.0,<2.1.0']\n",
            "\n",
            "Package icu conflicts for:\n",
            "icu\n",
            "libarchive -> libxml2[version='>=2.10.3,<2.11.0a0'] -> icu[version='>=58.2,<59.0a0|>=73.1,<74.0a0']\n",
            "\n",
            "Package libgomp conflicts for:\n",
            "libgcc-ng -> _openmp_mutex -> libgomp[version='>=7.5.0']\n",
            "libgomp\n",
            "_openmp_mutex -> libgomp[version='>=7.5.0']\n",
            "\n",
            "Package libsolv conflicts for:\n",
            "conda-libmamba-solver -> libmambapy[version='>=0.23'] -> libsolv[version='>=0.7.20,<0.8.0a0']\n",
            "libmambapy -> libmamba==1.5.1=hba0046a_0 -> libsolv[version='>=0.7.22|>=0.7.22,<0.8.0a0|>=0.7.24,<0.8.0a0']\n",
            "libmambapy -> libsolv[version='>=0.7.20,<0.8.0a0']\n",
            "libmamba -> libsolv[version='>=0.7.20,<0.8.0a0|>=0.7.22|>=0.7.22,<0.8.0a0|>=0.7.24,<0.8.0a0']\n",
            "libsolv\n",
            "\n",
            "Package truststore conflicts for:\n",
            "truststore\n",
            "conda-libmamba-solver -> conda[version='>=23.7.4'] -> truststore[version='>=0.8.0']\n",
            "conda[version='>=23.9.0'] -> truststore[version='>=0.8.0']\n",
            "\n",
            "Package pathlib2 conflicts for:\n",
            "pluggy -> importlib_metadata[version='>=0.12'] -> pathlib2\n",
            "conda-libmamba-solver -> importlib_metadata -> pathlib2\n",
            "\n",
            "Package ruamel.yaml conflicts for:\n",
            "ruamel.yaml\n",
            "conda-libmamba-solver -> conda[version='>=23.7.4'] -> ruamel.yaml[version='>=0.11.14,<0.18']\n",
            "conda[version='>=23.9.0'] -> ruamel.yaml[version='>=0.11.14,<0.18']\n",
            "\n",
            "Package reproc-cpp conflicts for:\n",
            "libmambapy -> libmamba==1.5.1=hba0046a_0 -> reproc-cpp[version='>=14.2|>=14.2,<15.0a0|>=14.2.4,<15.0a0']\n",
            "reproc-cpp\n",
            "libmamba -> reproc-cpp[version='>=14.2|>=14.2,<15.0a0|>=14.2.4,<15.0a0']\n",
            "\n",
            "Package libxml2 conflicts for:\n",
            "libmamba -> libarchive[version='>=3.6.2,<3.7.0a0'] -> libxml2[version='>=2.10.3,<2.11.0a0|>=2.9.14,<2.10.0a0|>=2.9.12,<2.10.0a0|>=2.9.9,<2.10.0a0']\n",
            "libarchive -> libxml2[version='>=2.10.3,<2.11.0a0|>=2.9.14,<2.10.0a0|>=2.9.12,<2.10.0a0|>=2.9.9,<2.10.0a0|>=2.9.8,<2.10.0a0|>=2.9.7,<2.10.0a0|>=2.9.4,<2.10.0a0']\n",
            "conda-package-handling -> libarchive[version='>=3.3.3'] -> libxml2[version='>=2.10.3,<2.11.0a0|>=2.9.14,<2.10.0a0|>=2.9.12,<2.10.0a0|>=2.9.9,<2.10.0a0|>=2.9.8,<2.10.0a0']\n",
            "libxml2\n",
            "\n",
            "Package conda-package-streaming conflicts for:\n",
            "conda-package-handling -> conda-package-streaming[version='>=0.7.0|>=0.9.0']\n",
            "conda-package-streaming\n",
            "conda[version='>=23.9.0'] -> conda-package-handling[version='>=2.2.0'] -> conda-package-streaming[version='>=0.9.0']\n",
            "\n",
            "Package pysocks conflicts for:\n",
            "pysocks\n",
            "requests -> urllib3[version='>=1.21.1,<2'] -> pysocks[version='>=1.5.6,<2.0,!=1.5.7']\n",
            "urllib3 -> pysocks[version='>=1.5.6,<2.0,!=1.5.7']\n",
            "\n",
            "Package brotli-python conflicts for:\n",
            "requests -> urllib3[version='>=1.21.1,<2'] -> brotli-python[version='>=1.0.9']\n",
            "urllib3 -> brotli-python[version='>=1.0.9']\n",
            "\n",
            "Package libev conflicts for:\n",
            "libcurl -> libnghttp2[version='>=1.57.0'] -> libev[version='>=4.11|>=4.33,<4.34.0a0']\n",
            "libnghttp2 -> libev[version='>=4.11|>=4.33,<4.34.0a0']\n",
            "libev\n",
            "\n",
            "Package zstandard conflicts for:\n",
            "conda-package-streaming -> zstandard[version='>=0.15']\n",
            "zstandard\n",
            "conda[version='>=23.9.0'] -> conda-package-handling[version='>=2.2.0'] -> zstandard[version='>=0.15']\n",
            "conda-package-handling -> zstandard[version='>=0.15']\n",
            "\n",
            "Package spdlog-fmt-embed conflicts for:\n",
            "libmamba -> spdlog-fmt-embed[version='>=1.9.2,<2.0a0']\n",
            "libmambapy -> libmamba==0.22.1=h1566912_0 -> spdlog-fmt-embed[version='>=1.9.2,<2.0a0']The following specifications were found to be incompatible with your system:\n",
            "\n",
            "  - feature:/linux-64::__glibc==2.35=0\n",
            "  - feature:|@/linux-64::__glibc==2.35=0\n",
            "  - brotlipy -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n",
            "  - bzip2 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n",
            "  - c-ares -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - cffi -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - conda-package-handling -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - cryptography -> libgcc-ng -> __glibc[version='>=2.17']\n",
            "  - fmt -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - icu -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - krb5 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - libarchive -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - libcurl -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - libedit -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - libev -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n",
            "  - libffi -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - libgcc-ng -> __glibc[version='>=2.17']\n",
            "  - libmamba -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - libmambapy -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - libnghttp2 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - libsolv -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - libssh2 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - libstdcxx-ng -> __glibc[version='>=2.17']\n",
            "  - libuuid -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - lz4-c -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - ncurses -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - openssl -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n",
            "  - pcre2 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - pycosat -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - python=3.12 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - readline -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - reproc -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n",
            "  - reproc-cpp -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n",
            "  - ruamel.yaml -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - sqlite -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - tk -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n",
            "  - xz -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - yaml-cpp -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']\n",
            "  - zlib -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - zstandard -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "  - zstd -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']\n",
            "\n",
            "Your installed version is: 2.35\n",
            "\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!conda install -c anaconda biopython==1.76"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Dpmg5VPQQ3w2",
        "outputId": "a7f45d83-e420-4ee3-a135-ce94722bc4ea"
      },
      "execution_count": 101,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting package metadata (current_repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "Solving environment: | \b\bunsuccessful initial attempt using frozen solve. Retrying with flexible solve.\n",
            "Collecting package metadata (repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "Solving environment: \\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bunsuccessful initial attempt using frozen solve. Retrying with flexible solve.\n",
            "Solving environment: | \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \n",
            "Found conflicts! Looking for incompatible packages.\n",
            "This can take several minutes.  Press CTRL-C to abort.\n",
            "\b\bfailed\n",
            "\n",
            "UnsatisfiableError: The following specifications were found\n",
            "to be incompatible with the existing python installation in your environment:\n",
            "\n",
            "Specifications:\n",
            "\n",
            "  - biopython==1.76 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']\n",
            "\n",
            "Your python: python=3.11\n",
            "\n",
            "If python is on the left-most side of the chain, that's the version you've asked for.\n",
            "When python appears to the right, that indicates that the thing on the left is somehow\n",
            "not available for the python version you are constrained to. Note that conda will not\n",
            "change your python version to a different minor version unless you explicitly specify\n",
            "that.\n",
            "\n",
            "The following specifications were found to be incompatible with your system:\n",
            "\n",
            "  - feature:/linux-64::__glibc==2.35=0\n",
            "  - feature:|@/linux-64::__glibc==2.35=0\n",
            "  - biopython==1.76 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']\n",
            "\n",
            "Your installed version is: 2.35\n",
            "\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.12 1"
      ],
      "metadata": {
        "id": "KQA_x2fGV63L"
      },
      "execution_count": 111,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!python3.12 --version"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "o9xTzDuhWAb9",
        "outputId": "801ff9d9-93e1-406f-b3ec-1435ba185b2b"
      },
      "execution_count": 113,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Python 3.12.0\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!mv /content/biopython-1.75/Bio/pairwise2.py /usr/local/lib/python3.12/dist-packages/Bio/"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "enphQ2O7WQ9c",
        "outputId": "d91fc21d-47ca-4e75-9c58-d891cd14b680"
      },
      "execution_count": 116,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "mv: cannot stat '/content/biopython-1.75/Bio/pairwise2.py': No such file or directory\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!mv /content/biopython-1.75/Bio/__init__.py /usr/local/lib/python3.12/dist-packages/Bio/"
      ],
      "metadata": {
        "id": "V92hoiAyWia7"
      },
      "execution_count": 118,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!mv /content/biopython-1.75/Bio/SubsMat/__init__.py /usr/local/lib/python3.12/dist-packages/Bio/SubsMat/\n",
        "!mv /content/biopython-1.75/Bio/Alphabet/__init__.py /usr/local/lib/python3.12/dist-packages/Bio/Alphabet/"
      ],
      "metadata": {
        "id": "SjJywNncWtDe"
      },
      "execution_count": 121,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!python3.12 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Uf1PeJGgQ8Pf",
        "outputId": "2fd9b9b5-cd79-441d-b20c-fec96bac9451"
      },
      "execution_count": 122,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 7, in <module>\n",
            "    import cv2\n",
            "ModuleNotFoundError: No module named 'cv2'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3.12 -m pip install opencv-python"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "7cvFpyg9ZmQu",
        "outputId": "8e3f09b1-1083-4c24-f7ba-9e686142808b"
      },
      "execution_count": 125,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting opencv-python\n",
            "  Downloading opencv_python-4.8.1.78-cp37-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (19 kB)\n",
            "Requirement already satisfied: numpy>=1.21.2 in /usr/local/lib/python3.12/dist-packages (from opencv-python) (1.26.1)\n",
            "Downloading opencv_python-4.8.1.78-cp37-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (61.7 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m61.7/61.7 MB\u001b[0m \u001b[31m13.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: opencv-python\n",
            "Successfully installed opencv-python-4.8.1.78\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3.12 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "8fYVlawsZxoD",
        "outputId": "a3c67eb9-d7fd-4e31-ca2b-1bee36d3b834"
      },
      "execution_count": 133,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 6, in <module>\n",
            "    import pandas as pd\n",
            "  File \"/usr/local/lib/python3.12/dist-packages/pandas/__init__.py\", line 46, in <module>\n",
            "    from pandas.core.api import (\n",
            "  File \"/usr/local/lib/python3.12/dist-packages/pandas/core/api.py\", line 9, in <module>\n",
            "    from pandas.core.dtypes.dtypes import (\n",
            "ImportError: cannot import name 'ArrowDtype' from 'pandas.core.dtypes.dtypes' (/usr/local/lib/python3.12/dist-packages/pandas/core/dtypes/dtypes/__init__.py)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!mv /usr/local/lib/python3.10/dist-packages/pandas/ /usr/local/lib/python3.12/dist-packages/pandas/"
      ],
      "metadata": {
        "id": "oQVQvK5CoBSB"
      },
      "execution_count": 134,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!mv /usr/local/lib/python3.11/dist-packages/pandas/core/dtypes/common.py /usr/local/lib/python3.12/dist-packages/pandas/core/dtypes/\n",
        "!mv /usr/local/lib/python3.11/dist-packages/pandas/core/dtypes/dtypes.py /usr/local/lib/python3.12/dist-packages/pandas/core/dtypes/\n",
        "!mv /usr/local/lib/python3.11/dist-packages/pandas/core/dtypes/base.py /usr/local/lib/python3.12/dist-packages/pandas/core/dtypes/\n",
        "!mv /usr/local/lib/python3.11/dist-packages/pandas/core/series.py /usr/local/lib/python3.12/dist-packages/pandas/core/\n",
        "!mv /usr/local/lib/python3.11/dist-packages/pandas/core/generic.py /usr/local/lib/python3.12/dist-packages/pandas/core/"
      ],
      "metadata": {
        "id": "2kI2ntBibgZq"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install pandas"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 516
        },
        "id": "qgEVPplgoYrS",
        "outputId": "739e7e96-8b5b-4646-c101-4a744c12519e"
      },
      "execution_count": 138,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting pandas\n",
            "  Obtaining dependency information for pandas from https://files.pythonhosted.org/packages/f1/c5/1e9c317a5e6af9280ad86a523ab6efe2ca70a0eb4bfb2220d8d08e255ae1/pandas-2.1.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading pandas-2.1.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (18 kB)\n",
            "Requirement already satisfied: numpy<2,>=1.23.2 in /usr/local/lib/python3.11/site-packages (from pandas) (1.26.1)\n",
            "Collecting python-dateutil>=2.8.2 (from pandas)\n",
            "  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m247.7/247.7 kB\u001b[0m \u001b[31m5.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting pytz>=2020.1 (from pandas)\n",
            "  Obtaining dependency information for pytz>=2020.1 from https://files.pythonhosted.org/packages/32/4d/aaf7eff5deb402fd9a24a1449a8119f00d74ae9c2efa79f8ef9994261fc2/pytz-2023.3.post1-py2.py3-none-any.whl.metadata\n",
            "  Downloading pytz-2023.3.post1-py2.py3-none-any.whl.metadata (22 kB)\n",
            "Collecting tzdata>=2022.1 (from pandas)\n",
            "  Downloading tzdata-2023.3-py2.py3-none-any.whl (341 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m341.8/341.8 kB\u001b[0m \u001b[31m8.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting six>=1.5 (from python-dateutil>=2.8.2->pandas)\n",
            "  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)\n",
            "Downloading pandas-2.1.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.2 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m12.2/12.2 MB\u001b[0m \u001b[31m47.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading pytz-2023.3.post1-py2.py3-none-any.whl (502 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m502.5/502.5 kB\u001b[0m \u001b[31m31.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: pytz, tzdata, six, python-dateutil, pandas\n",
            "Successfully installed pandas-2.1.2 python-dateutil-2.8.2 pytz-2023.3.post1 six-1.16.0 tzdata-2023.3\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.colab-display-data+json": {
              "pip_warning": {
                "packages": [
                  "dateutil",
                  "pytz",
                  "six"
                ]
              }
            }
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3.12 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Ngr1gGcBoQ3T",
        "outputId": "7f4e1025-d952-4438-acff-10e252a47fc3"
      },
      "execution_count": 151,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 6, in <module>\n",
            "    import pandas as pd\n",
            "  File \"/usr/local/lib/python3.12/dist-packages/pandas/__init__.py\", line 46, in <module>\n",
            "    from pandas.core.api import (\n",
            "  File \"/usr/local/lib/python3.12/dist-packages/pandas/core/api.py\", line 9, in <module>\n",
            "    from pandas.core.dtypes.dtypes import (\n",
            "ImportError: cannot import name 'ArrowDtype' from 'pandas.core.dtypes.dtypes' (/usr/local/lib/python3.12/dist-packages/pandas/core/dtypes/dtypes/__init__.py)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install cmake\n",
        "!python -m pip install cmake"
      ],
      "metadata": {
        "id": "DNtkIRgF3sq6"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!python3.12 textt.py"
      ],
      "metadata": {
        "id": "kqxol58Cpziu"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!sudo apt-get install gcc libpq-dev -y\n",
        "!sudo apt-get install python-dev  python-pip -y\n",
        "!sudo apt-get install python3-dev python3-pip python3-venv python3-wheel -y"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "YOUk2GS73wGY",
        "outputId": "aa56dd51-da5a-4b3b-91e9-7516cf1fd4bc"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "gcc is already the newest version (4:11.2.0-1ubuntu1).\n",
            "gcc set to manually installed.\n",
            "libpq-dev is already the newest version (14.9-0ubuntu0.22.04.1).\n",
            "libpq-dev set to manually installed.\n",
            "0 upgraded, 0 newly installed, 0 to remove and 18 not upgraded.\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "Package python-dev is not available, but is referred to by another package.\n",
            "This may mean that the package is missing, has been obsoleted, or\n",
            "is only available from another source\n",
            "However the following packages replace it:\n",
            "  python2-dev python2 python-dev-is-python3\n",
            "\n",
            "E: Package 'python-dev' has no installation candidate\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "python3-dev is already the newest version (3.10.6-1~22.04).\n",
            "python3-dev set to manually installed.\n",
            "The following additional packages will be installed:\n",
            "  python3-pip-whl python3-setuptools python3-setuptools-whl python3.10-venv\n",
            "Suggested packages:\n",
            "  python-setuptools-doc\n",
            "The following NEW packages will be installed:\n",
            "  python3-pip python3-pip-whl python3-setuptools python3-setuptools-whl\n",
            "  python3-venv python3-wheel python3.10-venv\n",
            "0 upgraded, 7 newly installed, 0 to remove and 18 not upgraded.\n",
            "Need to get 4,151 kB of archives.\n",
            "After this operation, 11.9 MB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 python3-setuptools all 59.6.0-1.2ubuntu0.22.04.1 [339 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 python3-wheel all 0.37.1-2ubuntu0.22.04.1 [32.0 kB]\n",
            "Get:3 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 python3-pip all 22.0.2+dfsg-1ubuntu0.3 [1,305 kB]\n",
            "Get:4 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 python3-pip-whl all 22.0.2+dfsg-1ubuntu0.3 [1,679 kB]\n",
            "Get:5 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 python3-setuptools-whl all 59.6.0-1.2ubuntu0.22.04.1 [788 kB]\n",
            "Get:6 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 python3.10-venv amd64 3.10.12-1~22.04.2 [5,724 B]\n",
            "Get:7 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 python3-venv amd64 3.10.6-1~22.04 [1,038 B]\n",
            "Fetched 4,151 kB in 1s (4,425 kB/s)\n",
            "debconf: unable to initialize frontend: Dialog\n",
            "debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 78, <> line 7.)\n",
            "debconf: falling back to frontend: Readline\n",
            "debconf: unable to initialize frontend: Readline\n",
            "debconf: (This frontend requires a controlling tty.)\n",
            "debconf: falling back to frontend: Teletype\n",
            "dpkg-preconfigure: unable to re-open stdin: \n",
            "Selecting previously unselected package python3-setuptools.\n",
            "(Reading database ... 121585 files and directories currently installed.)\n",
            "Preparing to unpack .../0-python3-setuptools_59.6.0-1.2ubuntu0.22.04.1_all.deb ...\n",
            "Unpacking python3-setuptools (59.6.0-1.2ubuntu0.22.04.1) ...\n",
            "Selecting previously unselected package python3-wheel.\n",
            "Preparing to unpack .../1-python3-wheel_0.37.1-2ubuntu0.22.04.1_all.deb ...\n",
            "Unpacking python3-wheel (0.37.1-2ubuntu0.22.04.1) ...\n",
            "Selecting previously unselected package python3-pip.\n",
            "Preparing to unpack .../2-python3-pip_22.0.2+dfsg-1ubuntu0.3_all.deb ...\n",
            "Unpacking python3-pip (22.0.2+dfsg-1ubuntu0.3) ...\n",
            "Selecting previously unselected package python3-pip-whl.\n",
            "Preparing to unpack .../3-python3-pip-whl_22.0.2+dfsg-1ubuntu0.3_all.deb ...\n",
            "Unpacking python3-pip-whl (22.0.2+dfsg-1ubuntu0.3) ...\n",
            "Selecting previously unselected package python3-setuptools-whl.\n",
            "Preparing to unpack .../4-python3-setuptools-whl_59.6.0-1.2ubuntu0.22.04.1_all.deb ...\n",
            "Unpacking python3-setuptools-whl (59.6.0-1.2ubuntu0.22.04.1) ...\n",
            "Selecting previously unselected package python3.10-venv.\n",
            "Preparing to unpack .../5-python3.10-venv_3.10.12-1~22.04.2_amd64.deb ...\n",
            "Unpacking python3.10-venv (3.10.12-1~22.04.2) ...\n",
            "Selecting previously unselected package python3-venv.\n",
            "Preparing to unpack .../6-python3-venv_3.10.6-1~22.04_amd64.deb ...\n",
            "Unpacking python3-venv (3.10.6-1~22.04) ...\n",
            "Setting up python3-setuptools-whl (59.6.0-1.2ubuntu0.22.04.1) ...\n",
            "Setting up python3-setuptools (59.6.0-1.2ubuntu0.22.04.1) ...\n",
            "Setting up python3-pip-whl (22.0.2+dfsg-1ubuntu0.3) ...\n",
            "Setting up python3-wheel (0.37.1-2ubuntu0.22.04.1) ...\n",
            "Setting up python3-pip (22.0.2+dfsg-1ubuntu0.3) ...\n",
            "Setting up python3.10-venv (3.10.12-1~22.04.2) ...\n",
            "Setting up python3-venv (3.10.6-1~22.04) ...\n",
            "Processing triggers for man-db (2.10.2-1) ...\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install wheel setuptools --upgrade\n",
        "!python -m pip install wheel setuptools --upgrade"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "BBqQ7_Kj30pX",
        "outputId": "9c1960c1-fc56-4e4b-82fb-c8e7c324ad86"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: wheel in /usr/local/lib/python3.12/dist-packages (0.41.2)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.12/dist-packages (68.2.2)\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0mRequirement already satisfied: wheel in /usr/local/lib/python3.12/dist-packages (0.41.2)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.12/dist-packages (68.2.2)\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "K1AUqx2_4Myr",
        "outputId": "f8d032f1-1a9c-44e3-a125-602350d65f38"
      },
      "execution_count": 153,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 1, in <module>\n",
            "    import google\n",
            "ModuleNotFoundError: No module named 'google'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "F9DP-l4e7UoN",
        "outputId": "5653f574-61d4-452f-dce6-c6a037957f9e"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2023-10-26 12:30:01--  https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh\n",
            "Resolving repo.anaconda.com (repo.anaconda.com)... 104.16.130.3, 104.16.131.3, 2606:4700::6810:8303, ...\n",
            "Connecting to repo.anaconda.com (repo.anaconda.com)|104.16.130.3|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 120771089 (115M) [application/x-sh]\n",
            "Saving to: ‘Miniconda3-latest-Linux-x86_64.sh’\n",
            "\n",
            "Miniconda3-latest-L 100%[===================>] 115.18M   254MB/s    in 0.5s    \n",
            "\n",
            "2023-10-26 12:30:02 (254 MB/s) - ‘Miniconda3-latest-Linux-x86_64.sh’ saved [120771089/120771089]\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!chmod +x Miniconda3-latest-Linux-x86_64.sh\n",
        "!./Miniconda3-latest-Linux-x86_64.sh -b -f -p /usr/local"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "0hTbnScJ7zKX",
        "outputId": "20179a7b-cbf7-40bf-d548-7258aadd9d2c"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PREFIX=/usr/local\n",
            "Unpacking payload ...\n",
            "\n",
            "Installing base environment...\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages\n",
            "\n",
            "Preparing transaction: - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\bdone\n",
            "Executing transaction: / \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\bdone\n",
            "installation finished.\n",
            "WARNING:\n",
            "    You currently have a PYTHONPATH environment variable set. This may cause\n",
            "    unexpected behavior when running the Python interpreter in Miniconda3.\n",
            "    For best results, please verify that your PYTHONPATH only points to\n",
            "    directories of packages that are compatible with the Python interpreter\n",
            "    in Miniconda3: /usr/local\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!sudo apt install python3.12 -y"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "g-OVQ1BhAZA1",
        "outputId": "0a42d684-cc12-404a-e7a8-b50077a678a3"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "python3.12 is already the newest version (3.12.0-1+jammy2).\n",
            "0 upgraded, 0 newly installed, 0 to remove and 30 not upgraded.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!sudo update-alternatives --config python3"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Qzj4tWW3-01U",
        "outputId": "44128ff2-822d-40ae-9d72-cb3053630c50"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "There are 3 choices for the alternative python3 (providing /usr/bin/python3).\n",
            "\n",
            "  Selection    Path                       Priority   Status\n",
            "------------------------------------------------------------\n",
            "* 0            /usr/bin/python3.12         2         auto mode\n",
            "  1            /usr/bin/python3.10         1         manual mode\n",
            "  2            /usr/bin/python3.12         2         manual mode\n",
            "  3            /usr/local/bin/python3.11   1         manual mode\n",
            "\n",
            "Press <enter> to keep the current choice[*], or type selection number: ^C\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!python3 textt.py"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "_k89FDwLARDj",
        "outputId": "d7c82d61-3782-4fd6-b515-319f039e0a04"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/usr/local/lib/python3.12/dist-packages/Bio/pairwise2.py:278: BiopythonDeprecationWarning: Bio.pairwise2 has been deprecated, and we intend to remove it in a future release of Biopython. As an alternative, please consider using Bio.Align.PairwiseAligner as a replacement, and contact the Biopython developers if you still need the Bio.pairwise2 module.\n",
            "  warnings.warn(\n",
            "Traceback (most recent call last):\n",
            "  File \"/content/textt.py\", line 5, in <module>\n",
            "    from Bio.SubsMat import MatrixInfo as matlist\n",
            "ModuleNotFoundError: No module named 'Bio.SubsMat'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip uninstall numpy -н\n",
        "!pip install numpy"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "BgY72rmlKtt9",
        "outputId": "bc696b17-332b-4560-f1b2-15532c0aad25"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "Usage:   \n",
            "  pip uninstall [options] <package> ...\n",
            "  pip uninstall [options] -r <requirements file> ...\n",
            "\n",
            "no such option: -н\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.11/site-packages (1.26.0)\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
            "\u001b[0m"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "QacU9HuypQM2"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}